{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Search engine"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Arguments to set: \n",
    "\n",
    "- search: string containing the search query\n",
    "        \n",
    "- pubdate_before: to get papers with publication dates before pubdate_before. format date : year-month-day\n",
    "- pubdate_after: to get papers with publication dates after pubdate_before. format date : year-month-day\n",
    "- pubdate_in: to get papers with publication dates between two dates. format date : year1-month1-day1 year2-month2-day2\n",
    "- tags: string containing tag words\n",
    "- topn: maximum number of results to be returned,\n",
    "- topn_sent: maximum number of sentences closely linked to search in a given paper\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To have an idea about what we are doing, here are some screenshots of what we get at the end of this notebook."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### <center> Interactive search bar <center> "
   ]
  },
  {
   "attachments": {
    "search.PNG": {
     "image/png": "iVBORw0KGgoAAAANSUhEUgAAATwAAAD0CAYAAAAGyZprAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAABV2SURBVHhe7Z3Ljts2FEDzTzO/45X/xHv/QldGgCAN0E1QFGi7mk8ouggKFLMN0ADdp1B1KVK6vCIl2WP5dc8BiIzEpyTymJQnw3cNAIATEB4AuAHhAYAbEB4AuAHhAYAbEB4AuAHhAYAbEB4AuAHhAYAbEB4AuAHhAYAbEB4AuAHhAYAbEN7K/PDDDwSCu3CrILyVueWHD7AGCM8xCA+8gfAcg/DAGwjPMQgPvIHwHIPwwBsIzzEID7yB8ByD8MAbCO8R+LJvnt9tm8/xcClzD/9l99TsXuKB8LJrnp4K5+TE66HZPO0aHbWIU/Ndk8k2vzaHjblHJUplhHNP4R4/Lb0ni++ftGvTHF7j4dF01yVt25xSyFrPObtnT/FkHYT3CKwkvNfDJuvccrzb7UbnwvGpHXqtgXA1FggvfnDkUntpdu25lE/u69Pm0JZ2Lt4ovLc+p1Wec37PpI45EN45+bxt3r17F4MV0OdmW4sLwkpx75ptipTzz9tm+6zz6HKem/2XmK6N324LZUww+/Clk/aDLg6Yl8I5OYgdetfOCtOnbTboQ7yN6zrsePBHinni+U1bV5hxpHwzZfXodFoAhfzZ9QvqHmR1qLyxXdm1K2TWHOq1ZYzqkjJ1+yL22sM9UuX0Mo3x8XTf9lCe/CxtjfVV8yQK9yzUm87Z67XpC/c2S6PqtNcn52rtk/PZPZsH4Z0NEdEgsi/75+adss7nVkbPwU42TvJFcQl6thZFOBTzpdm38huytoJ93jdfYrpUfn++O6oy//ClU6oOGjqXDBbdaePPcQD0s7+sM0o6NXj1ILUDtmcuTz7IRCSp7jA7KhqnG+h9lGqjzj+c15JokXrlvGnzKK9pWxF73anseNgWlF9/wl57Vo56Hi35fRiuJWvvZB7FqJ7Kswn1FO5xlsa0QfcVe32hrgXtC6QG1UF4Z8OIK8PG5XLMUXFafoI9TixNZ5h/+GrAS0eMHU06a/hRncs7fYs9zlCdeDKdZiqPGYA6raZal00/lCcDTIs0/JyVY/OaAV+jeA1DvjCwq8JT+UZtKeQJdM9y07ZtkJ0wlUdRvXeCuge1dKN26jSqDaP8C9sX0vEO77LEmVa23AyIxIbl5ji+TaGWo/3ytSSy0sxtNeFJ/9uEQZj+DbSfyDJotAxGHbXQ8UWUo6VJbYBEluXpOvuQTkJFFtksKjIqz4g+5DHnJtqfPhCytlsDFvJ157r0m8PLUJ/G5isep3p1/k7Em8MhztQV1TwKW09L9dnM3eNRWZV7mwjnptrXXZu9xSUQ3lpky8q5GZ15bzclvFI5Kwqv7dVhsL7sVEeTDlg6pztqdtwJqe+QOs7m6zkmj6QtlWGYrMuWl64tDkb97jIrx+ZdOPiqbUlUrsnmmypHLxXTdbQH2QeVJcujGF3zkc9zlF+nUfd76nqEUvvm8igQ3rmw783MsX6Hl8UVZRVnfyNxmXd4Kb5YxpmEJ52xHSj5p3Y3eOTlct/JbKfTx8W4mQ5+ZB6ZbfSDuDZoQ7vLA3Uqf7e8VPGm/lFeLYMao2tQg74l1FkqxObTx/a6s+NBeNnPk3kUup5iG3TZhXts8lTvty17afsWgvDOSHFZ2qOXtflyNnyJkeLit7JBakVxFcpZVXhdB7aDLyxn9LniIBiOkzRCyL7JjOXrvJFqHltXoJt1dOkHcYyppdPnTdmhPpV2VH+6hjZk1zZB6RpkMKc21Aa1zWeOw3MpXoeSXDgc8tXzKEw99ecplO6xfc6V+124L7PtkzwLJYjwHHPLDx9gDRCeYxAeeAPhOQbhgTcQnmMQHngD4TkG4YE3EJ5jEB54A+E5BuGBNxCeY+ThEwjewq2C8ADADQgPANyA8ADADQhvZUrvNwiERw+3CsJbmVt++ABrgPAcg/DAGwjPMQgPvIHwHIPwwBsIzzEID7yB8ByD8MAbCM8xCA+84Vp4sgfFsMn1FGbznBoL95KYY3m73gbCA28gPIQH4IYHEF635+u+3/lL7Qg2s5tXEMt2G/NZyajdwfROYkIoJ8b1+fRuYqmO0rl5ptt1PhAeeONBhNeKIe3zKnu+JrksEF5tf1iJy/aR7cUj9VWkOlFG2Ipxobmm2nVOEB5442GEN7hElp/L9msNMyllkuFYytSSmVrSqrRZ+UaMOt0M9XadF4QH3ngQ4WmxKEHMCm9hvpaShNKSszyjlHbpNBLy+mpU23VmEB544wGFd+oMT8/i7GzMxql8usyR8HQZy6m367wgPPDGgwivnT0lI8j7tvT+K8QNMgzv0Yzw+nwFGRbf4RUlWhbsqIy+XdNMteucIDzwxsPM8LbtMrC0bOwkJ+dFPvts1hVmUv23oXm+NmeYWYU48y2tLjOPS3lSHVHGxfLrTLUrk2ioT8UfKUeEB954GOEtlQkMIDzwBsK7NGEWlmZ9Nlz2OhAeeOMBhAengvDAGwjPMQgPvIHwHIPwwBsIzzEID7yB8ByD8MAbCM8x8vAJBG/hVkF4AOAGhAcAbkB4AOAGhAcAbkB4K1N6oUsgPHq4VRDeytzywwdYA4TnGIQH3kB4jkF44A2E5xiEB95AeI5BeOANhOcYhAfeQHiOQXjgDdfCC5vlLNrxZuFWiSvuMLYGCA+8gfAQHoAbHkB4sonPttn3WydObFtojvPtEK3Q1BaLZptGuxFPd15vyZjqKJ1bQLH884PwwBsPIrxWDGmT67BpdpTLAuH1+QpxxY24Q30VqU6UEfayXWSuifLPDMIDbzyM8AaXqM2prSwKQtIOGo6lTC2ZqSWtSpuVb8Sl0x3FqfnmQXjgjQcRnhZLeyaJa1Z4C/O1lOSYlpzlGaW0S6eRkNc3RbH8M4PwwBsPKLxTZ3h6FmdnVTZO5dNljoSny1jKRPlnBuGBNx5EeO0sKBlC3rel93IhbpBheI+m5BFmUSlfQYbFd3hWQOG4LNhRGX27Jpgq/8wgPPDGw8zwtu3ytLRs7CTXLQ2f9/ts1hVmeP23tFYq3awuxJlvaXWZeVzKk+qIMi6WX6devpFoqE+Va2U5A8IDbzyM8JbKBAYQHngD4V2aMAtLsz4bLnsdCA+88QDCg1NBeOANhOcYhAfeQHiOQXjgDYTnGIQH3kB4jkF44A2E5xh5+ASCt3CrIDwAcAPCAwA3IDwAcAPCAwA3ILyVKb3QJRAePdwqCG9lbvnhA6wBwnMMwgNvIDzHIDzwBsJzDMIDbyA8xyA88AbCcwzCA28gPMcgPPAGwltA2Oxn0c44ejvHCY7cbKfIGcpY9PBfD83m6al5imH3Es8HXppditscmtd4NiPk37UpFVmZJq7GbDm2bZYT26qZrG+i/FPzjViS9rU5bC5wL+4UhLeAmxTeGZh/+DIwNs0hjYgwCNJxPrBeD5vmyY6yl10c5HrgdIMtyzc30Kvl1NpmObWtmlPvxRvvYc+ytOG8ur9jznEv7hdHwpPNfrbNvt8CcWJ7Q3Ocb+dohaa2YjRbKnblxLg+n966MdVROjeDbqP8/Lxv9m0787qmmX340vHNYHjZpcEiA1kNCDMjkHRPMrBf8vMhXSY4IwRDtZzJtllObKvm1HvxhnuYsyBtuLe7ZqeENuYM9+KOcSa8VgZpM+ywubYShhZNQXh9vkJcccPuUF9FqhNlhD1pl9hqVN7xm34f//BlsEQ5hcG1QFx2YC7NZ7HljJgo59S2TnLCvQicmG82rczc5DifwY1Y5V7cD+6EN7hEbWJtBFQSknbQcCxlqnyTS1qVNivfiFGnm2IkPJXHHlc47uF3A2mTRobMXE4aOJJuGJDdEqwyyDWTA9C0zXJyW2uceC9Oztcyk1buY1fujPDOfi/uC2fCy/d97cU1I4zP24X5WkpyTMvM8oxS2qXTSMjrK6LLmGl/jeUPXwaFEYoMiFMHTjgnSycp86UdoF2+bjkVgx2x1QE4btuonLe0dcSp9+K4fEddQxaXC2/de3F/OBbeqTM8PYuzszEbp/JVBWXLWEi1vBZ7XGHRww8dvzRjkIGiBkRtgMwOHFNOjVI51bZZztTWan0z5Z+aL6OeNn1RYUMm154z3Ys7xZnw2tlTMlD2niuXYXiPpoQRZmkpn5GJxBXf4RUlVBbsqIwF79+yMop1qeMKsw+/OlCFfHlW/LZPGA0cGXDDjKKaz2LLmWyb5dS2Kk69F6fmG7E0bZeuWsw57sUd426Gt22Xp6VlYye5bkn5vN9ns64ww+u/pc3ztTnDrC7EmW9pdZl5XMqT6ogyLpZf4QLCC4OhHaw2DOND5BXPZ8skRWngyHukuXwWU8582ywntjVy6r04yz3sWZJ2TnjC2+7FPeNOeItk4oRbfvgAa4DwbpUwS0uzPhvOcx0ID7zhSHhgQXjgDYTnGIQH3kB4jkF44A2E5xiEB95AeI5BeOANhOcYefgEgrdwqyA8AHADwgMANyA8AHADwgMANyA8AHADwgMANyA8AHADwgMANyA8AHADwoM3899//zV//vln8/vvvzc//vhj+E17+VeO5bzEA9wCCA/exLdv35qffvqp+eWXX5q//vqr+ffff8N5+ffvv/9ufv311xD/zz//hPMA18SH8BbuPwHHIbL78OFD8+XL9J+GFvEdDgektzLZ/i4h0OctCA9OQpapMnObk11C0kl6lrfrkW91CiUcCE/vVpakVzonyPlts+8/KdW+FiLN532zl+0kY17PnUvezcky9hgkveSDNVB7QEMVlzO8+h61UYTpOOyBG/PFDX+O3tu2wG+//Xb34eeffw7v7I5B0ku+Unn3Em4X/SHOB3INh8LrZnFDX5Dj9MnYdZqho6hPTbsstsdHUBpI9xY+fvzYf0GxFEkv+Url3Uu4WUJ/1DM8Znwl/AlvJCrdMbT8Ovr3IggvCwjv9uGd3hhmeOG4JjxmeLUgS1P59vUYJD1L2suB8MbwDq/0Di/1Eh13RuE9AvLlg/yO3TFI+j/++CMewVkZvVOWD2u//bOGD+GFhy8vclMHiGILQXeKboa33Va+pUV4PenXUpbO8iTdp0+fmu/fv8czcG7y38PLX81AhxPhLcUuaWEK+UXi9+/fL/7F469fv8YzANcB4WUgvGMRicnMbeq/lkk8soNbAOHBm5FlqrybK/3xADnPMhZuBYQHAG5AeADgBoQHAG5AeADgBoQHAG5AeADgBoQHAG5AeADgBoQHAG5AeADgBoQHAG5AeADgBoQHAG5AeADgBoQHb0b++rH8yffSn4eS82y+DbcCwoM38e3bt/Cn3qf+AKjEy19HBrg2CA9ORmT34cOHxX/iHenBtbmc8B5t0xs28QkztznZJSSdpGd5uwLFvqg2qsp2M/MNwjsV58KTd3OyjD0GSS/54IzI9oxBbLovdrv0pd1Gw25mbFAbuJDwStsils4Jcn7b7Pst58xWie2n1X6b8h2x0XDfMWx9QqUt1fry9B6RLyTknd0xSHrJB+dB9lcO4+Oz/fDtxlDWj0d93idXm+HNboadjoOoYr5QRi3fFHkHsJ94ui1Z3FR9b+hEpR3t7y18/Pix/4JiKZJe8pXKu5dwk9i+KMfZuJD+z258wpWEZz6BsgfSCW/wkUzPY1zpwS6SztQDt3GqbVP1La57TGkg3VuQXz05BclXKu9ewk1i++JoIjDV/31xHeGNZKGkVng4MgMLArT5jpFOSJuWoVZw6byJn6rvmLoNpYF0b4EZ3g1R6qcIr8iNzvD0wznHDM8wWkJXypiq79S6HwR5Fye/bnIMkp53eCsw6oumTzvvq5rbfYeX1rRT782WPkg7xTfH1bZM1be07gdFvm2VXyo+BkkvG3PDmRn1xe5b2uJ7aedcTnjxIQzfgkaxhaAfVjfD224r39LqtEdIR6Q2LFltHt2WpfUN1+OR9Ht4S2d5ku7Tp0/N9+/f4xk4G8VxoPq0/rB3zgWFt5ROeL104GaR/znx/v37xf/T4uvXr/EMwHV4DOGFT7g0Q7MBea6JSExmblP/l1bikR3cAjcoPLg3ZJkq7+bkCwn711LkPMtYuBUQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeGtjPyvAwLBW7hVEN7K3PLDB1gDhOcYhAfeQHiOQXjgDYTnGIQH3kB4jkF44A2E5xiEB95AeI5BeOANhHdtjtjs59wc/fBfD83made8xMM6r81hs2kOr/FwTSbbJO14ana1Br/smqenpyFsDm2OyFS5i+8D3BoI79rck/AWc0HhTVIX3uth00oul9bLTkkPqT0kCO+q6C0Yo/TMpj/5lp16e7tts02bgIeobZ9nvNVjmdmHL4N+s2t2rTSCHKwEshmSloMWXiedbPbU89Ls+vxakPp8LDe0RZcR63ixYlJ5Y9vHwpM0JSErQcZr3YkEY3l9Ofo+xHYdSumEkLYSBxcH4V2bbIZndkUzs7/Rptx9Wsk3pFu6ufEi4elBqgd6kMYgmjBjGhL2wpNZ06Y41VNyEUSeUWhZnv78UGYgiuY1a1Mhb0kyqq4q8drH7WjRdU6lC/fItFm1FS4Pwrs2k0taLbJcam3GZt/P8IwoF7JMeGqAZse1WZLQyWnTCq0su5bq4M9FqusRqaby+p9HbdJ5jVQTIqXRScPUtdd+Fuxxhm0fXBqEd20KwpOZ3Gh5KumyXdq18ORQykl5lsnvbcJrCcdpuabl14lmczi0S8rKTEryluJGwlAzuz6POZfSj/KK2wrCq9WtsWXV6plKFwnvBvv7lMfBZUF41yYTnszU1Hu7UVxthmeQ5W4mxzJvFp4mW8oNQtKzsoxqWfUZXl+uvLdLdWXlLJzhZWVqVPqpa6/9LIzao+q3aeHiILxro6WWCa4lHA9Sq77Ds4K7hPDse7CK8PKfNUZGquzyO7yO7ttVFW/auOgdXktXTi6fMBMrirRFH9d+FmbjSvcCLgXCuzoyUxuWruELh7Q0Dd/E6m9quxlgiNvusxlecRk8w1tnePWlmpGcLaenmwF1+bUI9HmTz0pjVHYn0pC3+i1tJAqxD1rgU9de+1kwx0nQi9oDq4Pw7hU7GzyBW374AGuA8O6GNBNMYdkXE1MgPPAGwnMMwgNvIDzHIDzwBsJzDMIDbyA8xyA88AbCcwzCA28gPMfIwycQvIVbBeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgC4AeEBgBsQHgA4oWn+B5b0E08nBTHYAAAAAElFTkSuQmCC"
    }
   },
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "![search.PNG](attachment:search.PNG)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### <center> Results <center> "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "\n",
    "import gensim\n",
    "from gensim.models.doc2vec import Doc2Vec, TaggedDocument\n",
    "from gensim.models import Word2Vec\n",
    "from gensim.parsing.preprocessing import preprocess_string\n",
    "\n",
    "from ipywidgets import interact\n",
    "from IPython.display import display, HTML\n",
    "\n",
    "from gensim.summarization.textcleaner import get_sentences\n",
    "\n",
    "import re\n",
    "from sklearn.metrics.pairwise import cosine_similarity\n",
    "import time\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_data(dir_path):\n",
    "    \n",
    "    \"\"\"\n",
    "    Take as input a directory path containing json files from \n",
    "    biorxiv_medrxiv, comm_use_subset, noncomm_use_subset or custom_license.\n",
    "    Four dataframes are returned: papers_df, authors_df, affiliations_df, bib_entries_df\n",
    "    \"\"\"\n",
    "    \n",
    "    files = os.listdir(dir_path)\n",
    "    papers_df = pd.DataFrame(columns= [\"paper_id\", \"title\", \"authors\", \"abstract\", \"body_text\", \"bib_titles\", \"dataset\"])\n",
    "    authors_df = pd.DataFrame(columns= [\"author\", \"affiliation\"])\n",
    "    affiliations_df = pd.DataFrame(columns= [\"affiliation\", \"country\"])\n",
    "    bib_entries_df = pd.DataFrame(columns=[\"title\", \"authors\", \"year\", \"venue\"])\n",
    "    \n",
    "    line_author_df = 0\n",
    "    line_affiliations_df = 0\n",
    "    line_bib_entries_df = 0\n",
    "    \n",
    "    \n",
    "    \n",
    "    for line,file in enumerate(files):\n",
    "        \n",
    "        #loading information\n",
    "        n_files = len(files)\n",
    "        if (line%300==0)or(line==n_files-1):\n",
    "            print(\"Line\", line, \" \", round(100*line/n_files, 2),\"%\")\n",
    "            \n",
    "        file_path = os.path.join(\"../data/{}\".format(dir),file)\n",
    "    \n",
    "        with open(file_path) as f:\n",
    "            data = json.load(f)\n",
    "        \n",
    "        \n",
    "        #Papers\n",
    "        \n",
    "        paper_id = data[\"paper_id\"]\n",
    "        \n",
    "        title = data[\"metadata\"][\"title\"]\n",
    "        \n",
    "        authors, affiliations, countries = \"\", \"\", \"\"\n",
    "        for author in data[\"metadata\"][\"authors\"]:\n",
    "            first_last_name = author[\"first\"] + \" \" + author[\"last\"]\n",
    "            authors = authors + \" || \" + first_last_name\n",
    "            \n",
    "            if author[\"affiliation\"]=={}:\n",
    "                affiliation = \"NA\"  #Non Available\n",
    "                affiliations = affiliations + \" || \" + affiliation.strip()\n",
    "                \n",
    "                country = \"NA\"\n",
    "                countries = countries + \" || \" + country.strip()\n",
    "                continue\n",
    "                \n",
    "            affiliation = author[\"affiliation\"][\"laboratory\"] + \" \" + author[\"affiliation\"][\"institution\"]\n",
    "            affiliations = affiliations + \" || \" + affiliation.strip()\n",
    "            \n",
    "            if \"country\" not in author[\"affiliation\"][\"location\"].keys():\n",
    "                country = \"NA\"\n",
    "                countries = countries + \" || \" + country\n",
    "                continue\n",
    "            \n",
    "            country = author[\"affiliation\"][\"location\"][\"country\"]\n",
    "            countries = countries + \" || \" + country\n",
    "            \n",
    "        authors = authors[4:]   #extracting out the first separator \" || name1 || name2\" => \"name1 || name2\"\n",
    "        affiliations = affiliations[4:]\n",
    "        countries = countries[4:]\n",
    "        \n",
    "        abstract = \"\"\n",
    "        for info in data[\"abstract\"]:\n",
    "            abstract = abstract + \" \" + info[\"text\"]\n",
    "        abstract = abstract.strip()\n",
    "\n",
    "        body_text = \"\"\n",
    "        for info in data[\"body_text\"]:\n",
    "            body_text = body_text + \" \" + info[\"text\"]\n",
    "        body_text = body_text.strip()\n",
    "\n",
    "        bib_titles, bib_authors, years, venues = \"\", \"\", \"\", \"\"\n",
    "        for bib in data[\"bib_entries\"]:\n",
    "            bib_titles = bib_titles + \" || \" + data[\"bib_entries\"][bib][\"title\"]\n",
    "            \n",
    "            \n",
    "            year = data[\"bib_entries\"][bib][\"year\"]\n",
    "            \n",
    "            years = years + \" || \" + str(year)\n",
    "            \n",
    "            venue = data[\"bib_entries\"][bib][\"venue\"]\n",
    "            venues = venues + \" || \" + venue\n",
    "   \n",
    "            bib_author = [author[\"first\"] + \" \" + author[\"last\"] for author in data[\"bib_entries\"][bib][\"authors\"]]\n",
    "            bib_author = \" | \".join(bib_author)\n",
    "            bib_authors = bib_authors + \" || \" + bib_author\n",
    "            \n",
    "        bib_titles, bib_authors, years, venues = bib_titles[4:] , bib_authors[4:], years[4:], venues[4:] \n",
    "        \n",
    "        ##appending to paper_df\n",
    "        papers_df.loc[line, :] = [paper_id, title, authors, abstract, body_text, bib_titles, dir]\n",
    "        \n",
    "        \n",
    "        #Authors\n",
    "        authors_list = authors.split(\" || \")\n",
    "        affiliations_list = affiliations.split(\" || \")\n",
    "        \n",
    "        for i in range(len(authors_list)):\n",
    "            authors_df.loc[line_author_df, :] = authors_list[i],  affiliations_list[i]\n",
    "            \n",
    "            line_author_df+=1\n",
    "        \n",
    "        #Affiliations\n",
    "        countries_list = countries.split(\" || \")\n",
    "        \n",
    "        for i in range(len(affiliations_list)):\n",
    "            affiliations_df.loc[line_affiliations_df,:] = affiliations_list[i], countries_list[i]\n",
    "            \n",
    "            line_affiliations_df+=1\n",
    "        \n",
    "        \n",
    "        #Bibliograpphy\n",
    "        bib_titles_list = bib_titles.split(\" || \")\n",
    "        bib_authors_list = bib_authors.split(\" || \")\n",
    "        years_list = years.split(\" || \")\n",
    "        venues_list = venues.split(\" || \")\n",
    "        \n",
    "        \n",
    "        for i in range(len(bib_titles_list)):\n",
    "            \n",
    "            bib_entries_df.loc[line_bib_entries_df, :] = bib_titles_list[i], bib_authors_list[i], years_list[i], venues_list[i]\n",
    "            line_bib_entries_df+=1\n",
    "            \n",
    "    authors_df = authors_df.drop_duplicates().reset_index(drop=True)\n",
    "    affiliations_df = affiliations_df.drop_duplicates().reset_index(drop=True)\n",
    "    bib_entries_df = bib_entries_df.drop_duplicates().reset_index(drop=True)\n",
    "    \n",
    "    return papers_df, authors_df, affiliations_df, bib_entries_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paper_id</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>body_text</th>\n",
       "      <th>bib_titles</th>\n",
       "      <th>dataset</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0015023cc06b5362d332b3baf348d11567ca2fbb</td>\n",
       "      <td>The RNA pseudoknots in foot-and-mouth disease ...</td>\n",
       "      <td>Joseph Ward || Lidia Lasecka-Dykes || Chris Ne...</td>\n",
       "      <td>word count: 194 22 Text word count: 5168 23 24...</td>\n",
       "      <td>VP3, and VP0 (which is further processed to VP...</td>\n",
       "      <td>Genetic economy in 598 picornaviruses: Foot-an...</td>\n",
       "      <td>../data/unstructured/biorxiv_medrxiv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>004f0f8bb66cf446678dc13cf2701feec4f36d76</td>\n",
       "      <td>Healthcare-resource-adjusted vulnerabilities t...</td>\n",
       "      <td>Hanchu Zhou || Jiannan Yang || Kaicheng Tang |...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The 2019-nCoV epidemic has spread across China...</td>\n",
       "      <td>World Health Organizations. Novel Coronavirus ...</td>\n",
       "      <td>../data/unstructured/biorxiv_medrxiv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>00d16927588fb04d4be0e6b269fc02f0d3c2aa7b</td>\n",
       "      <td>Real-time, MinION-based, amplicon sequencing f...</td>\n",
       "      <td>Salman Butt || Eric Erwood || Jian Zhang || Ho...</td>\n",
       "      <td>Infectious bronchitis (IB) causes significant ...</td>\n",
       "      <td>Infectious bronchitis (IB), which is caused by...</td>\n",
       "      <td>Emergence of novel strains of avian infectious...</td>\n",
       "      <td>../data/unstructured/biorxiv_medrxiv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0139ea4ca580af99b602c6435368e7fdbefacb03</td>\n",
       "      <td>A Combined Evidence Approach to Prioritize Nip...</td>\n",
       "      <td>Nishi Kumari || Ayush Upadhyay || Kishan Kalia...</td>\n",
       "      <td>Nipah Virus (NiV) came into limelight recently...</td>\n",
       "      <td>Nipah is an infectious negative-sense single-s...</td>\n",
       "      <td>Molecular biology of Hendra and Nipah viruses ...</td>\n",
       "      <td>../data/unstructured/biorxiv_medrxiv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>013d9d1cba8a54d5d3718c229b812d7cf91b6c89</td>\n",
       "      <td>Assessing spread risk of Wuhan novel coronavir...</td>\n",
       "      <td>Shengjie Lai || Isaac Bogoch || Nick Ruktanonc...</td>\n",
       "      <td>Background: A novel coronavirus (2019-nCoV) em...</td>\n",
       "      <td>In December 2019, a cluster of patients with p...</td>\n",
       "      <td>A Novel Coronavirus Genome Identified in a Clu...</td>\n",
       "      <td>../data/unstructured/biorxiv_medrxiv</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                   paper_id  \\\n",
       "0  0015023cc06b5362d332b3baf348d11567ca2fbb   \n",
       "1  004f0f8bb66cf446678dc13cf2701feec4f36d76   \n",
       "2  00d16927588fb04d4be0e6b269fc02f0d3c2aa7b   \n",
       "3  0139ea4ca580af99b602c6435368e7fdbefacb03   \n",
       "4  013d9d1cba8a54d5d3718c229b812d7cf91b6c89   \n",
       "\n",
       "                                               title  \\\n",
       "0  The RNA pseudoknots in foot-and-mouth disease ...   \n",
       "1  Healthcare-resource-adjusted vulnerabilities t...   \n",
       "2  Real-time, MinION-based, amplicon sequencing f...   \n",
       "3  A Combined Evidence Approach to Prioritize Nip...   \n",
       "4  Assessing spread risk of Wuhan novel coronavir...   \n",
       "\n",
       "                                             authors  \\\n",
       "0  Joseph Ward || Lidia Lasecka-Dykes || Chris Ne...   \n",
       "1  Hanchu Zhou || Jiannan Yang || Kaicheng Tang |...   \n",
       "2  Salman Butt || Eric Erwood || Jian Zhang || Ho...   \n",
       "3  Nishi Kumari || Ayush Upadhyay || Kishan Kalia...   \n",
       "4  Shengjie Lai || Isaac Bogoch || Nick Ruktanonc...   \n",
       "\n",
       "                                            abstract  \\\n",
       "0  word count: 194 22 Text word count: 5168 23 24...   \n",
       "1                                                NaN   \n",
       "2  Infectious bronchitis (IB) causes significant ...   \n",
       "3  Nipah Virus (NiV) came into limelight recently...   \n",
       "4  Background: A novel coronavirus (2019-nCoV) em...   \n",
       "\n",
       "                                           body_text  \\\n",
       "0  VP3, and VP0 (which is further processed to VP...   \n",
       "1  The 2019-nCoV epidemic has spread across China...   \n",
       "2  Infectious bronchitis (IB), which is caused by...   \n",
       "3  Nipah is an infectious negative-sense single-s...   \n",
       "4  In December 2019, a cluster of patients with p...   \n",
       "\n",
       "                                          bib_titles  \\\n",
       "0  Genetic economy in 598 picornaviruses: Foot-an...   \n",
       "1  World Health Organizations. Novel Coronavirus ...   \n",
       "2  Emergence of novel strains of avian infectious...   \n",
       "3  Molecular biology of Hendra and Nipah viruses ...   \n",
       "4  A Novel Coronavirus Genome Identified in a Clu...   \n",
       "\n",
       "                                dataset  \n",
       "0  ../data/unstructured/biorxiv_medrxiv  \n",
       "1  ../data/unstructured/biorxiv_medrxiv  \n",
       "2  ../data/unstructured/biorxiv_medrxiv  \n",
       "3  ../data/unstructured/biorxiv_medrxiv  \n",
       "4  ../data/unstructured/biorxiv_medrxiv  "
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.read_csv(\"/kaggle/input/papers/papers.csv\", sep=\";\", nrows=100)\n",
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(100, 7)"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#dropping eventual duplicates\n",
    "df = df.drop_duplicates()\n",
    "df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/opt/conda/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3063: DtypeWarning: Columns (14) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>cord_uid</th>\n",
       "      <th>sha</th>\n",
       "      <th>source_x</th>\n",
       "      <th>title</th>\n",
       "      <th>doi</th>\n",
       "      <th>pmcid</th>\n",
       "      <th>pubmed_id</th>\n",
       "      <th>license</th>\n",
       "      <th>abstract</th>\n",
       "      <th>publish_time</th>\n",
       "      <th>authors</th>\n",
       "      <th>journal</th>\n",
       "      <th>Microsoft Academic Paper ID</th>\n",
       "      <th>WHO #Covidence</th>\n",
       "      <th>arxiv_id</th>\n",
       "      <th>has_pdf_parse</th>\n",
       "      <th>has_pmc_xml_parse</th>\n",
       "      <th>full_text_file</th>\n",
       "      <th>url</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>zjufx4fo</td>\n",
       "      <td>b2897e1277f56641193a6db73825f707eed3e4c9</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Sequence requirements for RNA strand transfer ...</td>\n",
       "      <td>10.1093/emboj/20.24.7220</td>\n",
       "      <td>PMC125340</td>\n",
       "      <td>11742998.0</td>\n",
       "      <td>unk</td>\n",
       "      <td>Nidovirus subgenomic mRNAs contain a leader se...</td>\n",
       "      <td>2001-12-17</td>\n",
       "      <td>Pasternak, Alexander O.; van den Born, Erwin; ...</td>\n",
       "      <td>The EMBO Journal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>http://europepmc.org/articles/pmc125340?pdf=re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ymceytj3</td>\n",
       "      <td>e3d0d482ebd9a8ba81c254cc433f314142e72174</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Crystal structure of murine sCEACAM1a[1,4]: a ...</td>\n",
       "      <td>10.1093/emboj/21.9.2076</td>\n",
       "      <td>PMC125375</td>\n",
       "      <td>11980704.0</td>\n",
       "      <td>unk</td>\n",
       "      <td>CEACAM1 is a member of the carcinoembryonic an...</td>\n",
       "      <td>2002-05-01</td>\n",
       "      <td>Tan, Kemin; Zelus, Bruce D.; Meijers, Rob; Liu...</td>\n",
       "      <td>The EMBO Journal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>http://europepmc.org/articles/pmc125375?pdf=re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>wzj2glte</td>\n",
       "      <td>00b1d99e70f779eb4ede50059db469c65e8c1469</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Synthesis of a novel hepatitis C virus protein...</td>\n",
       "      <td>10.1093/emboj/20.14.3840</td>\n",
       "      <td>PMC125543</td>\n",
       "      <td>11447125.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>Hepatitis C virus (HCV) is an important human ...</td>\n",
       "      <td>2001-07-16</td>\n",
       "      <td>Xu, Zhenming; Choi, Jinah; Yen, T.S.Benedict; ...</td>\n",
       "      <td>EMBO J</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2sfqsfm1</td>\n",
       "      <td>cf584e00f637cbd8f1bb35f3f09f5ed07b71aeb0</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Structure of coronavirus main proteinase revea...</td>\n",
       "      <td>10.1093/emboj/cdf327</td>\n",
       "      <td>PMC126080</td>\n",
       "      <td>12093723.0</td>\n",
       "      <td>unk</td>\n",
       "      <td>The key enzyme in coronavirus polyprotein proc...</td>\n",
       "      <td>2002-07-01</td>\n",
       "      <td>Anand, Kanchan; Palm, Gottfried J.; Mesters, J...</td>\n",
       "      <td>The EMBO Journal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>http://europepmc.org/articles/pmc126080?pdf=re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>i0zym7iq</td>\n",
       "      <td>dde02f11923815e6a16a31dd6298c46b109c5dfa</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Discontinuous and non-discontinuous subgenomic...</td>\n",
       "      <td>10.1093/emboj/cdf635</td>\n",
       "      <td>PMC136939</td>\n",
       "      <td>12456663.0</td>\n",
       "      <td>unk</td>\n",
       "      <td>Arteri-, corona-, toro- and roniviruses are ev...</td>\n",
       "      <td>2002-12-01</td>\n",
       "      <td>van Vliet, A.L.W.; Smits, S.L.; Rottier, P.J.M...</td>\n",
       "      <td>The EMBO Journal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>http://europepmc.org/articles/pmc136939?pdf=re...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   cord_uid                                       sha source_x  \\\n",
       "0  zjufx4fo  b2897e1277f56641193a6db73825f707eed3e4c9      PMC   \n",
       "1  ymceytj3  e3d0d482ebd9a8ba81c254cc433f314142e72174      PMC   \n",
       "2  wzj2glte  00b1d99e70f779eb4ede50059db469c65e8c1469      PMC   \n",
       "3  2sfqsfm1  cf584e00f637cbd8f1bb35f3f09f5ed07b71aeb0      PMC   \n",
       "4  i0zym7iq  dde02f11923815e6a16a31dd6298c46b109c5dfa      PMC   \n",
       "\n",
       "                                               title  \\\n",
       "0  Sequence requirements for RNA strand transfer ...   \n",
       "1  Crystal structure of murine sCEACAM1a[1,4]: a ...   \n",
       "2  Synthesis of a novel hepatitis C virus protein...   \n",
       "3  Structure of coronavirus main proteinase revea...   \n",
       "4  Discontinuous and non-discontinuous subgenomic...   \n",
       "\n",
       "                        doi      pmcid   pubmed_id license  \\\n",
       "0  10.1093/emboj/20.24.7220  PMC125340  11742998.0     unk   \n",
       "1   10.1093/emboj/21.9.2076  PMC125375  11980704.0     unk   \n",
       "2  10.1093/emboj/20.14.3840  PMC125543  11447125.0   no-cc   \n",
       "3      10.1093/emboj/cdf327  PMC126080  12093723.0     unk   \n",
       "4      10.1093/emboj/cdf635  PMC136939  12456663.0     unk   \n",
       "\n",
       "                                            abstract publish_time  \\\n",
       "0  Nidovirus subgenomic mRNAs contain a leader se...   2001-12-17   \n",
       "1  CEACAM1 is a member of the carcinoembryonic an...   2002-05-01   \n",
       "2  Hepatitis C virus (HCV) is an important human ...   2001-07-16   \n",
       "3  The key enzyme in coronavirus polyprotein proc...   2002-07-01   \n",
       "4  Arteri-, corona-, toro- and roniviruses are ev...   2002-12-01   \n",
       "\n",
       "                                             authors           journal  \\\n",
       "0  Pasternak, Alexander O.; van den Born, Erwin; ...  The EMBO Journal   \n",
       "1  Tan, Kemin; Zelus, Bruce D.; Meijers, Rob; Liu...  The EMBO Journal   \n",
       "2  Xu, Zhenming; Choi, Jinah; Yen, T.S.Benedict; ...            EMBO J   \n",
       "3  Anand, Kanchan; Palm, Gottfried J.; Mesters, J...  The EMBO Journal   \n",
       "4  van Vliet, A.L.W.; Smits, S.L.; Rottier, P.J.M...  The EMBO Journal   \n",
       "\n",
       "   Microsoft Academic Paper ID WHO #Covidence arxiv_id  has_pdf_parse  \\\n",
       "0                          NaN            NaN      NaN           True   \n",
       "1                          NaN            NaN      NaN           True   \n",
       "2                          NaN            NaN      NaN           True   \n",
       "3                          NaN            NaN      NaN           True   \n",
       "4                          NaN            NaN      NaN           True   \n",
       "\n",
       "   has_pmc_xml_parse  full_text_file  \\\n",
       "0               True  custom_license   \n",
       "1               True  custom_license   \n",
       "2               True  custom_license   \n",
       "3               True  custom_license   \n",
       "4               True  custom_license   \n",
       "\n",
       "                                                 url  \n",
       "0  http://europepmc.org/articles/pmc125340?pdf=re...  \n",
       "1  http://europepmc.org/articles/pmc125375?pdf=re...  \n",
       "2  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  \n",
       "3  http://europepmc.org/articles/pmc126080?pdf=re...  \n",
       "4  http://europepmc.org/articles/pmc136939?pdf=re...  "
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "meta_df = pd.read_csv(\"../input/CORD-19-research-challenge/metadata.csv\")\n",
    "meta_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "#dropping any paper not present in the meta dataset. NB: paper_id (df) == sha (meta_df)\n",
    "paper_in_meta = df.paper_id.isin(meta_df.sha)\n",
    "df = df[paper_in_meta]\n",
    "df = df.sort_values(by=\"paper_id\", axis=0).reset_index(drop=True)\n",
    "\n",
    "#Keeping meta data of papers present in the papers dataframe and sorting it acording to sha\n",
    "meta_df = meta_df.loc[meta_df.sha.isin(df.paper_id.values)]\n",
    "meta_df = meta_df.sort_values(by=\"sha\", axis=0).reset_index(drop=True)\n",
    "\n",
    "#We can now transfert information from meta to papers dataframe: both dataframe share now the same number of papers order in the same way\n",
    "df[\"title\"] = meta_df[\"title\"]\n",
    "df[\"abstract\"] = meta_df[\"abstract\"]\n",
    "df[\"authors\"] = meta_df[\"authors\"]\n",
    "df[\"publish_time\"] = meta_df[\"publish_time\"]\n",
    "df[\"url\"] = meta_df[\"url\"]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = df.loc[(df.title.notna()&df.authors.notna()&df.abstract.notna()), :].reset_index(drop=True)\n",
    "\n",
    "df[\"paper\"] = df.title + \" \" + df.abstract + \" \" + df.body_text\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "df[\"paper_token\"] = df.paper.apply(preprocess_string)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "n_papers = df.shape[0]\n",
    "df[\"paper_tag\"] = [TaggedDocument(words=df.paper_token[i],tags=[i]) for i in range(n_papers)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "model_doc2vec = Doc2Vec(df.paper_tag.values, dm=1, vector_size=200, window=5, min_count=1, workers=8, epochs=5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "def TagInText(text, tag_list):\n",
    "    tagIn = False\n",
    "    for tag in tag_list:\n",
    "        match = re.match(\".* {}.*|.* {}.*|.* {}.*\".format(tag.lower(), tag.upper(), tag.capitalize()), text)\n",
    "        if match:\n",
    "            tagIn=True\n",
    "    \n",
    "    return tagIn"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "model_word2vec = Word2Vec(sentences=df.paper_token, size=100,window=5,min_count=1, seed=1,  sg =0)\n",
    "\n",
    "#model_word2vec.save(\"../data/embedding/final/word2vec_cbow.model\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def SimilarWords(tags):\n",
    "    tag_list = preprocess_string(tags)\n",
    "    similar_words_list = []\n",
    "    for tag in tag_list:\n",
    "        #try is important here to avoid errors if a word is not in the vocabulary\n",
    "        try:\n",
    "            similar_words = model_word2vec.wv.most_similar(tag.lower())\n",
    "            similar_words = [word_sim[0] for word_sim in similar_words]\n",
    "            similar_words_list = similar_words_list + similar_words\n",
    "        except:\n",
    "            pass\n",
    "    \n",
    "    return list(set(similar_words_list)) #using set allows to filter out duplicates\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['januari',\n",
       " 'risk',\n",
       " 'spread',\n",
       " 'februari',\n",
       " 'china',\n",
       " 'provinc',\n",
       " 'decemb',\n",
       " 'citi',\n",
       " 'outbreak',\n",
       " 'outsid',\n",
       " 'mainland',\n",
       " 'wuhan']"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#example of words closed to China\n",
    "tags = \"China Wuhan\"\n",
    "SimilarWords(tags)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "\n",
    "def SearchIndex(search, data, column, model_doc2vec, pubdate_before=None, pubdate_after=None, pubdate_in=None, tags=None, topn=10, seed=0):\n",
    "    \n",
    "    \"\"\"\n",
    "    Take a search string and return list of line indexes of n closest papers\n",
    "    \n",
    "    search: string containing the query\n",
    "    data: papers dataframe in the right format\n",
    "    column: column in \"data\" containing papers texts\n",
    "    model_doc2vec: a Doc2Vec model trained on papers from \"data\"\n",
    "    pubdate_before: to get papers which publication dates are before \"pubdate_before\"; format date : year-month-day\n",
    "    pubdate_after: to get papers which publication dates are after pubdate_before. format date : year-month-day\n",
    "    pubdate_in: to get papers which publication dates are between date1 and date2. format date : year1-month1-day1 year2-month2-day2\n",
    "    tags: string with tag words\n",
    "    topn: maximum number of results to be returned,\n",
    "    seed: an integer for random seed; useful for the doc2vec model. \n",
    "    \"\"\"\n",
    "    \n",
    "    #filtering according to date\n",
    "    if pubdate_before:\n",
    "        data = data[data.publish_time<pubdate_before]\n",
    "    if pubdate_after:\n",
    "        data = data[data.publish_time>pubdate_after]\n",
    "    if pubdate_in:\n",
    "        date1 = pubdate_in.split()[0]\n",
    "        date2 = pubdate_in.split()[1]\n",
    "        data = data[(data.publish_time>=date1)&(data.publish_time<=date2)]\n",
    "        \n",
    "    n = data.shape[0]\n",
    "    model_doc2vec.random.seed(seed)\n",
    "    search = preprocess_string(search)\n",
    "    search_vec = model_doc2vec.infer_vector(search)\n",
    "    similar_text_index = model_doc2vec.docvecs.most_similar([search_vec], topn = n)\n",
    "    similar_text_index = [index_sim[0] for index_sim in similar_text_index  if index_sim[0] in data.index]\n",
    "    \n",
    "    if tags: \n",
    "        tag_list = preprocess_string(tags)\n",
    "        similar_words = SimilarWords(tags)\n",
    "        tag_list = tag_list + similar_words\n",
    "        \n",
    "        filter_papers_index = []\n",
    "\n",
    "        #finding articles containing tags\n",
    "        tagged = data[column].apply(lambda paper: TagInText(paper,tag_list))\n",
    "        tagged_papers_index = data[tagged].index\n",
    "        \n",
    "        similar_text_index = [index for index in similar_text_index if index in tagged_papers_index]\n",
    "\n",
    "    \n",
    "    similar_text_index = similar_text_index[:topn]\n",
    "\n",
    "    \n",
    "    \n",
    "    return similar_text_index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[29, 6, 37, 24, 27]"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "search = \"What is covid-19 risks factors?\"\n",
    "tags = \"risk covid China\"\n",
    "\n",
    "select_indexes = SearchIndex(search=search, data=df, column=\"paper\", model_doc2vec=model_doc2vec, pubdate_before=\"2020-02-01\", topn=5)\n",
    "select_indexes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paper_index</th>\n",
       "      <th>sentence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>29</td>\n",
       "      <td>Protein-protein interactions are central to al...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>29</td>\n",
       "      <td>Consequently the effects of changes in protein...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>29</td>\n",
       "      <td>Methods which measure, predict or optimise the...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>29</td>\n",
       "      <td>[3] ), designing combinatorial protein librari...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>29</td>\n",
       "      <td>[24] ), uncovering the effects of pathological...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  paper_index                                           sentence\n",
       "0          29  Protein-protein interactions are central to al...\n",
       "1          29  Consequently the effects of changes in protein...\n",
       "2          29  Methods which measure, predict or optimise the...\n",
       "3          29  [3] ), designing combinatorial protein librari...\n",
       "4          29  [24] ), uncovering the effects of pathological..."
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sentences_df = pd.DataFrame(columns=[\"paper_index\", \"sentence\"])\n",
    "\n",
    "for paper_index in select_indexes:\n",
    "\n",
    "    sentences_list = list(get_sentences(df.loc[paper_index, \"body_text\"]))\n",
    "    n_sent = len(sentences_list)\n",
    "    new_sent_df = pd.DataFrame(list(zip([paper_index]*n_sent, sentences_list)) , columns=[\"paper_index\", \"sentence\"])\n",
    "    sentences_df = sentences_df.append(new_sent_df)\n",
    "\n",
    "sentences_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "#filtering out non relevant sentences \n",
    "\n",
    "pattern = \".*doi.*|.*http.*|.*copyright.*|.*author.*|.*license.*|.*without permission.*|.*CC-BY-NC-ND.*|.*author.*|.*funder.*|.*medrxiv.*|.*fig.*|.*all rights.*\"\n",
    "sentences_df[\"filtered\"] = sentences_df[\"sentence\"].apply(lambda sent: True if re.match(pattern, sent.lower()) else False)\n",
    "\n",
    "sentences_df = sentences_df.loc[~sentences_df.filtered,:]\n",
    "sentences_df = sentences_df.reset_index(drop=True)\n",
    "\n",
    "sentences_df.head()\n",
    "\n",
    "sentences_df[\"sentence_token\"] = sentences_df.sentence.apply(preprocess_string)\n",
    "\n",
    "#### Filtering out sentences with less than 5 tokens\n",
    "\n",
    "sentences_df.sentence_token = sentences_df.sentence_token.apply(lambda sent: sent if len(sent)>5 else np.nan)\n",
    "sentences_df = sentences_df.dropna().reset_index(drop=True)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "def RelevantSent(search, papers_indexes, topn_sent=5, tags=None):\n",
    "    \"\"\"\n",
    "    This is a function that look for more relevant sentences in a list of papers given a search.\n",
    "    It returns a dictionary which keys are papers indexes and values are list of top n relevant sentences\n",
    "    \"\"\"\n",
    "    model_doc2vec.random.seed(1)\n",
    "    search = preprocess_string(search)\n",
    "    search_vec = model_doc2vec.infer_vector(search)\n",
    "\n",
    "    sentences_df[\"sent_vec\"] = sentences_df.sentence_token.apply(model_doc2vec.infer_vector).values\n",
    "    sentences_df[\"cos_similarity\"] = sentences_df.sent_vec.apply(lambda sent_vec: cosine_similarity(search_vec.reshape(-1,1), sent_vec.reshape(-1,1))[0][0])\n",
    "\n",
    "    topn_sent_dict = {}\n",
    "    \n",
    "    if tags:\n",
    "        tags = preprocess_string(tags) + SimilarWords(tags)\n",
    "            \n",
    "    for paper_index in papers_indexes:\n",
    "        sent_paper_order = sentences_df.loc[sentences_df.paper_index==paper_index, :]\n",
    "        sent_paper_order = sent_paper_order.sort_values(by=\"cos_similarity\", axis=0, ascending=False).reset_index(drop=True)\n",
    "\n",
    "        \n",
    "        #keeping only sentences with at least one tag word\n",
    "        sent_list_tag = []\n",
    "        if tags:\n",
    "            sent_list = list(sent_paper_order.loc[:1000, \"sentence\"])\n",
    "            for sent in sent_list:\n",
    "                tagIn = TagInText(sent, tags)\n",
    "                if tagIn is True:\n",
    "                    sent_list_tag.append(sent)\n",
    "\n",
    "        else:\n",
    "            sent_list_tag = list(sent_paper_order.loc[:topn_sent-1, \"sentence\"])\n",
    "            \n",
    "        topn_sent_dict[paper_index] = sent_list_tag[:topn_sent]\n",
    "        \n",
    "    return topn_sent_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['The range of ∆∆G int values rarely fall outside of the -5 to 3 kcal.mol −1 range.',\n",
       " 'SKEMPI has also been used to study human disease [56] , [16] , [55] , assessing the role of dynamics on binding [69] , exploring the conservation of binding regions [28] , evaluating experimental affinity measurement methods [22] , as well serving as a data source for models which predict dissociation rate changes upon mutation [1] , pathological mutations [23] , hotspot residues (e.g.',\n",
       " 'Consequently the effects of changes in protein sequence on the structure, thermodynamics and kinetics of protein-protein interactions has wide implications for constraining the permissible substitutions that accrue over the course of evolution, and for understanding the molecular etiology of disease.']"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "top_sent = RelevantSent(search, select_indexes, topn_sent=10, tags=tags)\n",
    "paper_index = select_indexes[0]\n",
    "top_sent[paper_index]\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "It is possible to not get any relevant sentence cause we only keep the ones that contain at least one tag or one similar word. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Presenting results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def SearchDisplayPaper(search, model_doc2vec, pubdate_before=None, pubdate_after=None, pubdate_in=None, topn=10, topn_sent=10, tags=None):\n",
    "    \"\"\"\n",
    "    Take a request and return most relevant papers, with their titles, authors, abstracts and most relevant sentences\n",
    "    \n",
    "    search: string containing the query\n",
    "    model_doc2vec: a Doc2Vec model trained on papers from \"data\"\n",
    "    pubdate_before: to get papers which publication dates are before \"pubdate_before\"; format date : year-month-day\n",
    "    pubdate_after: to get papers which publication dates are after pubdate_before. format date : year-month-day\n",
    "    pubdate_in: to get papers which publication dates are between date1 and date2. format date : year1-month1-day1 year2-month2-day2\n",
    "    tags: string with tag words\n",
    "    topn: maximum number of results to be returned,\n",
    "    seed: an integer for random seed; useful for the doc2vec model. \n",
    "    \"\"\"\n",
    "    \n",
    "    select_indexes = SearchIndex(search=search, data=df, column=\"paper\", model_doc2vec=model_doc2vec, pubdate_before=pubdate_before, pubdate_after=pubdate_after, pubdate_in=pubdate_in, tags=tags, topn=topn)\n",
    "    top_sent = RelevantSent(search, select_indexes, topn_sent=topn_sent, tags=tags)\n",
    "\n",
    "    if tags:\n",
    "        tag_list = tags.split()\n",
    "        similar_words = SimilarWords(tags)\n",
    "            \n",
    "    summary = \"\"\n",
    "    \n",
    "    for i,index in enumerate(select_indexes):\n",
    "        title, authors, date, url, abstract = list(df.loc[index, [\"title\", \"authors\", \"publish_time\", \"url\", \"abstract\"]])\n",
    "        \n",
    "        sentences = top_sent[index]\n",
    "        top_sentences = [\n",
    "                \"\"\"\n",
    "                <p>&nbsp;</p>\n",
    "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence {nb}:</h3>\n",
    "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">{sent}</span></span></h4>\n",
    "                \"\"\".format(nb=i+1, sent=sent) for i, sent in enumerate(sentences) \n",
    "                ]\n",
    "\n",
    "        top_sentences = \" \".join(top_sentences)\n",
    "        \n",
    "        \n",
    "        if tags:\n",
    "            \n",
    "            #highlighting tags in magenta\n",
    "            for tag in similar_words:\n",
    "                abstract = re.sub(\" {}| {}| {}\".format(tag.lower(), tag.upper(), tag.capitalize()), \"\"\"<span style=\"color: #ff00ff;\"> {}</span>\"\"\".format(tag.capitalize()), abstract)\n",
    "                top_sentences = re.sub(\" {}| {}| {}\".format(tag.lower(), tag.upper(), tag.capitalize()), \"\"\"<span style=\"color: #ff00ff;\"> {}</span>\"\"\".format(tag.capitalize()), top_sentences)\n",
    "            #highlighting tags in red\n",
    "            \n",
    "            for tag in tag_list:\n",
    "                abstract = re.sub(\" {}| {}| {}\".format(tag.lower(), tag.upper(), tag.capitalize()), \"\"\"<span style=\"color: #ff0000;\"> {}</span>\"\"\".format(tag.capitalize()), abstract)\n",
    "                top_sentences = re.sub(\" {}| {}| {}\".format(tag.lower(), tag.upper(), tag.capitalize()), \"\"\"<span style=\"color: #ff0000;\"> {}</span>\"\"\".format(tag.capitalize()), top_sentences)\n",
    "             \n",
    "            \n",
    "        summary = summary + \"\"\"\n",
    "        <p>&nbsp;</p>\n",
    "        <p>&nbsp;</p>\n",
    "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper {nb}.</span></h1>\n",
    "        <p>&nbsp;</p>\n",
    "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> {title} </span></span></h1>\n",
    "        <p>&nbsp;</p>\n",
    "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: {authors} </span></span></strong></p>\n",
    "        <p><span style=\"color: #003366;\"><strong>Publication date: {date} </strong></span></p>\n",
    "        <p><a href={url} ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
    "        <p>&nbsp;</p>\n",
    "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
    "        <p>&nbsp;</p>\n",
    "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> {abstract} </span></h4>\n",
    "        \n",
    "        \"\"\".format(nb=i+1, title=title, authors=authors, date=date, url=url, abstract=abstract)\n",
    "        \n",
    "        \n",
    "        summary = summary + top_sentences\n",
    "        \n",
    "    summary = HTML(summary)\n",
    "    \n",
    "    return display(summary)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <p>&nbsp;</p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 1.</span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation </span></span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Justina Jankauskaitė; Brian Jiménez-García; Justas Dapkūnas; Juan Fernández-Recio; Iain H. Moal </span></span></strong></p>\n",
       "        <p><span style=\"color: #003366;\"><strong>Publication date: 2018-06-07 </strong></span></p>\n",
       "        <p><a href=https://doi.org/10.1101/341735 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
       "        <p>&nbsp;</p>\n",
       "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> AbstractMotivationUnderstanding the relationship between the sequence, structure, binding energy, binding kinetics and binding thermodynamics of protein-protein interactions is crucial to understanding cellular signaling, the assembly and regulation of molecular complexes, the mechanisms through which mutations lead to<span style=\"color: #ff00ff;\"> Diseas</span>e, and protein engineering.ResultsWe present SKEMPI 2.0, a major update to our database of binding free energy changes upon mutation for structurally resolved protein-protein interactions. This version now contains manually curated binding data for 7085 mutations, an increase of 133%, including changes in kinetics for 1844 mutations, enthalpy and entropy changes for 443 mutations, and 440 mutations which abolish detectable binding.AvailabilityThe database is available at https://life.bsc.es/pid/skempi2/ </span></h4>\n",
       "        \n",
       "        \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The range of ∆∆G int values rarely fall<span style=\"color: #ff00ff;\"> Outsid</span>e of the -5 to 3 kcal.mol −1 range.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">SKEMPI has also been used to study human<span style=\"color: #ff00ff;\"> Diseas</span>e [56] , [16] , [55] , assessing the role of dynamics on binding [69] , exploring the conservation of binding regions [28] , evaluating experimental affinity measurement methods [22] , as well serving as a data source for models which predict dissociation rate changes upon mutation [1] , pathological mutations [23] , hotspot residues (e.g.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Consequently the effects of changes in protein sequence on the structure, thermodynamics and kinetics of protein-protein interactions has wide implications for constraining the permissible substitutions that accrue over the course of evolution, and for understanding the molecular etiology of<span style=\"color: #ff00ff;\"> Diseas</span>e.</span></span></h4>\n",
       "                \n",
       "        <p>&nbsp;</p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 2.</span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> Viruses are a dominant driver of protein adaptation in mammals </span></span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: David Enard; Le Cai; Carina Gwenapp; Dmitri A Petrov </span></span></strong></p>\n",
       "        <p><span style=\"color: #003366;\"><strong>Publication date: 2015-10-18 </strong></span></p>\n",
       "        <p><a href=https://doi.org/10.1101/029397 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
       "        <p>&nbsp;</p>\n",
       "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> Viruses interact with hundreds to thousands of proteins in mammals, yet adaptation against viruses has only been studied in a few proteins specialized in antiviral defense. Whether adaptation to viruses typically involves only specialized antiviral proteins or affects a broad array of proteins is unknown. Here, we analyze adaptation in ~1,300 virus-interacting proteins manually curated from a set of 9,900 proteins conserved across mammals. We show that viruses (i) use the more evolutionarily constrained proteins from the cellular functions they hijack and that (ii) despite this high constraint, virus-interacting proteins account for a high proportion of all protein adaptation in humans and other mammals. Adaptation is elevated in virus-interacting proteins across all functional categories, including both immune and non-immune functions. Our results demonstrate that viruses are one of the most dominant drivers of evolutionary change across mammalian and human proteomes. </span></h4>\n",
       "        \n",
       "        \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Most of them (95%) correspond to an interaction 17 between a human protein and a virus<span style=\"color: #ff00ff;\"> Infecti</span>ng humans (Table S1 ).</span></span></h4>\n",
       "                \n",
       "        <p>&nbsp;</p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 3.</span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> p53 is not necessary for DUX4 pathology </span></span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Darko Bosnakovski; Erik A. Toso; Olivia O. Recht; Anja Cucak; Abhinav K Jain; Michelle C. Barton; Michael Kyba </span></span></strong></p>\n",
       "        <p><span style=\"color: #003366;\"><strong>Publication date: 2017-03-19 </strong></span></p>\n",
       "        <p><a href=https://doi.org/10.1101/118315 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
       "        <p>&nbsp;</p>\n",
       "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> AbstractsFSHD is a genetically dominant myopathy caused by mutations that cause expression of the normally silent DUX4 gene. This transcription factor has been shown to interfere with myogenesis when misexpressed at very low levels in myoblasts, and to cause cell death when overexpressed at high levels. A previous report using adeno-associated virus to deliver high levels of DUX4 to mouse skeletal muscle demonstrated severe pathology that was suppressed on a p53 knockout background, implying that DUX4 acted through the p53 pathway. Here, we investigate the p53-dependence of DUX4 using both in vitro cellular and the transgenic iDUX4[2.7] mouse models. We find that inhibiting p53 has no effect on the cytoxicity of DUX4 in vitro. When crossed onto the p53 null background, we find no suppression of the male-specific lethality or skin phenotypes of the DUX4 transgene, and find that primary myoblasts from this mouse are still killed by DUX4 expression. These data challenge the notion that the p53 pathway is central to the pathogenicity of DUX4.Summary StatementDUX4 is thought to mediate cytopathology through p53. Here, DUX4 is shown to kill primary myoblasts and promote pathological phenotypes in the iDUX4[2.7] mouse model on the p53-null background, calling into question this notion. </span></h4>\n",
       "        \n",
       "        \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">It is also theoretically possible that some aspect of the AAV experimental system combines with the effect of DUX4 to make the pathology more severe than it would otherwise have been, and the p53 effect ascribed to DUX4 was in fact downstream of the cellular response to AAV<span style=\"color: #ff00ff;\"> Infecti</span>on.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">2007 ) leads to DUX4 expression and explains why<span style=\"color: #ff00ff;\"> Diseas</span>e is associated only with contractions of the D4Z4 repeats on chr4, and only on specific 4qter alleles, so called permissive alleles, which harbor the poly(A) signal (Lemmers et al.</span></span></h4>\n",
       "                \n",
       "        <p>&nbsp;</p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 4.</span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> A Combined Evidence Approach to Prioritize Nipah Virus Inhibitors </span></span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Nishi Kumari; Ayush Upadhyay; Kishan Kalia; Rakesh Kumar; Kanika Tuteja; Priyanka Rani Paul; Eugenia Covernton; Tina Sharma; Vinod Scaria; Anshu Bhardwaj </span></span></strong></p>\n",
       "        <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-12 </strong></span></p>\n",
       "        <p><a href=https://doi.org/10.1101/2020.03.12.977918 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
       "        <p>&nbsp;</p>\n",
       "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> AbstractBackgroundNipah Virus (NiV) came into limelight recently due to an<span style=\"color: #ff00ff;\"> Outbreak</span> in Kerala, India. NiV causes severe<span style=\"color: #ff00ff;\"> Diseas</span>e and death in people with over 75% case fatality rate. It is a public health concern and has the potential to become a global pandemic. Lack of treatment has forced the containment methods to be restricted to isolation and surveillance. WHO’s ‘R&D Blueprint list of priority<span style=\"color: #ff00ff;\"> Diseas</span>es’ (2018) indicates that there is an urgent need for accelerated research & development for addressing NiV.Materials & MethodsIn the quest for druglike NiV inhibitors (NVIs) a thorough literature search followed by systematic data curation was conducted. Rigorous data analysis was done with curated NVIs for prioritizing druglike compounds. For the same, more than 1800 descriptors of NVIs were computed and comparative analysis was performed with the FDA approved small molecules and antivirals. These compounds were further evaluated through PAINS filter to study their toxicity profile. Simultaneously, compounds were also prioritized based on the robustness of the assays through which they were identified.ResultsOur efforts lead to the creation of a well-curated structured knowledgebase of 182 NVIs with 98 small molecule inhibitors. The reported IC50/EC50 values for some of these inhibitors are in the nanomolar range – as low as 0.47 nM. In order to prioritize these inhibitors, we performed several tests and applied filters to identify drug-like non-toxic compounds. Of 98, a few compounds passed DruLito & PAINS filters exhibiting drug-like properties and were also prioritized in an independent screen based only the assay robustness. The NVIs have diverse structural features and offer a wide spectrum of ways in which they can be developed further as druglike molecules.ConclusionWe report a knowledgebase for furthering the development of NVIs. The platform has a diverse set of 98 NVIs of which a few have been prioritized based on a combined evidence strategy. The platform has the provision to submit new inhibitors as and when reported by the community for further enhancement of NiV inhibitor landscape. </span></h4>\n",
       "        \n",
       "        \n",
       "        <p>&nbsp;</p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 5.</span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID-19) protease structure </span></span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Stephen A. Wells </span></span></strong></p>\n",
       "        <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-12 </strong></span></p>\n",
       "        <p><a href=https://doi.org/10.1101/2020.03.10.986190 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
       "        <p>&nbsp;</p>\n",
       "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> AbstractThe rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the<span style=\"color: #ff00ff;\"> Infecti</span>ous agent of the<span style=\"color: #ff0000;\"> Covid</span>-19 respiratory<span style=\"color: #ff00ff;\"> Diseas</span>e, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions in vivo which directly affect the geometry of a known inhibitor binding site and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral therapies for the treatment of<span style=\"color: #ff0000;\"> Covid</span>-19. </span></h4>\n",
       "        \n",
       "        \n",
       "        <p>&nbsp;</p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 6.</span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> Artesunate interacts with Vitamin D receptor to reverse mouse model of sepsis-induced immunosuppression via enhancing autophagy </span></span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Shenglan Shang; Jiaqi Wu; Xiaoli Li; Xin Liu; Pan Li; Chunli Zheng; Yonghua Wang; Songqing Liu; Jiang Zheng; Hong Zhou </span></span></strong></p>\n",
       "        <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-27 </strong></span></p>\n",
       "        <p><a href=https://doi.org/10.1101/2020.02.26.966143 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
       "        <p>&nbsp;</p>\n",
       "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> AbstractBackground and PurposeImmunosuppression is the predominant cause of mortality for sepsis due to failure to eradicate invading pathogens. Unfortunately, no effective and specific drugs capable of reversing immunosuppression are available for clinical use. Increasing evidence implicates vitamin D receptor (VDR) involved in sepsis-induced immunosuppression. Herein, artesunate (AS) was discovered to reverse sepsis-induced immunosuppression and its molecular mechanism is investigated.Experimental ApproachEffect of artesunate on sepsis-induced immunosuppression was investigated in mice and in vitro. VDR was predicted to be an interacted candidate of AS by bioinformatics predict, then identified using PCR and immunoblotting. VDR, ATG16L1 and NF-κB p65 were modified to investigate the alteration of AS’s effect on pro-inflammatory cytokines release, bacteria clearance and autophagy activities in sepsis-induced immunosuppression.Key ResultsAS significantly reduced the mortality of cecal ligation and puncture (CLP)-induced sepsis immunosuppression mice challenged with Pseudomonas Aeruginosa, and enhanced proinflammatory cytokines release and bacterial clearance to reverse sepsis-induced immunosuppression in vivo and in vitro. Mechanically, AS interacted with VDR thereby inhibited the nuclear translocation of VDR, then influencing ATG16L1 transcription and subsequent autophagy activity. In addition, AS inhibited physical interaction between VDR and NF-κB p65 in LPS tolerance macrophages, then promoted nuclear translocation of NF-κB p65, which activated the transcription of NF-κB p65 target genes such as pro-inflammatory cytokines.Conclusion and ImplicationsOur findings provide an evidence that AS interacted with VDR to reverse sepsis-induced immunosuppression in an autophagy and NF-κB dependent way, highlighting a novel approach for sepsis treatment and drug repurposing of AS in the future. </span></h4>\n",
       "        \n",
       "        \n",
       "        <p>&nbsp;</p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 7.</span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> Mono-ADP-ribosylation by ARTD10 restricts Chikungunya virus replication by interfering with the proteolytic activity of nsP2 </span></span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Sarah Krieg; Fabian Pott; Laura Eckei; Maud Verheirstraeten; Mareike Bütepage; Barbara Lippok; Christine Goffinet; Bernhard Lüscher; Patricia Verheugd </span></span></strong></p>\n",
       "        <p><span style=\"color: #003366;\"><strong>Publication date: 2020-01-08 </strong></span></p>\n",
       "        <p><a href=https://doi.org/10.1101/2020.01.07.896977 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
       "        <p>&nbsp;</p>\n",
       "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> AbstractA subset of intracellular mono-ADP-ribosyltransferases diphtheria toxin-like (ARTDs, aka mono-PARPs) is induced by type I interferons. Some of these mono-ARTDs feature antiviral activity while certain RNA viruses, including Chikungunya virus (CHIKV), encode mono-ADP-ribosylhydrolases, suggesting a role for mono-ADP-ribosylation (MARylation) in host-virus conflicts. CHIKV expresses four non-structural proteins (nsP1-nsP4), with nsP3 containing a macrodomain that hydrolyzes and thereby reverses protein MARylation in vitro and in cells. This de-MARylation activity is essential as hydrolase inactivating mutations result in replication defective virus. However, the substrates of MARylation during CHIKV<span style=\"color: #ff00ff;\"> Infecti</span>on are unknown and thus it is unclear how the macrodomain contributes to virus replication and how mono-ARTD-dependent MARylation confers antiviral immunity. We identified ARTD10 and ARTD12 as restriction factors for CHIKV replication in a catalytic activity-dependent manner. CHIKV replication requires processing of the non-structural polyprotein nsP1-4 by the nsP2-encoded protease and the assembly of the four individual nsPs into a functional replication complex. Expression of ARTD10 and ARTD12 resulted in a reduction of processed nsPs. Similarly, MAR hydrolase inactive CHIKV replicon mutants revealed a decrease in processed nsPs, comparable to an nsP2 protease defective mutant. This suggested that the macrodomain contributes to nsP2 protease activity. In support, a hydrolase-deficient virus was complemented by a protease-deficient virus. We hypothesized that MARylation regulates the proteolytic function of nsP2. Indeed, we found that nsP2 is MARylated by ARTD10. This inhibited nsP2 protease activity, thereby preventing polyprotein processing and consequently virus replication. This inhibition was antagonized by the MAR hydrolase activity of nsP3. Together, our findings provide a mechanistic explanation for the need of the viral MAR hydrolase for efficient replication of CHIKV.Author SummaryInfectious<span style=\"color: #ff00ff;\"> Diseas</span>es still pose major health threats. Especially fast evolving viruses find ever new strategies to manipulate the immune response. With climate warming and increased human mobility vector-borne pathogens like Chikungunya virus (CHIKV)<span style=\"color: #ff00ff;\"> Spread</span> and cause world-wide<span style=\"color: #ff00ff;\"> Epidemi</span>cs. Beyond the acute phase, CHIKV<span style=\"color: #ff00ff;\"> Patient</span>s regularly suffer from chronic rheumatism. This entails a decline in life quality and an economic burden. To date no drugs are approved and the mode of pathogenesis remains elusive. Here we describe a mechanistic function of the CHIKV nsP3 macrodomain. We found that the viral nsP2 is mono-ADP-ribosylated interfering with its auto-proteolytic function. The nsP3 macrodomain removes this modification and restores the protease activity that is essential for replication. Because macrodomains are highly conserved they might represent broad antiviral targets. </span></h4>\n",
       "        \n",
       "        \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Taken together, we identified CHIKV nsP2 as a new substrate for MARylation 290 in vitro and in cells in context of a viral<span style=\"color: #ff00ff;\"> Infecti</span>on.</span></span></h4>\n",
       "                \n",
       "        <p>&nbsp;</p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 8.</span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> Comparisons of a Novel Air Sampling Filter Material, Wash Buffers and Extraction Methods in the Detection and Quantification of Influenza Virus </span></span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Terrance A. Thedell; Corey L. Boles; David M. Cwiertny; Jiajie Qian; Grant D. Brown; Matthew W. Nonnenmann </span></span></strong></p>\n",
       "        <p><span style=\"color: #003366;\"><strong>Publication date: 2018-10-11 </strong></span></p>\n",
       "        <p><a href=https://doi.org/10.1101/441154 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
       "        <p>&nbsp;</p>\n",
       "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> AbstractQuantification of aerosolized influenza virus is used for determining inhalation exposure. Several bioaerosol samplers and analytical methods have been used; however, the detection and quantification of influenza virus among aerosol samples remains challenging. Therefore, improved viral aerosol measurement methods are needed. This study evaluated influenza virus recovery among three filter types polytetrafluoroethylene, polyvinylchloride and polystyrene. Polytetrafluoroethylene, polyvinylchloride are fabricated filter materials and commonly used in the scientific literature to sample for viral aerosols. A novel, electrospun polystyrene filter material may improve viral aerosol recovery during filter-based air sampling. The filter materials were compared across the following conditions: treated with or without air, filter wash buffer (HBSS or PBS), and viral RNA extraction method (QIAamp Viral RNA Mini Kit or Trizol). Twenty trials were completed in a chamber and samples were analyzed using RT-qPCR. Viral recovery was significantly different (p-value < .0001) by filter type. Polystyrene filter use resulted in recovery of the most viral RNA. Air sampling did not affect the recovery of viral RNA from the filter materials (p-values > 0.05). Viral RNA concentrations were significantly different across extraction methods for all comparisons (p-values < 0.05). Our results demonstrated that the novel polystyrene filter material resulted in the highest concentration of extracted RNA compared to the commonly used polytetrafluoroethylene and polyvinylchloride, which we speculate may be related to the chemical composition of the filter material (e.g., polystyrene is an aromatic hydrocarbon whereas polytetrafluoroethylene and polyvinylchloride contain more polar, and thus potentially reactive, carbon-halogen bonds). Air sampling did not have an effect on viral RNA recovery. Using Hanks Balanced Salt Solution with QIAamp Viral RNA Mini Kit, and Phosphate-buffered saline with the Trizol extraction, resulted in the most viral RNA recovery. </span></h4>\n",
       "        \n",
       "        \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Although the optimal 311 comfort level for humans is 40-60% RH 14 , and previous studies have found that higher RHs (i.e., 312 50-56%) have no effect on viral<span style=\"color: #ff00ff;\"> Infecti</span>vity.</span></span></h4>\n",
       "                \n",
       "        <p>&nbsp;</p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 9.</span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> TALC: Transcription Aware Long Read Correction </span></span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Lucile Broseus; Aubin Thomas; Andrew J. Oldfield; Dany Severac; Emeric Dubois; William Ritchie </span></span></strong></p>\n",
       "        <p><span style=\"color: #003366;\"><strong>Publication date: 2020-01-11 </strong></span></p>\n",
       "        <p><a href=https://doi.org/10.1101/2020.01.10.901728 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
       "        <p>&nbsp;</p>\n",
       "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> ABSTRACTMotivationLong-read sequencing technologies are invaluable for determining complex RNA transcript architectures but are error-prone. Numerous “hybrid correction” algorithms have been developed for genomic data that correct long reads by exploiting the accuracy and depth of short reads sequenced from the same sample. These algorithms are not suited for correcting more complex transcriptome sequencing data.ResultsWe have created a novel algorithm called TALC (Transcription Aware Long Read Correction) which models changes in RNA expression and isoform representation in a weighted De-Bruijn graph to correct long reads from transcriptome studies. We show that transcription aware correction by TALC improves the accuracy of the whole spectrum of downstream RNA-seq applications and is thus necessary for transcriptome analyses that use long read technology.Availability and ImplementationTALC is implemented in C++ and available at https://gitlab.igh.cnrs.fr/lbroseus/TALC.Contactwilliam.ritchie@igh.cnrs.fr </span></h4>\n",
       "        \n",
       "        \n",
       "        <p>&nbsp;</p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 10.</span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <h1><span style=\"color: #800000; background-color: #999999;\">Title:<span style=\"background-color: #ffffff;\"> In trans variant calling reveals enrichment for compound heterozygous variants in genes involved in neuronal development and growth. </span></span></h1>\n",
       "        <p>&nbsp;</p>\n",
       "        <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Allison J Cox; Fillan Grady; Gabriel Velez; Vinit B Mahajan; Polly J Ferguson; Andrew Kitchen; Benjamin W Darbro; Alexander G Bassuk </span></span></strong></p>\n",
       "        <p><span style=\"color: #003366;\"><strong>Publication date: 2018-12-13 </strong></span></p>\n",
       "        <p><a href=https://doi.org/10.1101/496133 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "        <p>&nbsp;</p>\n",
       "        <h2><span style=\"background-color: #999999;\">Abstract: </span></h2>\n",
       "        <p>&nbsp;</p>\n",
       "        <h4  style=\"text-align: justify;\"><span style=\"background-color: #ffffff;\"> Compound heterozygotes occur when different mutations at the same locus on both maternal and paternal chromosomes produce a recessive trait. Here we present the tool VarCount for the quantification of mutations at the individual level. We used VarCount to characterize compound heterozygous coding variants in<span style=\"color: #ff00ff;\"> Patient</span>s with epileptic encephalopathy and in the 1000 genomes participants. The Epi4k data contains variants identified by whole exome sequencing in<span style=\"color: #ff00ff;\"> Patient</span>s with either Lennox-Gastaut Syndrome (LGS) or Infantile Spasms (IS), as well as their parents. We queried the Epi4k dataset (264 trios) and the phased 1000 genomes data (2504 participants) for recessive variants. To assess enrichment, transcript counts were compared between the Epi4k and 1000 genomes participants using minor allele frequency (MAF) cutoffs of 0.5% and 1.0%, and including all ancestries or only probands of European ancestry. In the Epi4k participants, we found enrichment for rare, compound heterozygous mutations in six genes, including three involved in neuronal growth and development; PRTG (p=0.00086, 1% MAF, combined ancestries), TNC (p=0.0221% MAF, combined ancestries), and MACF1 (p=0.0245, 0.5% MAF, EU ancestry). Due the total number of transcripts considered in these analyses, the enrichment detected was not significant after correction for multiple testing and higher powered or prospective studies are necessary to validate the candidacy of these genes. However, PRTG, TNC, and MACF1 are potential novel recessive epilepsy genes and our results highlight that compound heterozygous mutations should be considered in sporadic epilepsy. </span></h4>\n",
       "        \n",
       "        "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "SearchDisplayPaper(search, model_doc2vec=model_doc2vec, pubdate_before=\"2020-04-01\", topn=10, topn_sent=5, tags=tags)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "class Search():\n",
    "    def __init__(self, papers_df, meta_df=None, papers_preprocessed=False, model_doc2vec=None, model_word2vec=None):\n",
    "        \"\"\"\n",
    "         papers_df: papers dataframe in the right fromat\n",
    "         meta_df: meta dataframe. if \"papers_preprocessed=True\", meta_df must be provided\n",
    "         papers_preprocessed: True if papers need to be preprocessed, False otherwise\n",
    "         model_doc2vec: A doc2vec model train on papers from \"papers_df[\"paper\"]\"\n",
    "         model_word2vec: A word3vec model preferably trained on \"papers_df[\"paper\"]\"\n",
    "        \"\"\"\n",
    "        \n",
    "        if papers_preprocessed:\n",
    "            self.papers_df = papers_df\n",
    "            self.model_doc2vec = model_doc2vec \n",
    "            self.model_word2vec = model_word2vec\n",
    "        else:   \n",
    "            self.papers_df = self.preprocessing_papers(papers_df, meta_df)\n",
    "        \n",
    "             \n",
    "        \n",
    "            \n",
    "        self.html_index = 0  #for html hidden links\n",
    "        \n",
    "    def preprocessing_papers(self, papers_df, meta_df):\n",
    "\n",
    "        print(\"Processing papers!\", time.strftime(\"%H:%M:%S\", time.localtime()))\n",
    "\n",
    "        \n",
    "        print(\"Keeping only papers with meta data!\", time.strftime(\"%H:%M:%S\", time.localtime()))\n",
    "\n",
    "        paper_in_meta = papers_df.paper_id.isin(meta_df.sha)\n",
    "        papers_df = papers_df[paper_in_meta]\n",
    "        papers_df = papers_df.sort_values(by=\"paper_id\", axis=0).reset_index(drop=True)\n",
    "\n",
    "        ##Keeping meta data of papers present in the papers dataframe and sorting it acording to sha/paper_id\n",
    "        meta_df = meta_df.loc[meta_df.sha.isin(papers_df.paper_id.values)]\n",
    "        meta_df = meta_df.sort_values(by=\"sha\", axis=0).reset_index(drop=True)\n",
    "\n",
    "        ##We can now transfert information from meta to papers dataframe\n",
    "        papers_df[\"title\"] = meta_df[\"title\"]\n",
    "        papers_df[\"abstract\"] = meta_df[\"abstract\"]\n",
    "        papers_df[\"authors\"] = meta_df[\"authors\"]\n",
    "        papers_df[\"publish_time\"] = meta_df[\"publish_time\"]\n",
    "        papers_df[\"url\"] = meta_df[\"url\"]\n",
    "        \n",
    "        print(\"Concatenation of titles, abstracts and body_texts!\", time.strftime(\"%H:%M:%S\", time.localtime()))\n",
    "        papers_df[\"paper\"] = papers_df.title + \" \" + papers_df.abstract + \" \" + papers_df.body_text\n",
    "        \n",
    "        \n",
    "        print(\"Dropping papers without title, authors names or abstract!\", time.strftime(\"%H:%M:%S\", time.localtime()))\n",
    "        papers_df = papers_df.loc[(papers_df.title.notna()&papers_df.authors.notna()&papers_df.abstract.notna()), :].reset_index(drop=True)\n",
    "        \n",
    "        print(\"Tokenization!\", time.strftime(\"%H:%M:%S\", time.localtime()))\n",
    "        papers_df[\"paper_token\"] = papers_df.paper.apply(preprocess_string)\n",
    "        \n",
    "\n",
    "        print(\"Building a Doc2Vec model with preprocessed data\", time.strftime(\"%H:%M:%S\", time.localtime()))\n",
    "        n_papers = papers_df.shape[0]\n",
    "        papers_df[\"paper_tag\"] = [TaggedDocument(words=papers_df.paper_token[i],tags=[i]) for i in range(n_papers)]\n",
    "        model_doc2vec = Doc2Vec(papers_df.paper_tag.values, dm=1, vector_size=200, window=5, min_count=1, workers=8, epochs=5)\n",
    "        self.model_doc2vec = model_doc2vec \n",
    "        \n",
    "        print(\"Building a Word2Vec model with preprocessed data\", time.strftime(\"%H:%M:%S\", time.localtime()))\n",
    "        model_word2vec = Word2Vec(sentences=papers_df.paper_token, size=100,window=5,min_count=1, seed=1,  sg =0)\n",
    "        self.model_word2vec = model_word2vec \n",
    "\n",
    "        print(\"Processing over! Do not forget saving the preprocessed data (self.papers_df), the doc2vec and word2vec models (self.model_doc2vec and self.model_word2vec) to go faster next time\")\n",
    "\n",
    "        return papers_df\n",
    "    \n",
    "    \n",
    "        \n",
    "    def preprocessing_sentences(self, select_indexes):\n",
    "        \n",
    "        \"\"\"\n",
    "        Generating a dataframe with columns [\"paper_index\", \"sentence\"]: \n",
    "        each line with a given sentence and the paper_index of the paper containing the sentence\n",
    "        \"\"\"\n",
    "        sentences_df = pd.DataFrame(columns=[\"paper_index\", \"sentence\"])\n",
    "\n",
    "        for paper_index in select_indexes:\n",
    "\n",
    "            sentences_list = list(get_sentences(self.papers_df.loc[paper_index, \"body_text\"]))\n",
    "            n_sent = len(sentences_list)\n",
    "            new_sent_df = pd.DataFrame(list(zip([paper_index]*n_sent, sentences_list)) , columns=[\"paper_index\", \"sentence\"])\n",
    "            sentences_df = sentences_df.append(new_sent_df)\n",
    "\n",
    "    \n",
    "        #filtering out non relevant sentences \n",
    "\n",
    "        pattern = \".*doi.*|.*http.*|.*copyright.*|.*author.*|.*license.*|.*without permission.*|.*CC-BY-NC-ND.*|.*author.*|.*funder.*|.*medrxiv.*|.*fig.*|.*all rights.*\"\n",
    "        sentences_df[\"filtered\"] = sentences_df[\"sentence\"].apply(lambda sent: True if re.match(pattern, sent.lower()) else False)\n",
    "\n",
    "        sentences_df = sentences_df.loc[~sentences_df.filtered,:]\n",
    "\n",
    "        sentences_df[\"sentence_token\"] = sentences_df.sentence.apply(preprocess_string)\n",
    "\n",
    "        #### Filtering out sentences with less than 5 tokens\n",
    "\n",
    "        sentences_df.sentence_token = sentences_df.sentence_token.apply(lambda sent: sent if len(sent)>5 else np.nan)\n",
    "        sentences_df = sentences_df.dropna().reset_index(drop=True)\n",
    "        \n",
    "        return sentences_df\n",
    "    \n",
    "    \n",
    "    def TagInText(self, text, tag_list):\n",
    "        tagIn = False\n",
    "        for tag in tag_list:\n",
    "            match = re.match(\".* {}.*|.* {}.*|.* {}.*\".format(tag.lower(), tag.upper(), tag.capitalize()), text)\n",
    "            if match:\n",
    "                tagIn=True\n",
    "\n",
    "        return tagIn\n",
    "    \n",
    "    def SimilarWords(self,tags):\n",
    "        \"\"\"\n",
    "        Find 5 words similar to each tag. Return a list of words filtered out of duplicates.\n",
    "        \"\"\"\n",
    "        tag_list = preprocess_string(tags)\n",
    "        similar_words_list = []\n",
    "        for tag in tag_list:\n",
    "            #try is important here to avoid errors if a word is not in the vocabulary\n",
    "            try:\n",
    "                similar_words = model_word2vec.wv.most_similar(tag.lower())\n",
    "                similar_words = [word_sim[0] for word_sim in similar_words]\n",
    "                similar_words_list = similar_words_list + similar_words\n",
    "            except:\n",
    "                pass\n",
    "\n",
    "        return list(set(similar_words_list)) #using set allows to filter out duplicates\n",
    "\n",
    "\n",
    "    def RelevantSent(self, search, papers_indexes, topn_sent, tags=None):\n",
    "        \"\"\"\n",
    "        This is a function that look for more relevant sentences in a list of papers given a search.\n",
    "        It returns a dictionary which keys are papers indexes and values are list of top n relevant sentences\n",
    "        \"\"\"\n",
    "        \n",
    "        sentences_df = self.preprocessing_sentences(papers_indexes)\n",
    "        model_doc2vec = self.model_doc2vec\n",
    "        \n",
    "        model_doc2vec.random.seed(1)\n",
    "        search = preprocess_string(search)\n",
    "        search_vec = model_doc2vec.infer_vector(search)\n",
    "\n",
    "        sentences_df[\"sent_vec\"] = sentences_df.sentence_token.apply(model_doc2vec.infer_vector).values\n",
    "        sentences_df[\"cos_similarity\"] = sentences_df.sent_vec.apply(lambda sent_vec: cosine_similarity(search_vec.reshape(-1,1), sent_vec.reshape(-1,1))[0][0])\n",
    "\n",
    "        topn_sent_dict = {}\n",
    "        \n",
    "        if tags:\n",
    "            tag_list = preprocess_string(tags) + self.similar_words #including similar words to tags\n",
    "            \n",
    "        for paper_index in papers_indexes:\n",
    "            sent_paper_order = sentences_df.loc[sentences_df.paper_index==paper_index, :]\n",
    "            sent_paper_order = sent_paper_order.sort_values(by=\"cos_similarity\", axis=0, ascending=False).reset_index(drop=True)\n",
    "\n",
    "\n",
    "            #keeping only sentences with at least one tag word\n",
    "            sent_list_tag = []\n",
    "            if tags:\n",
    "                sent_list = list(sent_paper_order.loc[:1000, \"sentence\"])\n",
    "                for sent in sent_list:\n",
    "                    tagIn = self.TagInText(sent, tag_list)\n",
    "                    if tagIn:\n",
    "                        sent_list_tag.append(sent)\n",
    "\n",
    "            else:\n",
    "                sent_list_tag = list(sent_paper_order.loc[:topn_sent-1, \"sentence\"])\n",
    "\n",
    "            topn_sent_dict[paper_index] = sent_list_tag[:topn_sent]\n",
    "        \n",
    "\n",
    "        return topn_sent_dict\n",
    "\n",
    "        \n",
    "    \n",
    "    def SearchIndex(self, search, pubdate_before=None, pubdate_after=None, pubdate_in=None, tags=None, topn=10, seed=0):\n",
    "\n",
    "        \"\"\"\n",
    "        Take a search string and return list of indexes of n closest papers\n",
    "\n",
    "        search: string containing the query\n",
    "        pubdate_before: to get papers which publication dates are before \"pubdate_before\"; format date : year-month-day\n",
    "        pubdate_after: to get papers which publication dates are after pubdate_before. format date : year-month-day\n",
    "        pubdate_in: to get papers which publication dates are between date1 and date2. format date : year1-month1-day1 year2-month2-day2\n",
    "        tags: string with tag words\n",
    "        topn: maximum number of results to be returned,\n",
    "        seed: an integer for random seed; useful for the doc2vec model. \n",
    "        \"\"\"\n",
    "        data = self.papers_df\n",
    "        model_doc2vec=self.model_doc2vec\n",
    "        \n",
    "        #filtering according to date\n",
    "        if pubdate_before:\n",
    "            data = data[data.publish_time<pubdate_before]\n",
    "        if pubdate_after:\n",
    "            data = data[data.publish_time>pubdate_after]\n",
    "        if pubdate_in:\n",
    "            date1 = pubdate_in.split()[0]\n",
    "            date2 = pubdate_in.split()[1]\n",
    "            data = data[(data.publish_time>=date1)&(data.publish_time<=date2)]\n",
    "            \n",
    "        #Ranging papers indexes according to their similarity to the search query\n",
    "        n = data.shape[0]\n",
    "        model_doc2vec.random.seed(seed)\n",
    "        search = preprocess_string(search)\n",
    "        search_vec = self.model_doc2vec.infer_vector(search)\n",
    "        similar_text_index = self.model_doc2vec.docvecs.most_similar([search_vec], topn = n)\n",
    "        similar_text_index = [index_sim[0] for index_sim in similar_text_index  if index_sim[0] in data.index]\n",
    "        \n",
    "        if tags:\n",
    "            tag_list = tags.split()\n",
    "            self.similar_words = self.SimilarWords(tags)\n",
    "            tag_list = tag_list + self.similar_words\n",
    "            \n",
    "            filter_papers_index = []\n",
    "\n",
    "            #finding articles containing tags\n",
    "            \n",
    "            tagged = data[\"paper\"].apply(lambda paper: self.TagInText(paper,tag_list))\n",
    "            tagged_papers_index = data[tagged].index\n",
    "\n",
    "            similar_text_index = [index for index in similar_text_index if index in tagged_papers_index]\n",
    "\n",
    "        similar_text_index = similar_text_index[:topn]\n",
    "\n",
    "        return similar_text_index\n",
    "\n",
    "\n",
    "    def SearchDisplayPaper(self, search, pubdate_before=None, pubdate_after=None, pubdate_in=None, topn=10, topn_sent=5, tags=None):\n",
    "        \"\"\"\n",
    "        Take a request and return most relevant papers, with their titles, authors, abstracts and most relevant sentences\n",
    "\n",
    "        search: string containing the request\n",
    "        pubdate_before: Publication dates should be before pubdate_before. format date : year-month-day\n",
    "        pubdate_after: Publication dates should be after pubdate_before. format date : year-month-day\n",
    "        pubdate_in: Publication dates should be between pubdate_before. format date : year1-month1-day1 format date year2-month2-day2\n",
    "        tags: string with tag words\n",
    "        topn: maximum number of results to be returned,\n",
    "        topn_sent: maximum number of sentences closely linked to search in a given paper\n",
    "        seed: an integer for random seed\n",
    "        \"\"\"\n",
    "        \n",
    "         \n",
    "        select_indexes = self.SearchIndex(search=search, pubdate_before=pubdate_before, pubdate_after=pubdate_after, pubdate_in=pubdate_in, tags=tags, topn=topn)\n",
    "        top_sent = self.RelevantSent(search, select_indexes, topn_sent=topn_sent, tags=tags)\n",
    "\n",
    "        \n",
    "    \n",
    "        summary = \"\"\"\n",
    "                <script>\n",
    "                function visibilite(thingId)\n",
    "                {\n",
    "                var targetElement;\n",
    "                targetElement = document.getElementById(thingId) ;\n",
    "                if (targetElement.style.display == \"none\")\n",
    "                {\n",
    "                targetElement.style.display = \"\" ;\n",
    "                } else {\n",
    "                targetElement.style.display = \"none\" ;\n",
    "                }\n",
    "                }\n",
    "                </script>\n",
    "                </body>\n",
    "                \"\"\"\n",
    "        if tags!=None:   \n",
    "            tag_list = preprocess_string(tags)\n",
    "            \n",
    "        paper_df = self.papers_df\n",
    "        \n",
    "        for i,index in enumerate(select_indexes):\n",
    "            title, authors, date, url, abstract, body_text = list(paper_df.loc[index, [\"title\", \"authors\", \"publish_time\", \"url\", \"abstract\", \"body_text\"]])\n",
    "            sentences = top_sent[index]\n",
    "            #adding relevant sentences from the paper\n",
    "            top_sentences = [\n",
    "                \"\"\"\n",
    "                <p>&nbsp;</p>\n",
    "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence {nb}:</h3>\n",
    "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">{sent}</span></span></h4>\n",
    "                \"\"\".format(nb=i+1, sent=sent) for i, sent in enumerate(sentences) \n",
    "                ]\n",
    "\n",
    "            top_sentences = \" \".join(top_sentences)\n",
    "            \n",
    "            \n",
    "            if tags!=None:\n",
    "                \n",
    "                #highlighting similar words to tags in magenta color\n",
    "                for tag in self.similar_words:\n",
    "                    abstract = re.sub(\" {}| {}| {}\".format(tag.lower(), tag.upper(), tag.capitalize()), \"\"\"<span style=\"color: #ff00ff;\"> {}</span>\"\"\".format(tag.capitalize()), abstract)\n",
    "                    top_sentences = re.sub(\" {}| {}| {}\".format(tag.lower(), tag.upper(), tag.capitalize()), \"\"\"<span style=\"color: #ff00ff;\"> {}</span>\"\"\".format(tag.capitalize()), top_sentences)\n",
    "                    body_text = re.sub(\" {}| {}| {}\".format(tag.lower(), tag.upper(), tag.capitalize()), \"\"\"<span style=\"color: #ff00ff;\"> {}</span>\"\"\".format(tag.capitalize()), body_text)\n",
    "            \n",
    "                #highlighting tags in red color\n",
    "                for tag in tag_list:\n",
    "                    abstract = re.sub(\" {}| {}| {}\".format(tag.lower(), tag.upper(), tag.capitalize()), \"\"\"<span style=\"color: #ff0000;\"> {}</span>\"\"\".format(tag.capitalize()), abstract)\n",
    "                    top_sentences = re.sub(\" {}| {}| {}\".format(tag.lower(), tag.upper(), tag.capitalize()), \"\"\"<span style=\"color: #ff0000;\"> {}</span>\"\"\".format(tag.capitalize()), top_sentences)\n",
    "                    body_text = re.sub(\" {}| {}| {}\".format(tag.lower(), tag.upper(), tag.capitalize()), \"\"\"<span style=\"color: #ff0000;\"> {}</span>\"\"\".format(tag.capitalize()), body_text)\n",
    "                \n",
    "                \n",
    "            #adding paper's summary\n",
    "            summary = summary + \"\"\"\n",
    "            <p>&nbsp;</p>\n",
    "            <p>&nbsp;</p>\n",
    "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper {nb}.</span></h1>\n",
    "            <p>&nbsp;</p>\n",
    "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> {title} </span></span></h1>\n",
    "            <p>&nbsp;</p>\n",
    "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: {authors} </span></span></strong></p>\n",
    "            <p><span style=\"color: #003366;\"><strong>Publication date: {date} </strong></span></p>\n",
    "            <p><a href={url} ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
    "            <p>&nbsp;</p>\n",
    "            <h2>\n",
    "                <a href=\"javascript:visibilite('divid1{html_id}');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
    "                | <a href=\"javascript:visibilite('divid2{html_id}');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
    "                | <a href=\"javascript:visibilite('divid3{html_id}');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
    "            </h2>\n",
    "            <p>&nbsp;</p>\n",
    "            <div id=\"divid1{html_id}\" style=\"display:none;\"></p>\n",
    "                <h4  style=\"text-align: justify;\">{abstract}</strong></h4>\n",
    "            </div>\n",
    "            <div id=\"divid2{html_id}\" style=\"display:none;\"></p>\n",
    "                <p  style=\"text-align: justify;\">{top_sentences}</strong></p>\n",
    "            </div>\n",
    "            <div id=\"divid3{html_id}\" style=\"display:none;\"></p>\n",
    "                <p  style=\"text-align: justify;\">{body_text}</strong></p>\n",
    "            </div>\n",
    "            \"\"\".format(nb=i+1, html_id=self.html_index, title=title, authors=authors, date=date, url=url, abstract=abstract, top_sentences=top_sentences, body_text=body_text)\n",
    "            \n",
    "            self.html_index+=1 \n",
    "            \n",
    "            \n",
    "            \n",
    "        summary = HTML(summary)\n",
    "    \n",
    "        return display(summary)\n",
    "    \n",
    "    \n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/opt/conda/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3063: DtypeWarning: Columns (14) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    }
   ],
   "source": [
    "papers_preprocessed_df = pd.read_pickle(\"../input/preprocessedcord-19/papers_preprocessed.pkl\") \n",
    "meta_df = pd.read_csv(\"../input/CORD-19-research-challenge/metadata.csv\")\n",
    "model_doc2vec =  Doc2Vec.load(\"../input/preprocessedcord-19/doc2vec_model_tag_papers\")\n",
    "model_word2vec =  Word2Vec.load(\"../input/preprocessedcord-19/word2vec_cbow.model\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paper_id</th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>body_text</th>\n",
       "      <th>bib_titles</th>\n",
       "      <th>dataset</th>\n",
       "      <th>publish_time</th>\n",
       "      <th>url</th>\n",
       "      <th>paper</th>\n",
       "      <th>paper_token</th>\n",
       "      <th>paper_tag</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0001418189999fea7f7cbe3e82703d71c85a6fe5</td>\n",
       "      <td>Absence of surface expression of feline infect...</td>\n",
       "      <td>Cornelissen, E.; Dewerchin, H.L.; Van Hamme, E...</td>\n",
       "      <td>Abstract Feline infectious peritonitis virus (...</td>\n",
       "      <td>Feline infectious peritonitis (FIP) is a fatal...</td>\n",
       "      <td>Using direct immunofluorescence to detect coro...</td>\n",
       "      <td>../data/unstructured/custom_license</td>\n",
       "      <td>2007-03-31</td>\n",
       "      <td>https://doi.org/10.1016/j.vetmic.2006.11.026</td>\n",
       "      <td>Absence of surface expression of feline infect...</td>\n",
       "      <td>[absenc, surfac, express, felin, infecti, peri...</td>\n",
       "      <td>([absenc, surfac, express, felin, infecti, per...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>001259ae6d9bfa9376894f61aa6b6c5f18be2177</td>\n",
       "      <td>Microheterogeneity of S-glycoprotein of mouse ...</td>\n",
       "      <td>Oleszak, Emilia L.; Knisley, Keith; Rodkey, L....</td>\n",
       "      <td>Abstract Mouse hepatitis virus (MHV) strain JH...</td>\n",
       "      <td>(Accepted 10 January 1992) is a neurotropic co...</td>\n",
       "      <td>Selection of a single antibody-forming cell cl...</td>\n",
       "      <td>../data/unstructured/custom_license</td>\n",
       "      <td>1992-07-31</td>\n",
       "      <td>https://doi.org/10.1016/0166-0934(92)90173-b</td>\n",
       "      <td>Microheterogeneity of S-glycoprotein of mouse ...</td>\n",
       "      <td>[microheterogen, glycoprotein, mous, hepat, vi...</td>\n",
       "      <td>([microheterogen, glycoprotein, mous, hepat, v...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>00142f93c18b07350be89e96372d240372437ed9</td>\n",
       "      <td>Immunity to Pathogens Taught by Specialized Hu...</td>\n",
       "      <td>Geginat, Jens; Nizzoli, Giulia; Paroni, Moira;...</td>\n",
       "      <td>Dendritic cells (DCs) are specialized antigen-...</td>\n",
       "      <td>iNTRODUCTiON Human beings are constantly expos...</td>\n",
       "      <td>The dendritic cell system and its role in immu...</td>\n",
       "      <td>../data/unstructured/comm_use_subset</td>\n",
       "      <td>2015-10-13</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4...</td>\n",
       "      <td>Immunity to Pathogens Taught by Specialized Hu...</td>\n",
       "      <td>[immun, pathogen, taught, special, human, dend...</td>\n",
       "      <td>([immun, pathogen, taught, special, human, den...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0015023cc06b5362d332b3baf348d11567ca2fbb</td>\n",
       "      <td>The RNA pseudoknots in foot-and-mouth disease ...</td>\n",
       "      <td>Joseph C. Ward; Lidia Lasecka-Dykes; Chris Nei...</td>\n",
       "      <td>AbstractThe positive stranded RNA genomes of p...</td>\n",
       "      <td>VP3, and VP0 (which is further processed to VP...</td>\n",
       "      <td>Genetic economy in 598 picornaviruses: Foot-an...</td>\n",
       "      <td>../data/unstructured/biorxiv_medrxiv</td>\n",
       "      <td>2020-01-11</td>\n",
       "      <td>https://doi.org/10.1101/2020.01.10.901801</td>\n",
       "      <td>The RNA pseudoknots in foot-and-mouth disease ...</td>\n",
       "      <td>[rna, pseudoknot, foot, mouth, diseas, viru, d...</td>\n",
       "      <td>([rna, pseudoknot, foot, mouth, diseas, viru, ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0016983545c217458b711d5aaad5fba583d81171</td>\n",
       "      <td>Enhancement of feline infectious peritonitis v...</td>\n",
       "      <td>Hohdatsu, Tsutomu; Tatekawa, Takahiro; Koyama,...</td>\n",
       "      <td>Abstract The effects of centrifugation on the ...</td>\n",
       "      <td>Feline infectious peritonitis virus (FIPV), fa...</td>\n",
       "      <td>Characterization of monoclonal antibodies agai...</td>\n",
       "      <td>../data/unstructured/custom_license</td>\n",
       "      <td>1995-02-28</td>\n",
       "      <td>https://doi.org/10.1016/0166-0934(94)00119-2</td>\n",
       "      <td>Enhancement of feline infectious peritonitis v...</td>\n",
       "      <td>[enhanc, felin, infecti, periton, viru, type, ...</td>\n",
       "      <td>([enhanc, felin, infecti, periton, viru, type,...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                   paper_id  \\\n",
       "0  0001418189999fea7f7cbe3e82703d71c85a6fe5   \n",
       "1  001259ae6d9bfa9376894f61aa6b6c5f18be2177   \n",
       "2  00142f93c18b07350be89e96372d240372437ed9   \n",
       "3  0015023cc06b5362d332b3baf348d11567ca2fbb   \n",
       "4  0016983545c217458b711d5aaad5fba583d81171   \n",
       "\n",
       "                                               title  \\\n",
       "0  Absence of surface expression of feline infect...   \n",
       "1  Microheterogeneity of S-glycoprotein of mouse ...   \n",
       "2  Immunity to Pathogens Taught by Specialized Hu...   \n",
       "3  The RNA pseudoknots in foot-and-mouth disease ...   \n",
       "4  Enhancement of feline infectious peritonitis v...   \n",
       "\n",
       "                                             authors  \\\n",
       "0  Cornelissen, E.; Dewerchin, H.L.; Van Hamme, E...   \n",
       "1  Oleszak, Emilia L.; Knisley, Keith; Rodkey, L....   \n",
       "2  Geginat, Jens; Nizzoli, Giulia; Paroni, Moira;...   \n",
       "3  Joseph C. Ward; Lidia Lasecka-Dykes; Chris Nei...   \n",
       "4  Hohdatsu, Tsutomu; Tatekawa, Takahiro; Koyama,...   \n",
       "\n",
       "                                            abstract  \\\n",
       "0  Abstract Feline infectious peritonitis virus (...   \n",
       "1  Abstract Mouse hepatitis virus (MHV) strain JH...   \n",
       "2  Dendritic cells (DCs) are specialized antigen-...   \n",
       "3  AbstractThe positive stranded RNA genomes of p...   \n",
       "4  Abstract The effects of centrifugation on the ...   \n",
       "\n",
       "                                           body_text  \\\n",
       "0  Feline infectious peritonitis (FIP) is a fatal...   \n",
       "1  (Accepted 10 January 1992) is a neurotropic co...   \n",
       "2  iNTRODUCTiON Human beings are constantly expos...   \n",
       "3  VP3, and VP0 (which is further processed to VP...   \n",
       "4  Feline infectious peritonitis virus (FIPV), fa...   \n",
       "\n",
       "                                          bib_titles  \\\n",
       "0  Using direct immunofluorescence to detect coro...   \n",
       "1  Selection of a single antibody-forming cell cl...   \n",
       "2  The dendritic cell system and its role in immu...   \n",
       "3  Genetic economy in 598 picornaviruses: Foot-an...   \n",
       "4  Characterization of monoclonal antibodies agai...   \n",
       "\n",
       "                                dataset publish_time  \\\n",
       "0   ../data/unstructured/custom_license   2007-03-31   \n",
       "1   ../data/unstructured/custom_license   1992-07-31   \n",
       "2  ../data/unstructured/comm_use_subset   2015-10-13   \n",
       "3  ../data/unstructured/biorxiv_medrxiv   2020-01-11   \n",
       "4   ../data/unstructured/custom_license   1995-02-28   \n",
       "\n",
       "                                                 url  \\\n",
       "0       https://doi.org/10.1016/j.vetmic.2006.11.026   \n",
       "1       https://doi.org/10.1016/0166-0934(92)90173-b   \n",
       "2  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4...   \n",
       "3          https://doi.org/10.1101/2020.01.10.901801   \n",
       "4       https://doi.org/10.1016/0166-0934(94)00119-2   \n",
       "\n",
       "                                               paper  \\\n",
       "0  Absence of surface expression of feline infect...   \n",
       "1  Microheterogeneity of S-glycoprotein of mouse ...   \n",
       "2  Immunity to Pathogens Taught by Specialized Hu...   \n",
       "3  The RNA pseudoknots in foot-and-mouth disease ...   \n",
       "4  Enhancement of feline infectious peritonitis v...   \n",
       "\n",
       "                                         paper_token  \\\n",
       "0  [absenc, surfac, express, felin, infecti, peri...   \n",
       "1  [microheterogen, glycoprotein, mous, hepat, vi...   \n",
       "2  [immun, pathogen, taught, special, human, dend...   \n",
       "3  [rna, pseudoknot, foot, mouth, diseas, viru, d...   \n",
       "4  [enhanc, felin, infecti, periton, viru, type, ...   \n",
       "\n",
       "                                           paper_tag  \n",
       "0  ([absenc, surfac, express, felin, infecti, per...  \n",
       "1  ([microheterogen, glycoprotein, mous, hepat, v...  \n",
       "2  ([immun, pathogen, taught, special, human, den...  \n",
       "3  ([rna, pseudoknot, foot, mouth, diseas, viru, ...  \n",
       "4  ([enhanc, felin, infecti, periton, viru, type,...  "
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#papers are preprocessed \n",
    "papers_preprocessed_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 9 µs, sys: 3 µs, total: 12 µs\n",
      "Wall time: 14.5 µs\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "searching = Search(papers_df=papers_preprocessed_df, papers_preprocessed=True,\n",
    "                   model_doc2vec = model_doc2vec)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "                <script>\n",
       "                function visibilite(thingId)\n",
       "                {\n",
       "                var targetElement;\n",
       "                targetElement = document.getElementById(thingId) ;\n",
       "                if (targetElement.style.display == \"none\")\n",
       "                {\n",
       "                targetElement.style.display = \"\" ;\n",
       "                } else {\n",
       "                targetElement.style.display = \"none\" ;\n",
       "                }\n",
       "                }\n",
       "                </script>\n",
       "                </body>\n",
       "                \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 1.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Transmissible Gastroenteritis Virus of Pigs and Porcine Epidemic Diarrhea Virus </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Liu, Qiang; Gerdts, Volker </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2019-12-31 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1016/b978-0-12-809633-8.20928-x ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid10');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid20');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid30');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid10\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">Abstract TGEV and PEDV are porcine<span style=\"color: #ff00ff;\"> Coronavir</span>uses with typical<span style=\"color: #ff00ff;\"> Coronavir</span>us<span style=\"color: #ff00ff;\"> Virion</span> structures and<span style=\"color: #ff00ff;\"> Replic</span>ation cycles. Both<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es cause diarrhea in pigs with similar clinical<span style=\"color: #ff00ff;\"> Symptom</span>s.<span style=\"color: #ff0000;\"> Diseas</span>e severity is inversely related to the age of animals. While TGEV<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ions are currently under control, recent emergence of<span style=\"color: #ff00ff;\"> Viru</span>lent PEDV<span style=\"color: #ff00ff;\"> Strain</span>s results in significant<span style=\"color: #ff00ff;\"> Mortal</span>ity. No<span style=\"color: #ff00ff;\"> Effect</span>ive treatments are available once animals are<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ed. Detection of<span style=\"color: #ff0000;\"> Viral</span> antigen and/or genomic RNA is required for diagnosis. Prevention measures include enhanced biosecurity and vaccination with the latter being the most<span style=\"color: #ff00ff;\"> Effect</span>ive approach. Commercial vaccines for both<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es are available and numerous experimental vaccines are in development.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid20\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The appearance of PEDV in North America was reported in 2013 where the<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> continues to circulate in swine herds and there is a potential for new PEDV<span style=\"color: #ff00ff;\"> Strain</span>s to emerge.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Villous enterocytes in small and large intestine are the major target cells of TGEV and PEDV<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ions.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The<span style=\"color: #ff00ff;\"> Mortal</span>ity rate can reach greater than 90% and is inversely related to the age of pigs.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Both TGEV and PEDV cause enteritis in pigs with very similar clinical<span style=\"color: #ff00ff;\"> Symptom</span>s.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Enhanced biosecurity procedures are a general means to contain the<span style=\"color: #ff00ff;\"> Spread</span> and prevent the entrance of<span style=\"color: #ff0000;\"> Viral</span><span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ion in pig farms, but vaccination represents the most<span style=\"color: #ff00ff;\"> Effect</span>ive way in preventing TGEV and PEDV<span style=\"color: #ff00ff;\"> Outbreak</span>s.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Common diagnostic assays include<span style=\"color: #ff0000;\"> Viral</span> antigen detection by histoimmunochemistry and enzyme-linked immuosorbent assay (ELISA),<span style=\"color: #ff0000;\"> Viral</span> RNA detection and genotyping by PCR and sequencing,<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> isolation by cell culture, as well as antibody detection by serology.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Because clinical signs cannot distinguish between TGEV and PEDV<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ions, additional assays are required for diagnosis.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 8:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Because of high<span style=\"color: #ff00ff;\"> Mortal</span>ity in piglets, it has become a common practice to vaccinate sows in order to transfer lactogenic immunity to protect piglets from TGEV and PEDV<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ions.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 9:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Live<span style=\"color: #ff00ff;\"> Attenu</span>ated and inactivated<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> vaccines have been developed for both TGEV and PEDV.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 10:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">There is evidence to suggest that<span style=\"color: #ff0000;\"> Viral</span> proteins, such as spike and ORF3 proteins may<span style=\"color: #ff00ff;\"> Affect</span><span style=\"color: #ff0000;\"> Viral</span><span style=\"color: #ff00ff;\"> Viru</span>lence.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid30\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">The life cycles of TGEV and PEDV consist of<span style=\"color: #ff00ff;\"> Virion</span> attachment and entry,<span style=\"color: #ff0000;\"> Viral</span> RNA translation,<span style=\"color: #ff0000;\"> Viral</span> RNA<span style=\"color: #ff00ff;\"> Replic</span>ation and transcription,<span style=\"color: #ff00ff;\"> Virion</span> assembly and release (Fig. 2) .<span style=\"color: #ff00ff;\"> Virion</span> attachment to a host cell requires the interaction between<span style=\"color: #ff0000;\"> Viral</span> S protein with cellular receptors. Aminopeptidase N (APN) is a major receptor for TGEV. However, whether APN is an essential receptor for PEDV is debatable. PEDV S protein can bind to sialic acid on the cell surface that may mediate<span style=\"color: #ff00ff;\"> Virion</span> attachment. After receptor binding, the S protein undergoes proteolytic cleavage that in turn induces membrane fusion and<span style=\"color: #ff00ff;\"> Virion</span> entry.<span style=\"color: #ff0000;\"> Viral</span> RNA with a 5′ cap structure and a 3′ polyadenylated tail serves as an mRNA that is translated by cellular translation machinery to generate<span style=\"color: #ff0000;\"> Viral</span><span style=\"color: #ff00ff;\"> Replic</span>ase proteins pp1a and pp1ab.<span style=\"color: #ff00ff;\"> Replic</span>ation and transcription complexes formed by these proteins produce both genomic and subgenomic progeny RNA species. Structural and accessory proteins are translated from subgenomic RNAs. Nucleocapsids formed by the N protein and progeny<span style=\"color: #ff0000;\"> Viral</span> genomic RNA are enveloped in the endoplasmic reticulum -Golgi intermediate compartment with the involvement of S, E, and M proteins. The assembled<span style=\"color: #ff00ff;\"> Virion</span>s are transported to the cell surface and released.<span style=\"color: #ff00ff;\"> Epidem</span>iology TGEV, first described in 1946 in USA, has been detected all over the world. PEDV was first isolated in 1978 in Belgium and is widespread in Europe and Asia ever since. The appearance of PEDV in North America was reported in 2013 where the<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> continues to circulate in swine herds and there is a potential for new PEDV<span style=\"color: #ff00ff;\"> Strain</span>s to emerge. Both TGEV and PEDV cause enteritis in pigs with very similar clinical<span style=\"color: #ff00ff;\"> Symptom</span>s. Major clinical signs include vomiting, watery diarrhea, dehydration, and weight loss. The<span style=\"color: #ff00ff;\"> Mortal</span>ity rate can reach greater than 90% and is inversely related to the age of pigs. Penetration and uncoating occur after the S protein-mediated fusion of the<span style=\"color: #ff0000;\"> Viral</span> envelope with the plasma membrane. Following disassembly, the<span style=\"color: #ff0000;\"> Viral</span> genome is released into the cytoplasm and immediately translated to yield<span style=\"color: #ff00ff;\"> Replic</span>ases ppla and pp1ab. These polyproteins are proteolytically cleaved into 16 nsps comprising the<span style=\"color: #ff00ff;\"> Replic</span>ation and transcription complex (RTC) that first engages in the minus-strand RNA synthesis using genomic RNA. Both full-and sub genomic (sg)-length minus strands are produced and used to synthesize full-length genomic RNA and sg mRNAs. Each sg mRNA is translated to yield only the protein encoded by the 5'-most ORF of the sg mRNA. The envelope S, E, and M proteins are inserted in the ER and anchored in the Golgi apparatus. The N protein interacts with newly synthesized genomic RNA to form helical RNP complexes. The progeny<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> is assembled by budding of the preformed RNP at the ER-Golgi intermediate compartment (ERGIC) and then released by the exocytosis-like fusion of smoothwalled,<span style=\"color: #ff00ff;\"> Virion</span>-containing vesicles with the plasma membrane. Reproduced from Lee, C., 2015. Porcine<span style=\"color: #ff00ff;\"> Epidem</span>ic diarrhea<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>: An emerging and re-emerging epizootic swine<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>. Virology Journal, 12, 193, under BioMed Central license agreement. Villous enterocytes in small and large intestine are the major target cells of TGEV and PEDV<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ions.<span style=\"color: #ff0000;\"> Viral</span><span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ion causes cell death that results in villous atrophy followed by malabsorption, diarrhea, dehydration, anorexia, and eventually animal death. The molecular mechanisms for<span style=\"color: #ff00ff;\"> Pathogen</span>esis are not very well understood. There is evidence to suggest that<span style=\"color: #ff0000;\"> Viral</span> proteins, such as spike and ORF3 proteins may<span style=\"color: #ff00ff;\"> Affect</span><span style=\"color: #ff0000;\"> Viral</span><span style=\"color: #ff00ff;\"> Viru</span>lence. At the molecular level, TGEV and PEDV have been shown to modulate multiple cellular processes/pathways including endoplasmic reticulum stress, cell cycle, and mitogen-activated protein kinase signaling. Because clinical signs cannot distinguish between TGEV and PEDV<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ions, additional assays are required for diagnosis. Common diagnostic assays include<span style=\"color: #ff0000;\"> Viral</span> antigen detection by histoimmunochemistry and enzyme-linked immuosorbent assay (ELISA),<span style=\"color: #ff0000;\"> Viral</span> RNA detection and genotyping by PCR and sequencing,<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> isolation by cell culture, as well as antibody detection by serology. There is no specific treatment for TGEV and PEDV<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ions. Enhanced biosecurity procedures are a general means to contain the<span style=\"color: #ff00ff;\"> Spread</span> and prevent the entrance of<span style=\"color: #ff0000;\"> Viral</span><span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ion in pig farms, but vaccination represents the most<span style=\"color: #ff00ff;\"> Effect</span>ive way in preventing TGEV and PEDV<span style=\"color: #ff00ff;\"> Outbreak</span>s. Because of high<span style=\"color: #ff00ff;\"> Mortal</span>ity in piglets, it has become a common practice to vaccinate sows in order to transfer lactogenic immunity to protect piglets from TGEV and PEDV<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ions. Live<span style=\"color: #ff00ff;\"> Attenu</span>ated and inactivated<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> vaccines have been developed for both TGEV and PEDV. Since the spike protein is the major immunogen, numerous technologies have been employed to express the spike protein. These include DNA vaccine, vectored vaccine, subunit vaccine, and dendritic cell-based vaccine. Experimental and commercial TGEV and PEDV vaccines are listed in Table 1 .</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 2.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> A realistic two-strain model for MERS-CoV infection uncovers the high risk for epidemic propagation </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Sardar, Tridip; Ghosh, Indrajit; Rodó, Xavier; Chattopadhyay, Joydev </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-14 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7046297/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid11');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid21');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid31');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid11\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">Middle East Respiratory Syndrome<span style=\"color: #ff00ff;\"> Coronavir</span>us (MERS-CoV) causes severe acute respiratory<span style=\"color: #ff00ff;\"> Ill</span>ness with a case fatality rate (CFR) of 35,5%. The highest number of MERS-CoV cases are from Saudi-Arabia, the major worldwide hotspot for this<span style=\"color: #ff0000;\"> Diseas</span>e. In the absence of neither<span style=\"color: #ff00ff;\"> Effect</span>ive treatment nor a ready-to-use vaccine and with yet an incomplete understanding of its<span style=\"color: #ff00ff;\"> Epidem</span>iological cycle, prevention and containment measures can be derived from mathematical models of<span style=\"color: #ff0000;\"> Diseas</span>e<span style=\"color: #ff00ff;\"> Epidem</span>iology. We constructed 2-strain models to predict past<span style=\"color: #ff00ff;\"> Outbreak</span>s in the interval 2012–2016 and derive key<span style=\"color: #ff00ff;\"> Epidem</span>iological information for Macca, Madina and Riyadh. We approached variability in<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ion through three different<span style=\"color: #ff0000;\"> Diseas</span>e<span style=\"color: #ff00ff;\"> Incid</span>ence functions capturing<span style=\"color: #ff00ff;\"> Social</span><span style=\"color: #ff0000;\"> Behavior</span> in response to an<span style=\"color: #ff00ff;\"> Epidem</span>ic (e.g. Bilinear, BL; Non-monotone, NM; and Saturated, SAT models). The best model combination successfully anticipated the total number of MERS-CoV clinical cases for the 2015–2016 season and accurately predicted both the number of cases at the peak of seasonal<span style=\"color: #ff00ff;\"> Incid</span>ence and the overall shape of the<span style=\"color: #ff00ff;\"> Epidem</span>ic cycle. The evolution in the basic reproduction number (R(0)) warns that MERS-CoV may easily take an<span style=\"color: #ff00ff;\"> Epidem</span>ic form. The best model correctly captures this feature, indicating a high<span style=\"color: #ff00ff;\"> Epidem</span>ic<span style=\"color: #ff0000;\"> Risk</span> (1≤R(0)≤2,5) in Riyadh and Macca and confirming the alleged co-circulation of more than one<span style=\"color: #ff00ff;\"> Strain</span>. Accurate predictions of the future MERS-CoV peak week, as well as the number of cases at the peak are now possible. These results indicate public health agencies should be<span style=\"color: #ff00ff;\"> Awar</span>e that measures for strict containment are urgently needed before new<span style=\"color: #ff00ff;\"> Epidem</span>ics take off in the region.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid21\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Nucleic acids from the sample were extracted with the PureLink<span style=\"color: #ff0000;\"> Viral</span> RNA/DNA kit.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The complete genome sequence of the Mexican OC43<span style=\"color: #ff00ff;\"> Strain</span> was deposited at GenBank under the accession number KX344031.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">coronavirus (HCoV) OC43 belongs to the Betacoronavirus genus of the<span style=\"color: #ff00ff;\"> Coronavir</span>idae and its genome is formed by a positivesense, single-stranded RNA of ca.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">HCoV OC43 and HCoV 229E are responsible for one-third of all common colds,<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ing all age groups (2), but there have been reports of a more severe lower respiratory tract involvement (3, 4) .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The aim of the present study was to determine the full-length genome of the first HCoV OC43 isolated from Mexico, which was obtained from a<span style=\"color: #ff0000;\"> Viral</span> metagenomic analysis of a nasal washing from a child who required hospitalization due to clinical or radiological signs of pneumonia of unknown origin, negative for commonly associated respiratory<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es and bacteria, using the xTAG Bioplex respiratory<span style=\"color: #ff0000;\"> Viral</span> panel and multiplex PCR Seeplex Pneumobacter ACE detection kit, respectively (5) .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The sequencing was carried out with the<span style=\"color: #ff00ff;\"> Ill</span>umina Genome Analyzer IIx single-end platform.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The whole<span style=\"color: #ff0000;\"> Viral</span> genome sequence was obtained with an average coverage of 3,368ϫ.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 8:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The<span style=\"color: #ff00ff;\"> Strain</span> shows the typical OC43 organization, with the following 5= untranslated region (UTR) (nt 1 to 199), genes: orf1ab (nt 200 to Accession number(s).</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid31\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">coronavirus (HCoV) OC43 belongs to the Betacoronavirus genus of the<span style=\"color: #ff00ff;\"> Coronavir</span>idae and its genome is formed by a positivesense, single-stranded RNA of ca. 31.5 kb (1) . HCoV OC43 and HCoV 229E are responsible for one-third of all common colds,<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ing all age groups (2), but there have been reports of a more severe lower respiratory tract involvement (3, 4) . The aim of the present study was to determine the full-length genome of the first HCoV OC43 isolated from Mexico, which was obtained from a<span style=\"color: #ff0000;\"> Viral</span> metagenomic analysis of a nasal washing from a child who required hospitalization due to clinical or radiological signs of pneumonia of unknown origin, negative for commonly associated respiratory<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es and bacteria, using the xTAG Bioplex respiratory<span style=\"color: #ff0000;\"> Viral</span> panel and multiplex PCR Seeplex Pneumobacter ACE detection kit, respectively (5) . Nucleic acids from the sample were extracted with the PureLink<span style=\"color: #ff0000;\"> Viral</span> RNA/DNA kit. Before extraction, the sample was treated with Turbo DNase and RNase, and a random-primer amplification of the genetic material was performed as previously described (5) . The sequencing was carried out with the<span style=\"color: #ff00ff;\"> Ill</span>umina Genome Analyzer IIx single-end platform. A total of 12,330,418 reads were generated and 1,657,101 (13.44%) were mapped to the consensus sequence of HCoV OC43. Mapping was accomplished by SMALT v.0.7.6 (6) with default parameters (only Ϫy set to 0.8). The whole<span style=\"color: #ff0000;\"> Viral</span> genome sequence was obtained with an average coverage of 3,368ϫ. The same genome sequence was obtained by de novo assembly using Velvet (7) . The complete Mexican HCoV OC43 genome was composed of 30,712 nucleotides, with an average GϩC content of 35.51%. The<span style=\"color: #ff00ff;\"> Strain</span> shows the typical OC43 organization, with the following 5= untranslated region (UTR) (nt 1 to 199), genes: orf1ab (nt 200 to Accession number(s). The complete genome sequence of the Mexican OC43<span style=\"color: #ff00ff;\"> Strain</span> was deposited at GenBank under the accession number KX344031.</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 3.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> RNA decay during gammaherpesvirus infection reduces RNA polymerase II occupancy of host promoters but spares viral promoters </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Hartenian, Ella; Gilbertson, Sarah; Federspiel, Joel D.; Cristea, Ileana M.; Glaunsinger, Britt A. </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-07 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7032723/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid12');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid22');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid32');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid12\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">In mammalian cells, widespread acceleration of cytoplasmic mRNA degradation is linked to impaired RNA polymerase II (Pol II) transcription. This mRNA decay-induced transcriptional repression occurs during<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ion with gammaherpesviruses including Kaposi’s sarcoma-associated herpesvirus (KSHV) and murine gammaherpesvirus 68 (MHV68), which encode an mRNA endonuclease that initiates widespread RNA decay. Here, we show that MHV68-induced mRNA decay leads to a genome-wide reduction of Pol II occupancy at mammalian promoters. This reduced Pol II occupancy is accompanied by down-regulation of multiple Pol II subunits and TFIIB in the nucleus of<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ed cells, as revealed by mass spectrometry-based global measurements of protein abundance.<span style=\"color: #ff0000;\"> Viral</span> genes, despite the fact that they require Pol II for transcription, escape transcriptional repression. Protection is not governed by<span style=\"color: #ff0000;\"> Viral</span> promoter sequences; instead, location on the<span style=\"color: #ff0000;\"> Viral</span> genome is both necessary and sufficient to escape the transcriptional repression<span style=\"color: #ff00ff;\"> Effect</span>s of mRNA decay. We propose a model in which the ability to escape from transcriptional repression is linked to the localization of<span style=\"color: #ff0000;\"> Viral</span> DNA within<span style=\"color: #ff00ff;\"> Replic</span>ation compartments, providing a means for these<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es to counteract decay-induced transcript loss.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid22\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">family<span style=\"color: #ff00ff;\"> Coronavir</span>idae, is an enveloped<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> with a singlestranded, positive-sense RNA and causes porcine<span style=\"color: #ff00ff;\"> Epidem</span>ic diarrhea, an enteric<span style=\"color: #ff0000;\"> Diseas</span>e characterized by acute watery diarrhea, dehydration, vomiting, and high<span style=\"color: #ff00ff;\"> Mortal</span>ity in nursery piglets (1, 2) .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Sequence analysis showed that there was a 49-nt deletion in the ORF3 gene, and this characteristic was similar to a characteristic of the<span style=\"color: #ff00ff;\"> Attenu</span>ated DR13.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">In the autumn of 2010, severe diarrhea in piglets emerged in China,<span style=\"color: #ff00ff;\"> Affect</span>ing more than 1,000,000 piglets (3).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The genome sequence of PEDV<span style=\"color: #ff00ff;\"> Strain</span> JS2008 was deposited in GenBank under accession number KC109141.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">In this study, a PEDV<span style=\"color: #ff00ff;\"> Strain</span>, JS2008, was detected and identified in stool samples from a piglet with severe diarrhea on a vaccinated farm in eastern China.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Phylogenetic analysis of the complete genome showed that JS2008 has a close relationship to the<span style=\"color: #ff00ff;\"> Attenu</span>ated DR13 vaccine<span style=\"color: #ff00ff;\"> Strain</span> and a distant relationship to the<span style=\"color: #ff00ff;\"> Preval</span>ent PEDV variant in China.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Outbreaks of PEDV<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ion have been documented mainly in European and Asian countries since the<span style=\"color: #ff0000;\"> Diseas</span>e was first recognized in an acute<span style=\"color: #ff00ff;\"> Outbreak</span> of diarrhea in pigs (1) .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 8:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The availability of the genome sequence of JS2008 will facilitate future investigations of the<span style=\"color: #ff00ff;\"> Epidem</span>iology and evolution of PEDV.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 9:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">These findings may indicate that JS2008 is a recombinant of the PEDV vaccine<span style=\"color: #ff00ff;\"> Strain</span> and the variant.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 10:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Meanwhile, the E gene was 21 nt longer than that of the<span style=\"color: #ff00ff;\"> Attenu</span>ated DR13 and had the same sequence as that of the PEDV variant.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid32\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">family<span style=\"color: #ff00ff;\"> Coronavir</span>idae, is an enveloped<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> with a singlestranded, positive-sense RNA and causes porcine<span style=\"color: #ff00ff;\"> Epidem</span>ic diarrhea, an enteric<span style=\"color: #ff0000;\"> Diseas</span>e characterized by acute watery diarrhea, dehydration, vomiting, and high<span style=\"color: #ff00ff;\"> Mortal</span>ity in nursery piglets (1, 2) .<span style=\"color: #ff00ff;\"> Outbreak</span>s of PEDV<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ion have been documented mainly in European and Asian countries since the<span style=\"color: #ff0000;\"> Diseas</span>e was first recognized in an acute<span style=\"color: #ff00ff;\"> Outbreak</span> of diarrhea in pigs (1) . In the autumn of 2010, severe diarrhea in piglets emerged in China,<span style=\"color: #ff00ff;\"> Affect</span>ing more than 1,000,000 piglets (3). A PEDV variant has been identified as a main causative<span style=\"color: #ff00ff;\"> Agent</span> in the<span style=\"color: #ff0000;\"> Diseas</span>e (4) (5) (6) (7) . In this study, a PEDV<span style=\"color: #ff00ff;\"> Strain</span>, JS2008, was detected and identified in stool samples from a piglet with severe diarrhea on a vaccinated farm in eastern China. This<span style=\"color: #ff00ff;\"> Strain</span> can be easily isolated in Vero cells. Thus, we sought to analyze the genome sequence of PEDV<span style=\"color: #ff00ff;\"> Strain</span> JS2008 and understand its molecular characteristics. Twenty pairs of oligonucleotide primers to amplify regions of the JS2008 genome were designed from alignments of available PEDV genomes. The PCR products were purified and cloned into the pMD18-T vector (TaKaRa) and sequenced with an ABI3730XL genome sequencer. The terminal sequences were acquired by using a kit for rapid amplification of cDNA ends (RACE) (Clontech, Japan). The complete genome sequence of JS2008 is 27,953 nucleotides (nt) in length, excluding the poly(A) tail. The genome showed 99.7% and 98.0% nucleotide sequence similarity to the<span style=\"color: #ff00ff;\"> Attenu</span>ated DR13 and CV777 vaccine<span style=\"color: #ff00ff;\"> Strain</span>s. It shared 97.3 to 97.5% sequence identity with the PEDV variant that has occurred in recent years in China. The organization of the genome of JS2008 is similar to that of other reported PEDV genomes, with the characteristic gene order 5=-ORF1a/1b-S-ORF3-E-M-N-3= (2, 8, 9) . The ORF1a/1b gene is 20,321 nt in length. The sizes of the other genes, S, ORF3, E, M, and N, are 4,149 nt, 276 nt, 231 nt, 681 nt, and 1,326 nt, respectively. Sequence analysis showed that there was a 49-nt deletion in the ORF3 gene, and this characteristic was similar to a characteristic of the<span style=\"color: #ff00ff;\"> Attenu</span>ated DR13. Meanwhile, the E gene was 21 nt longer than that of the<span style=\"color: #ff00ff;\"> Attenu</span>ated DR13 and had the same sequence as that of the PEDV variant. These findings may indicate that JS2008 is a recombinant of the PEDV vaccine<span style=\"color: #ff00ff;\"> Strain</span> and the variant. Phylogenetic analysis of the complete genome showed that JS2008 has a close relationship to the<span style=\"color: #ff00ff;\"> Attenu</span>ated DR13 vaccine<span style=\"color: #ff00ff;\"> Strain</span> and a distant relationship to the<span style=\"color: #ff00ff;\"> Preval</span>ent PEDV variant in China. The availability of the genome sequence of JS2008 will facilitate future investigations of the<span style=\"color: #ff00ff;\"> Epidem</span>iology and evolution of PEDV. Nucleotide sequence accession number. The genome sequence of PEDV<span style=\"color: #ff00ff;\"> Strain</span> JS2008 was deposited in GenBank under accession number KC109141. This work was supported by the Special Fund for Independent Innovation of Agricultural Science and Technology in Jiangsu Province [CX (12) 3076].</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 4.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Yonghua Wu </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-04 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1101/2020.03.04.975995 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid13');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid23');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid33');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid13\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">AbstractCoronavirus<span style=\"color: #ff0000;\"> Diseas</span>e 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related<span style=\"color: #ff00ff;\"> Coronavir</span>uses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ing human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new<span style=\"color: #ff00ff;\"> Coronavir</span>us sources to<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span> humans in the future.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid23\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">615 (6.5%) samples tested positive for influenza A, while 385 (4.1%) tested positive for influenza B<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es and 10 (0.1%) were co-infections between influenza A and B.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Respiratory<span style=\"color: #ff00ff;\"> Pathogen</span>s continue to present an ever increasing threat to public health (1,2).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Influenza, Respiratory syncytial<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>, human metapneumovirus and other respiratory<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es are major etiological<span style=\"color: #ff00ff;\"> Agent</span>s for influenza like<span style=\"color: #ff00ff;\"> Ill</span>nesses (ILI) (3-5).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Establishment of<span style=\"color: #ff0000;\"> Viral</span> causes of ILI is critical for prevention and mitigation strategies to<span style=\"color: #ff0000;\"> Diseas</span>e threats.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Makerere University Walter Reed Project (MUWRP) together with the Ugandan Ministry of Health and partners undertook surveillance to determine<span style=\"color: #ff0000;\"> Viral</span> causes of influenza-like<span style=\"color: #ff00ff;\"> Ill</span>ness in Uganda.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Of the 2,062 influenza negative samples, results indicated positivity for the following organisms; adenoviruses (9.4%), respiratory syncytial B (7.3%), parainfluenza-3 (4.5%), parainfluenza-1 (4.3%), respiratory syncytial A (3.5%), human bocavirus (1.7%), human metapneumovirus (1.7%), human<span style=\"color: #ff00ff;\"> Coronavir</span>us (1.5%), parainfluenza-4 (1.4%) and parainfluenza-2 (0.9%) by PCR.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Influenza<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es account for about 11% of the causes of influenza like<span style=\"color: #ff00ff;\"> Ill</span>ness, with influenza A being the dominant type.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 8:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Detection of other<span style=\"color: #ff0000;\"> Viral</span> causes of ILI is an indication of the public health threats posed by respiratory<span style=\"color: #ff00ff;\"> Pathogen</span>s.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid33\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">Respiratory<span style=\"color: #ff00ff;\"> Pathogen</span>s continue to present an ever increasing threat to public health (1,2). Influenza, Respiratory syncytial<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>, human metapneumovirus and other respiratory<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es are major etiological<span style=\"color: #ff00ff;\"> Agent</span>s for influenza like<span style=\"color: #ff00ff;\"> Ill</span>nesses (ILI) (3-5). Establishment of<span style=\"color: #ff0000;\"> Viral</span> causes of ILI is critical for prevention and mitigation strategies to<span style=\"color: #ff0000;\"> Diseas</span>e threats. Makerere University Walter Reed Project (MUWRP) together with the Ugandan Ministry of Health and partners undertook surveillance to determine<span style=\"color: #ff0000;\"> Viral</span> causes of influenza-like<span style=\"color: #ff00ff;\"> Ill</span>ness in Uganda. From 2008, MUWRP established hospital-based sentinel sites for surveillance activities. A total of five hospital-based sites were established, where patients aged 6 months or older presenting with ILI were enrolled. Consents were obtained as required, and a throat and/ or nasopharyngeal swab collected. Samples were screened by PCR for<span style=\"color: #ff0000;\"> Viral</span> causes. From October 2008 to March 2017 a total of 9,472 participants were enrolled in the study from five hospital-based surveillance sentinel sites. Majority of participants were children under 5 years n= 8,169 (86.2%). 615 (6.5%) samples tested positive for influenza A, while 385 (4.1%) tested positive for influenza B<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es and 10 (0.1%) were co-infections between influenza A and B. Of the 2,062 influenza negative samples, results indicated positivity for the following organisms; adenoviruses (9.4%), respiratory syncytial B (7.3%), parainfluenza-3 (4.5%), parainfluenza-1 (4.3%), respiratory syncytial A (3.5%), human bocavirus (1.7%), human metapneumovirus (1.7%), human<span style=\"color: #ff00ff;\"> Coronavir</span>us (1.5%), parainfluenza-4 (1.4%) and parainfluenza-2 (0.9%) by PCR. Influenza<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span>es account for about 11% of the causes of influenza like<span style=\"color: #ff00ff;\"> Ill</span>ness, with influenza A being the dominant type. Among the other<span style=\"color: #ff0000;\"> Viral</span> causes of ILI, adenoviruses were the most dominant. Detection of other<span style=\"color: #ff0000;\"> Viral</span> causes of ILI is an indication of the public health threats posed by respiratory<span style=\"color: #ff00ff;\"> Pathogen</span>s. Influenza; Influenza Like<span style=\"color: #ff00ff;\"> Ill</span>ness; Surveillance</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 5.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> The Fight against the 2019-nCoV Outbreak: an Arduous March Has Just Begun </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Yoo, Jin-Hong </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-01-30 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6995816/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid14');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid24');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid34');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid14\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">&lt;![CDATA[No abstract available.]]&gt;</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid24\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">This<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> is a positive sense single stranded RNA<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> belonging to the family<span style=\"color: #ff00ff;\"> Coronavir</span>idae.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Our country is excellent at coping with this disaster, thanks to the experiences that we have gained during the 2015 MERS-CoV<span style=\"color: #ff00ff;\"> Outbreak</span>.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The Korea Center for<span style=\"color: #ff0000;\"> Diseas</span>e Control and Prevention (KCDC) must be the control tower of the present disaster in Korea and all other central or local governmental organizations must cooperate with the KCDC.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Because<span style=\"color: #ff00ff;\"> Epidem</span>ic is a national disaster, not only medical institutions but also governments have to be active.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">This disaster has been warned until recently that new mutants of<span style=\"color: #ff00ff;\"> Coronavir</span>us can occur anytime.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Therefore, it is likely to occur in an environment where human and animals are in close<span style=\"color: #ff00ff;\"> Contact</span>.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The current<span style=\"color: #ff00ff;\"> Outbreak</span> is also suspected of being caused by mutants from species<span style=\"color: #ff00ff;\"> Spillov</span>er in Wuhan's wild animal market.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 8:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The 2019-nCoV is rapidly<span style=\"color: #ff00ff;\"> Spread</span>ing throughout China and around the world in a relatively short period of time.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 9:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">As of 29 January 2020, a total of 6,140 cases were confirmed in 19 countries, including Korea, of which 132 died (in China only) and had a<span style=\"color: #ff00ff;\"> Mortal</span>ity of 2.1%.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 10:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">7 As much as the 2015 MERS-CoV<span style=\"color: #ff00ff;\"> Outbreak</span>, we are also learning a lot of lesson from this disaster.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid34\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">This<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> is a positive sense single stranded RNA<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Viru</span>s</span> belonging to the family<span style=\"color: #ff00ff;\"> Coronavir</span>idae. Most<span style=\"color: #ff00ff;\"> Coronavir</span>uses cause only mild upper respiratory<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ions, but sometimes they cause fatal respiratory<span style=\"color: #ff0000;\"> Diseas</span>e and<span style=\"color: #ff00ff;\"> Outbreak</span>s, as experienced in cases of SARS-CoV or MERS-CoV. This disaster has been warned until recently that new mutants of<span style=\"color: #ff00ff;\"> Coronavir</span>us can occur anytime. 5, 6 The emergence of these mutants is caused by species jumping between human and other animals. Therefore, it is likely to occur in an environment where human and animals are in close<span style=\"color: #ff00ff;\"> Contact</span>. The current<span style=\"color: #ff00ff;\"> Outbreak</span> is also suspected of being caused by mutants from species<span style=\"color: #ff00ff;\"> Spillov</span>er in Wuhan's wild animal market. The 2019-nCoV is rapidly<span style=\"color: #ff00ff;\"> Spread</span>ing throughout China and around the world in a relatively short period of time. As of 29 January 2020, a total of 6,140 cases were confirmed in 19 countries, including Korea, of which 132 died (in China only) and had a<span style=\"color: #ff00ff;\"> Mortal</span>ity of 2.1%. 7 As much as the 2015 MERS-CoV<span style=\"color: #ff00ff;\"> Outbreak</span>, we are also learning a lot of lesson from this disaster. Because<span style=\"color: #ff00ff;\"> Epidem</span>ic is a national disaster, not only medical institutions but also governments have to be active. Hence, honesty and transparency are, above all, the virtues that governments should have. Our country is excellent at coping with this disaster, thanks to the experiences that we have gained during the 2015 MERS-CoV<span style=\"color: #ff00ff;\"> Outbreak</span>. 8 Our defense system is at least more solid and faster than it used to be. What do you expect this nCoV<span style=\"color: #ff00ff;\"> Outbreak</span> to be in the future? Based on the SARS<span style=\"color: #ff00ff;\"> Epidem</span>ic precedence, the<span style=\"color: #ff00ff;\"> Outbreak</span> is expected to last at least three months. And this<span style=\"color: #ff00ff;\"> Outbreak</span> is expected to have a greater amount of<span style=\"color: #ff0000;\"> Transmiss</span>ion than the 2015 MERS-CoV. Recently, the possibility of<span style=\"color: #ff0000;\"> Transmiss</span>ion by asymptomatic<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ed people has also been raised carefully although its evidence is unclear. Therefore, we should also prepare for the<span style=\"color: #ff00ff;\"> Spread</span> to communities by asymptomatic<span style=\"color: #ff0000;\"><span style=\"color: #ff0000;\"> Infect</span></span>ions. 9,10 After the MERS-CoV<span style=\"color: #ff00ff;\"> Outbreak</span> in 2015, there was a self-tormenting expression in the medical profession<span style=\"color: #ff00ff;\"> Societ</span>y: 'Patients die, hospitals die, and civil servants are praised.' But the bad memories of the past should never be repeated. In fact, at present, cooperation between community health centers and private hospitals is not always harmonious. The government needs to be more active, not just to leave everything to the medical staffs. After all, the responsibility for the controlling nationwide<span style=\"color: #ff00ff;\"> Epidem</span>ics lies with health authorities of the government. The Korea Center for<span style=\"color: #ff0000;\"> Diseas</span>e Control and Prevention (KCDC) must be the control tower of the present disaster in Korea and all other central or local governmental organizations must cooperate with the KCDC. Toward the end of the 2019-nCoV<span style=\"color: #ff00ff;\"> Outbreak</span> we must go on a march of arduousness. Health workers, the government, and the people will need to unite to overcome this disaster.</strong></p>\n",
       "            </div>\n",
       "            "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "search_risk = \"What are covid-19 risk factors?\"\n",
    "tags_risk = \"\"\"risk smoking pulmonary disease co-infection respiratory viral infection \n",
    "transmissible virulent co-morbidities co-morbidity Neonate pregnant Socio-economic behavior economic impact\"\"\"\n",
    "\n",
    "searching.SearchDisplayPaper(search=search_risk, pubdate_in=\"2019-12-01 2020-04-01\", topn=5, topn_sent=10, tags=tags_risk)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "                <script>\n",
       "                function visibilite(thingId)\n",
       "                {\n",
       "                var targetElement;\n",
       "                targetElement = document.getElementById(thingId) ;\n",
       "                if (targetElement.style.display == \"none\")\n",
       "                {\n",
       "                targetElement.style.display = \"\" ;\n",
       "                } else {\n",
       "                targetElement.style.display = \"none\" ;\n",
       "                }\n",
       "                }\n",
       "                </script>\n",
       "                </body>\n",
       "                \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 1.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Transmissible Gastroenteritis Virus of Pigs and Porcine Epidemic Diarrhea Virus </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Liu, Qiang; Gerdts, Volker </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2019-12-31 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1016/b978-0-12-809633-8.20928-x ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid15');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid25');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid35');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid15\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">Abstract TGEV and PEDV are porcine coronaviruses with typical coronavirus virion structures and replication cycles. Both viruses cause diarrhea in pigs with similar clinical symptoms. Disease severity is inversely related to the age of animals. While TGEV infections are currently under control, recent emergence of virulent PEDV strains results in significant mortality. No effective treatments are available once animals are infected. Detection of viral antigen and/or genomic RNA is required for diagnosis. Prevention measures include enhanced biosecurity and vaccination with the latter being the most effective approach. Commercial vaccines for both viruses are available and numerous experimental vaccines are in development.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid25\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Both full-and sub genomic (sg)-length<span style=\"color: #ff00ff;\"> Min</span>us strands are produced and used to synthesize full-length genomic RNA and sg mRNAs.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Enhanced biosecurity procedures are a general<span style=\"color: #ff00ff;\"> Mean</span>s to contain the<span style=\"color: #ff00ff;\"> Spread</span> and prevent the entrance of viral infection in pig farms, but vaccination represents the most effective way in preventing TGEV and PEDV<span style=\"color: #ff00ff;\"> Outbreak</span>s.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The progeny virus is assembled by budding of the preformed RNP at the ER-Golgi intermediate<span style=\"color: #ff00ff;\"> Compartment</span> (ERGIC) and then released by the exocytosis-like fusion of smoothwalled, virion-containing vesicles with the plasma membrane.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">These polyproteins are proteolytically cleaved into 16 nsps comprising the replication and transcription complex (RTC) that first engages in the<span style=\"color: #ff00ff;\"> Min</span>us-strand RNA synthesis using genomic RNA.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Nucleocapsids formed by the N protein and progeny viral genomic RNA are enveloped in the endoplasmic reticulum -Golgi intermediate<span style=\"color: #ff00ff;\"> Compartment</span> with the involvement of S, E, and M proteins.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Because clinical signs cannot distinguish between TGEV and PEDV infections,<span style=\"color: #ff00ff;\"> Ad</span>ditional assays are required for diagnosis.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Porcine<span style=\"color: #ff00ff;\"> Epidem</span>ic diarrhea virus: An emerging and re-emerging epizootic swine virus.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid35\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">The life cycles of TGEV and PEDV consist of virion attachment and entry, viral RNA translation, viral RNA replication and transcription, virion assembly and release (Fig. 2) . Virion attachment to a host cell requires the interaction between viral S protein with cellular receptors. Aminopeptidase N (APN) is a major receptor for TGEV. However, whether APN is an essential receptor for PEDV is debatable. PEDV S protein can bind to sialic acid on the cell surface that may mediate virion attachment. After receptor binding, the S protein undergoes proteolytic cleavage that in turn induces membrane fusion and virion entry. Viral RNA with a 5′ cap structure and a 3′ polyadenylated tail serves as an mRNA that is translated by cellular translation machinery to generate viral replicase proteins pp1a and pp1ab. Replication and transcription complexes formed by these proteins produce both genomic and subgenomic progeny RNA species. Structural and accessory proteins are translated from subgenomic RNAs. Nucleocapsids formed by the N protein and progeny viral genomic RNA are enveloped in the endoplasmic reticulum -Golgi intermediate<span style=\"color: #ff00ff;\"> Compartment</span> with the involvement of S, E, and M proteins. The assembled virions are transported to the cell surface and released.<span style=\"color: #ff00ff;\"> Epidem</span>iology TGEV, first described in 1946 in USA, has been detected all over the world. PEDV was first isolated in 1978 in Belgium and is widespread in Europe and Asia ever since. The appearance of PEDV in North America was reported in 2013 where the virus continues to circulate in swine herds and there is a potential for new PEDV strains to emerge. Both TGEV and PEDV cause enteritis in pigs with very similar clinical symptoms. Major clinical signs include vomiting, watery diarrhea, dehydration, and weight loss. The mortality rate can reach greater than 90% and is inversely related to the age of pigs. Penetration and uncoating occur after the S protein-mediated fusion of the viral envelope with the plasma membrane. Following disassembly, the viral genome is released into the cytoplasm and immediately translated to yield replicases ppla and pp1ab. These polyproteins are proteolytically cleaved into 16 nsps comprising the replication and transcription complex (RTC) that first engages in the<span style=\"color: #ff00ff;\"> Min</span>us-strand RNA synthesis using genomic RNA. Both full-and sub genomic (sg)-length<span style=\"color: #ff00ff;\"> Min</span>us strands are produced and used to synthesize full-length genomic RNA and sg mRNAs. Each sg mRNA is translated to yield only the protein encoded by the 5'-most ORF of the sg mRNA. The envelope S, E, and M proteins are inserted in the ER and anchored in the Golgi apparatus. The N protein interacts with newly synthesized genomic RNA to form helical RNP complexes. The progeny virus is assembled by budding of the preformed RNP at the ER-Golgi intermediate<span style=\"color: #ff00ff;\"> Compartment</span> (ERGIC) and then released by the exocytosis-like fusion of smoothwalled, virion-containing vesicles with the plasma membrane. Reproduced from Lee, C., 2015. Porcine<span style=\"color: #ff00ff;\"> Epidem</span>ic diarrhea virus: An emerging and re-emerging epizootic swine virus. Virology Journal, 12, 193, under BioMed Central license agreement. Villous enterocytes in small and large intestine are the major target cells of TGEV and PEDV infections. Viral infection causes cell death that results in villous atrophy followed by malabsorption, diarrhea, dehydration, anorexia, and eventually animal death. The molecular mechanisms for pathogenesis are not very well understood. There is evidence to suggest that viral proteins, such as spike and ORF3 proteins may affect viral virulence. At the molecular level, TGEV and PEDV have been shown to modulate multiple cellular processes/pathways including endoplasmic reticulum stress, cell cycle, and mitogen-activated protein kinase signaling. Because clinical signs cannot distinguish between TGEV and PEDV infections,<span style=\"color: #ff00ff;\"> Ad</span>ditional assays are required for diagnosis. Common diagnostic assays include viral antigen detection by histoimmunochemistry and enzyme-linked immuosorbent assay (ELISA), viral RNA detection and genotyping by PCR and sequencing, virus isolation by cell culture, as well as antibody detection by serology. There is no specific treatment for TGEV and PEDV infections. Enhanced biosecurity procedures are a general<span style=\"color: #ff00ff;\"> Mean</span>s to contain the<span style=\"color: #ff00ff;\"> Spread</span> and prevent the entrance of viral infection in pig farms, but vaccination represents the most effective way in preventing TGEV and PEDV<span style=\"color: #ff00ff;\"> Outbreak</span>s. Because of high mortality in piglets, it has become a common practice to vaccinate sows in order to transfer lactogenic immunity to protect piglets from TGEV and PEDV infections. Live attenuated and inactivated virus vaccines have been developed for both TGEV and PEDV. Since the spike protein is the major immunogen, numerous technologies have been employed to express the spike protein. These include DNA vaccine, vectored vaccine, subunit vaccine, and dendritic cell-based vaccine. Experimental and commercial TGEV and PEDV vaccines are listed in Table 1 .</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 2.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> RNA decay during gammaherpesvirus infection reduces RNA polymerase II occupancy of host promoters but spares viral promoters </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Hartenian, Ella; Gilbertson, Sarah; Federspiel, Joel D.; Cristea, Ileana M.; Glaunsinger, Britt A. </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-07 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7032723/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid16');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid26');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid36');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid16\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">In mammalian cells, widespread acceleration of cytoplasmic mRNA degradation is linked to impaired RNA polymerase II (Pol II) transcription. This mRNA decay-induced transcriptional repression occurs during infection with gammaherpesviruses including Kaposi’s sarcoma-associated herpesvirus (KSHV) and murine gammaherpesvirus 68 (MHV68), which encode an mRNA endonuclease that initiates widespread RNA decay. Here, we show that MHV68-induced mRNA decay leads to a genome-wide reduction of Pol II occupancy at mammalian promoters. This reduced Pol II occupancy is accompanied by down-regulation of multiple Pol II subunits and TFIIB in the nucleus of infected cells, as revealed by mass spectrometry-based global measurements of protein abundance. Viral genes, despite the fact that they require Pol II for transcription, escape transcriptional repression. Protection is not governed by viral promoter sequences; instead, location on the viral genome is both necessary and sufficient to escape the transcriptional repression effects of mRNA decay. We propose a model in which the ability to escape from transcriptional repression is linked to the localization of viral DNA within replication<span style=\"color: #ff00ff;\"> Compartment</span>s, providing a<span style=\"color: #ff00ff;\"> Mean</span>s for these viruses to counteract decay-induced transcript loss.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid26\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">family Coronaviridae, is an enveloped virus with a singlestranded, positive-sense RNA and causes porcine<span style=\"color: #ff00ff;\"> Epidem</span>ic diarrhea, an enteric disease characterized by acute watery diarrhea, dehydration, vomiting, and high mortality in nursery piglets (1, 2) .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Outbreaks of PEDV infection have been documented mainly in European and Asian countries since the disease was first recognized in an acute<span style=\"color: #ff00ff;\"> Outbreak</span> of diarrhea in pigs (1) .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The availability of the genome sequence of JS2008 will facilitate future investigations of the<span style=\"color: #ff00ff;\"> Epidem</span>iology and evolution of PEDV.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid36\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">family Coronaviridae, is an enveloped virus with a singlestranded, positive-sense RNA and causes porcine<span style=\"color: #ff00ff;\"> Epidem</span>ic diarrhea, an enteric disease characterized by acute watery diarrhea, dehydration, vomiting, and high mortality in nursery piglets (1, 2) .<span style=\"color: #ff00ff;\"> Outbreak</span>s of PEDV infection have been documented mainly in European and Asian countries since the disease was first recognized in an acute<span style=\"color: #ff00ff;\"> Outbreak</span> of diarrhea in pigs (1) . In the autumn of 2010, severe diarrhea in piglets emerged in China, affecting more than 1,000,000 piglets (3). A PEDV variant has been identified as a main causative agent in the disease (4) (5) (6) (7) . In this study, a PEDV strain, JS2008, was detected and identified in stool samples from a piglet with severe diarrhea on a vaccinated farm in eastern China. This strain can be easily isolated in Vero cells. Thus, we sought to analyze the genome sequence of PEDV strain JS2008 and understand its molecular characteristics. Twenty pairs of oligonucleotide primers to amplify regions of the JS2008 genome were designed from alignments of available PEDV genomes. The PCR products were purified and cloned into the pMD18-T vector (TaKaRa) and sequenced with an ABI3730XL genome sequencer. The terminal sequences were acquired by using a kit for rapid amplification of cDNA ends (RACE) (Clontech, Japan). The complete genome sequence of JS2008 is 27,953 nucleotides (nt) in length, excluding the poly(A) tail. The genome showed 99.7% and 98.0% nucleotide sequence similarity to the attenuated DR13 and CV777 vaccine strains. It shared 97.3 to 97.5% sequence identity with the PEDV variant that has occurred in recent years in China. The organization of the genome of JS2008 is similar to that of other reported PEDV genomes, with the characteristic gene order 5=-ORF1a/1b-S-ORF3-E-M-N-3= (2, 8, 9) . The ORF1a/1b gene is 20,321 nt in length. The sizes of the other genes, S, ORF3, E, M, and N, are 4,149 nt, 276 nt, 231 nt, 681 nt, and 1,326 nt, respectively. Sequence analysis showed that there was a 49-nt deletion in the ORF3 gene, and this characteristic was similar to a characteristic of the attenuated DR13.<span style=\"color: #ff00ff;\"> Mean</span>while, the E gene was 21 nt longer than that of the attenuated DR13 and had the same sequence as that of the PEDV variant. These findings may indicate that JS2008 is a recombinant of the PEDV vaccine strain and the variant. Phylogenetic analysis of the complete genome showed that JS2008 has a close relationship to the attenuated DR13 vaccine strain and a distant relationship to the prevalent PEDV variant in China. The availability of the genome sequence of JS2008 will facilitate future investigations of the<span style=\"color: #ff00ff;\"> Epidem</span>iology and evolution of PEDV. Nucleotide sequence accession number. The genome sequence of PEDV strain JS2008 was deposited in GenBank under accession number KC109141. This work was supported by the Special Fund for Independent Innovation of Agricultural Science and Technology in Jiangsu Province [CX (12) 3076].</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 3.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> A realistic two-strain model for MERS-CoV infection uncovers the high risk for epidemic propagation </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Sardar, Tridip; Ghosh, Indrajit; Rodó, Xavier; Chattopadhyay, Joydev </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-14 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7046297/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid17');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid27');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid37');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid17\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) causes severe acute respiratory illness with a case fatality rate (CFR) of 35,5%. The highest number of MERS-CoV cases are from Saudi-Arabia, the major worldwide hotspot for this disease. In the absence of neither effective treatment nor a ready-to-use vaccine and with yet an incomplete understanding of its<span style=\"color: #ff00ff;\"> Epidem</span>iological cycle, prevention and containment measures can be derived from mathematical models of disease<span style=\"color: #ff00ff;\"> Epidem</span>iology. We constructed 2-strain models to predict past<span style=\"color: #ff00ff;\"> Outbreak</span>s in the<span style=\"color: #ff0000;\"> Interv</span>al 2012–2016 and derive key<span style=\"color: #ff00ff;\"> Epidem</span>iological information for Macca, Madina and Riyadh. We approached variability in infection through three different disease incidence functions capturing social behavior in response to an<span style=\"color: #ff00ff;\"> Epidem</span>ic (e.g. Bilinear, BL; Non-monotone, NM; and Saturated, SAT models). The best model combination successfully anticipated the total number of MERS-CoV clinical cases for the 2015–2016 season and accurately predicted both the number of cases at the peak of seasonal incidence and the overall shape of the<span style=\"color: #ff00ff;\"> Epidem</span>ic cycle. The evolution in the basic<span style=\"color: #ff0000;\"> Reproduct</span>ion number (R(0)) warns that MERS-CoV may easily take an<span style=\"color: #ff00ff;\"> Epidem</span>ic form. The best model correctly captures this feature, indicating a high<span style=\"color: #ff00ff;\"> Epidem</span>ic risk (1≤R(0)≤2,5) in Riyadh and Macca and confirming the alleged co-circulation of more than one strain. Accurate predictions of the future MERS-CoV peak week, as well as the number of cases at the peak are now possible. These results indicate public health agencies should be aware that measures for strict containment are urgently needed before new<span style=\"color: #ff00ff;\"> Epidem</span>ics take off in the region.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid27\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The aim of the present study was to determine the full-length genome of the first HCoV OC43 isolated from Mexico, which was obtained from a viral metagenomic analysis of a nasal<span style=\"color: #ff00ff;\"> Wash</span>ing from a child who required hospitalization due to clinical or radiological signs of pneumonia of unknown origin, negative for commonly associated respiratory viruses and bacteria, using the xTAG Bioplex respiratory viral panel and multiplex PCR Seeplex Pneumobacter ACE detection kit, respectively (5) .</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid37\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">coronavirus (HCoV) OC43 belongs to the Betacoronavirus genus of the Coronaviridae and its genome is formed by a positivesense, single-stranded RNA of ca. 31.5 kb (1) . HCoV OC43 and HCoV 229E are responsible for one-third of all common colds, infecting all age groups (2), but there have been reports of a more severe lower respiratory tract involvement (3, 4) . The aim of the present study was to determine the full-length genome of the first HCoV OC43 isolated from Mexico, which was obtained from a viral metagenomic analysis of a nasal<span style=\"color: #ff00ff;\"> Wash</span>ing from a child who required hospitalization due to clinical or radiological signs of pneumonia of unknown origin, negative for commonly associated respiratory viruses and bacteria, using the xTAG Bioplex respiratory viral panel and multiplex PCR Seeplex Pneumobacter ACE detection kit, respectively (5) . Nucleic acids from the sample were extracted with the PureLink Viral RNA/DNA kit. Before extraction, the sample was treated with Turbo DNase and RNase, and a random-primer amplification of the genetic material was performed as previously described (5) . The sequencing was carried out with the Illumina Genome Analyzer IIx single-end platform. A total of 12,330,418 reads were generated and 1,657,101 (13.44%) were mapped to the consensus sequence of HCoV OC43. Mapping was accomplished by SMALT v.0.7.6 (6) with default parameters (only Ϫy set to 0.8). The whole viral genome sequence was obtained with an average coverage of 3,368ϫ. The same genome sequence was obtained by de novo assembly using Velvet (7) . The complete Mexican HCoV OC43 genome was composed of 30,712 nucleotides, with an average GϩC content of 35.51%. The strain shows the typical OC43 organization, with the following 5= untranslated region (UTR) (nt 1 to 199), genes: orf1ab (nt 200 to Accession number(s). The complete genome sequence of the Mexican OC43 strain was deposited at GenBank under the accession number KX344031.</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 4.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Rapid detection of respiratory organisms with FilmArray respiratory panel and its impact on clinical decisions in Shanghai, China, 2016‐2018 </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Qian, Yiyi; Ai, Jingwen; Wu, Jing; Yu, Shenglei; Cui, Peng; Gao, Yan; Jin, Jialin; Weng, Xinhua; Zhang, Wenhong </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2019-12-01 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7040966/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid18');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid28');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid38');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid18\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">BACKGROUND: In this study, we evaluated the diagnostic potential and clinical impact of an automated multiplex PCR platform (the FilmArray Respiratory Panel; FA‐RP), specially designed for pathogen detection in respiratory tract infections in<span style=\"color: #ff00ff;\"> Ad</span>ults with unexplained pneumonia (UP). METHODS: A total of 112 UP patients in Shanghai, China, were enrolled prospectively and assessed using the FA‐RP from October 2016 to March 2018. We examined the test results and their influence on clinical decisions. Furthermore, as a control group, we retrospectively obtained the clinical data of 70 UP patients between October 2014 and March 2016 (before the FA‐RP was available). The two patient groups were compared with respect to factors, including general antimicrobial use and defined daily dose (DDD) numbers. RESULTS: Between October 2016 and March 2018, the positive rate obtained using FA‐RP for UP was 76.8%. The primary pathogens in<span style=\"color: #ff00ff;\"> Ad</span>ults with UP were Influenza A/B (47.3%, 53/112). Compared with the patients before FA‐RP was available, patients who underwent FA‐RP testing had higher rates of antiviral drug use and antibiotic de‐escalation during clinical treatment. FA‐RP significantly decreased the total DDDs of antibiotic or antifungal drugs DDDs by 7 days after<span style=\"color: #ff00ff;\"> Ad</span>mission (10.6 ± 2.5 vs 14.1 ± 8.8, P < .01). CONCLUSIONS: The FA‐RP is a rapid and sensitive nucleic acid amplification test method for UP diagnosis in<span style=\"color: #ff00ff;\"> Ad</span>ults. The application of FA‐RP may lead to a more accurately targeted antimicrobial treatment and reduced use of antibiotic/antifungal drugs.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid28\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">After the severe acute respiratory syndrome<span style=\"color: #ff00ff;\"> Outbreak</span>, in order to track similarly highly contagious and severe lower respiratory illness with pneumonia symptoms, the Chinese government has paid great attention to unexplained pneumonia (UP), a phrase coined by the Chinese Human Unexplained Pneumonia Surveillance Network (CHUPSN) and published in 2004.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">When we compared antimicrobial use between the<span style=\"color: #ff0000;\"> Interv</span>ention and control groups, we found that the two groups were comparable for the most baseline characteristics (Table 3) .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Compared with the control group, the<span style=\"color: #ff0000;\"> Interv</span>ention group showed no significant decrease of antimicrobial DDDs (control group 14.3 ± 9.6 vs<span style=\"color: #ff0000;\"> Interv</span>ention group 13.8 ± 4.6, P > .05).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">In the present study, we found that 53.8% (60/112) of the patients in the FA-RP group received antiviral treatment, which is significantly higher than that in the control group (14.3%, 10/70; randomized controlled trial discovered that use of the FA-RP was not associated with a reduction in the overall<span style=\"color: #ff00ff;\"> Durat</span>ion of antibiotic usage.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">As a result,<span style=\"color: #ff0000;\"> Interv</span>ention group unexpectedly showed longer<span style=\"color: #ff00ff;\"> Durat</span>ion of hospital stay than the control group (12.1 vs 7.6 days, P = .001).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">One reasonable explanation is that subjects in the<span style=\"color: #ff0000;\"> Interv</span>ention group had more severe cases of pneumonia than those in the control group (higher PSI, This present study is one of the several that have examined the application of FA-RP in Asia.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">[19] [20] [21] [22] [23] [24] Compared with other published studies, our study focused on<span style=\"color: #ff00ff;\"> Ad</span>ults with more severe pneumonia conditions, and to the best of our knowledge, it is the only study that Our study does, however, have a few limitations.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 8:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Notably, the sample size was limited, and the study was<span style=\"color: #ff00ff;\"> Conduc</span>ted only at a single location, which may thus limit the general applicability of the findings.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 9:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">In<span style=\"color: #ff00ff;\"> Ad</span>dition, the study included two cohorts from different time<span style=\"color: #ff00ff;\"> Period</span>s; hence, there may have been differences in factors that influenced clinician behavior between the two<span style=\"color: #ff00ff;\"> Period</span>s that were not accounted in our analysis.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 10:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Furthermore, the incidence of various respiratory viruses may have varied across the two<span style=\"color: #ff00ff;\"> Period</span>s that may thus limit the validity of a direct comparison of clinical outcomes.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid38\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">After the severe acute respiratory syndrome<span style=\"color: #ff00ff;\"> Outbreak</span>, in order to track similarly highly contagious and severe lower respiratory illness with pneumonia symptoms, the Chinese government has paid great attention to unexplained pneumonia (UP), a phrase coined by the Chinese Human Unexplained Pneumonia Surveillance Network (CHUPSN) and published in 2004. UP refers to pneumonia, which could be life-threatening, characterized by rapid progression, a normal white blood cell count range, and poor response to antibiotics. 1 Although the common pathogens causing UP are viruses or atypical pathogens, their overlapping clinical manifestations can impede the ability of clinicians to directly diagnose the causative pathogens, which may lead to unnecessary antimicrobial usage. Thus, rapid and precise diagnosis of the causative agents is critical for the prompt management of UP. To date, culture-based and other traditional methods have been used to help the clinical approach to diagnose UP; however, these methods have certain disadvantages. For example, rapid antigen testing has only a 40%-100% sensitivity for<span style=\"color: #ff00ff;\"> Ad</span>ult influenza and is dependent on the use of specific scientific techniques, whereas culturing viruses can take up to 10 days. 2 In recent years, there has been an increase in the use of nucleic acid amplification test (NAAT)-based methods for the detection of viral pathogens due to their excellent diagnostic ability in identifying a broad spectrum of pathogens. 3  The prospective cohort was enrolled in Shanghai, China, between  Within 24<span style=\"color: #ff00ff;\"> Hour</span>s of patient<span style=\"color: #ff00ff;\"> Ad</span>mission, respiratory tract specimens were collected, including bronchoalveolar lavage fluid (BALF: among those patients who gave consent to undergo bronchoscopy), sputum (if expectoration was evident), and nasopharyngeal swab (NPS, if no bronchoscopy was performed or no expectoration sample was collected). Notably, lower respiratory tract specimens were given higher priority (BALF > sputum > NPS) during the study. As soon as possible after sampling, the specimens were transferred to the Laboratory of Infectious Diseases Research Institute, Huashan Hospital, Shanghai. Professional pulmonary physicians oversaw BALF sampling, while the collection of NPS and sputum was performed by clinical physicians per the clinical standard operating procedure. Notably, those samples of NPS and BALF for which the viscosity was low required no further sample preparation before FilmArray panel testing, whereas viscous samples, such as sputum, were pre-treated using the previously described dunk and swirl method. 5 The remaining specimens were stored in a −80°C freezer for further use. In this study, we used version 1.7 of the FA-RP. It is an automated multiplex PCR detection platform designed to detect 20 common pathogens causing respiratory infection and has a single specimen turnaround time of <2<span style=\"color: #ff00ff;\"> Hour</span>s. Tests using the panel were performed following the manufacturer's instructions. For specimens that the FA-RP reported as \"Influenza A, not sub-  For both patient groups, traditional standard-of-care laboratory diagnostic tests for respiratory infections were performed according to the requests of physicians, including smears, cultures, and serological tests. The serological test in our study was the PNEUMOSLIDE IgM (Vircell) commercial kit based on indirect immunofluorescence. In this study, clinical data, including antimicrobial use, and<span style=\"color: #ff00ff;\"> Ad</span>justment or de-escalation of antimicrobial therapy, were compared between the FA-RP group and retrospective control group. Turnaround time was defined as the length of time from sample collection to receiving the FA-RP results. Antimicrobial use was quantified as the defined daily dose (DDD)) number, calculated as ∑ (Drug Total Dosage (g)/Drug DDD index). 6<span style=\"color: #ff00ff;\"> Ad</span>justment of antimicrobial therapy was defined as a change in antimicrobial regimen immediately after receiving the FA-RP results. Overview of Patient Enrollment Workflow. A total of 112 patients selected prospectively were submitted to FA-RP analyses and are referred to as the \"FA-RP group.\" Those who were not lost to follow-up and had complete data are referred to as the \"intervention group.\" Comparison was made between it and the historical control group The Mann-Whitney U test was used for the comparison of quantitative data. spss 20.0 (IBM) was used for data analyses. We used Graphpad 6.0 for drawing graphs. The Ethics Committee of the Huashan Hospital, Shanghai, approved the study (KY2013-251; July 2, 2013). All participants gave their written informed consent. Table 1 . Mixed infections of multiple pathogens were detected in six patients: Influenza virus/RSV (n = 1), Influenza virus/human metapneumovirus (n = 1), Influenza virus/parainfluenza-2 (n = 1), M pneumoniae/human rhinovirus/enterovirus (n = 1), human rhinovirus/ enterovirus/parainfluenza-2 (n = 1), and<span style=\"color: #ff00ff;\"> Ad</span>enovirus/coronavirus HKU1/coronavirus OC43 (n = 1). For 13 samples, the FA-RP results were \"Influenza A/not subtyped.\" In these cases, further subtyping was performed using real-time RT-PCR. As a consequence, Influenza A virus subtype H7N9 was detected in 11 samples and H1N1-2009 in the remaining two samples. As a consequence of the findings of the FA-FP regarding Influenza A, 15 patients were isolated in a negative pressure ward or single ward, including seven highly pathogenic H7N9 cases. In contrast, in the control group, no comparable isolation measures were undertaken. When we compared antimicrobial use between the<span style=\"color: #ff0000;\"> Interv</span>ention and control groups, we found that the two groups were comparable for the most baseline characteristics (Table 3) . Compared with the control group, the<span style=\"color: #ff0000;\"> Interv</span>ention group showed no significant decrease of antimicrobial DDDs (control group 14.3 ± 9.6 vs<span style=\"color: #ff0000;\"> Interv</span>ention group 13.8 ± 4.6, P > .05). However, after excluding antiviral drugs, we detected significantly lower antibiotic/antifungal DDDs  In this study, we applied FA-RP on diagnosis of UP, which is a certain type of pneumonia causing mainly by viruses and atypical pathogens. We found that the FA-RP detected pathogens in 76.8% (86/112) of all specimens, which is higher than that the detection rate obtained when this technique was applied for general community-acquired pneumonia (38.6%, 49/127). 7 These results, therefore, show that when applied clinically for UP, the FA-RP would provide a higher probability of identifying causal pathogenic microorganisms. In the present study, we found that 53.8% (60/112) of the patients in the FA-RP group received antiviral treatment, which is significantly higher than that in the control group (14.3%, 10/70; randomized controlled trial discovered that use of the FA-RP was not associated with a reduction in the overall<span style=\"color: #ff00ff;\"> Durat</span>ion of antibiotic usage. 18 Nevertheless, a higher number of patients in the FA-RP group received single doses or brief courses of antibiotics than those in the control group. Given these inconclusive findings, further studies are required concerning the impact of FA-RP on the clinical decisions regarding pneumonia. We compared the<span style=\"color: #ff00ff;\"> Durat</span>ion of hospital stay between the two groups. As a result,<span style=\"color: #ff0000;\"> Interv</span>ention group unexpectedly showed longer<span style=\"color: #ff00ff;\"> Durat</span>ion of hospital stay than the control group (12.1 vs 7.6 days, P = .001). One reasonable explanation is that subjects in the<span style=\"color: #ff0000;\"> Interv</span>ention group had more severe cases of pneumonia than those in the control group (higher PSI, This present study is one of the several that have examined the application of FA-RP in Asia. [19] [20] [21] [22] [23] [24] Compared with other published studies, our study focused on<span style=\"color: #ff00ff;\"> Ad</span>ults with more severe pneumonia conditions, and to the best of our knowledge, it is the only study that Our study does, however, have a few limitations. Notably, the sample size was limited, and the study was<span style=\"color: #ff00ff;\"> Conduc</span>ted only at a single location, which may thus limit the general applicability of the findings. In<span style=\"color: #ff00ff;\"> Ad</span>dition, the study included two cohorts from different time<span style=\"color: #ff00ff;\"> Period</span>s; hence, there may have been differences in factors that influenced clinician behavior between the two<span style=\"color: #ff00ff;\"> Period</span>s that were not accounted in our analysis. Furthermore, the incidence of various respiratory viruses may have varied across the two<span style=\"color: #ff00ff;\"> Period</span>s that may thus limit the validity of a direct comparison of clinical outcomes. In conclusion, the FA-RP is a rapid and sensitive NAAT method that can be used for the diagnosis of UP in<span style=\"color: #ff00ff;\"> Ad</span>ults. In the two recent years in Shanghai, the most common pathogen causing UP was identified as the Influenza virus. However, in this regard, it should be taken into consideration that the current version of the FA-RP is unable to directly detect H7N9. Nevertheless, the results obtained in this study show that the FA-RP may make a valuable contribution to clinical decision making and facilitate the reasonable use of antibiotics. We would like to thank Yinlei Zhang, Tracy Wang, Xuyang Wang, and Xinchang Chen for their substantial contributions to this study. Yiyi Qian https://orcid.org/0000-0002-0714-3894</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 5.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Yonghua Wu </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-04 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1101/2020.03.04.975995 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid19');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid29');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid39');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid19\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In<span style=\"color: #ff00ff;\"> Ad</span>dition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong<span style=\"color: #ff00ff;\"> Ad</span>aptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their<span style=\"color: #ff00ff;\"> Ad</span>aptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid29\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Makerere University Walter Reed Project (MUWRP) together with the Ugandan<span style=\"color: #ff00ff;\"> Min</span>istry of Health and partners undertook surveillance to determine viral causes of influenza-like illness in Uganda.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Of the 2,062 influenza negative samples, results indicated positivity for the following organisms;<span style=\"color: #ff00ff;\"> Ad</span>enoviruses (9.4%), respiratory syncytial B (7.3%), parainfluenza-3 (4.5%), parainfluenza-1 (4.3%), respiratory syncytial A (3.5%), human bocavirus (1.7%), human metapneumovirus (1.7%), human coronavirus (1.5%), parainfluenza-4 (1.4%) and parainfluenza-2 (0.9%) by PCR.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid39\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">Respiratory pathogens continue to present an ever increasing threat to public health (1,2). Influenza, Respiratory syncytial virus, human metapneumovirus and other respiratory viruses are major etiological agents for influenza like illnesses (ILI) (3-5). Establishment of viral causes of ILI is critical for prevention and mitigation strategies to disease threats. Makerere University Walter Reed Project (MUWRP) together with the Ugandan<span style=\"color: #ff00ff;\"> Min</span>istry of Health and partners undertook surveillance to determine viral causes of influenza-like illness in Uganda. From 2008, MUWRP established hospital-based sentinel sites for surveillance activities. A total of five hospital-based sites were established, where patients aged 6 months or older presenting with ILI were enrolled. Consents were obtained as required, and a throat and/ or nasopharyngeal swab collected. Samples were screened by PCR for viral causes. From October 2008 to March 2017 a total of 9,472 participants were enrolled in the study from five hospital-based surveillance sentinel sites. Majority of participants were children under 5 years n= 8,169 (86.2%). 615 (6.5%) samples tested positive for influenza A, while 385 (4.1%) tested positive for influenza B viruses and 10 (0.1%) were co-infections between influenza A and B. Of the 2,062 influenza negative samples, results indicated positivity for the following organisms;<span style=\"color: #ff00ff;\"> Ad</span>enoviruses (9.4%), respiratory syncytial B (7.3%), parainfluenza-3 (4.5%), parainfluenza-1 (4.3%), respiratory syncytial A (3.5%), human bocavirus (1.7%), human metapneumovirus (1.7%), human coronavirus (1.5%), parainfluenza-4 (1.4%) and parainfluenza-2 (0.9%) by PCR. Influenza viruses account for about 11% of the causes of influenza like illness, with influenza A being the dominant type. Among the other viral causes of ILI,<span style=\"color: #ff00ff;\"> Ad</span>enoviruses were the most dominant. Detection of other viral causes of ILI is an indication of the public health threats posed by respiratory pathogens. Influenza; Influenza Like Illness; Surveillance</strong></p>\n",
       "            </div>\n",
       "            "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "search_transmission = \"\"\"\"Transmission dynamics of the virus, including the basic reproductive number, incubation period, serial interval, modes of transmission and environmental factors\"\"\"\n",
    "tags_transmission = \"\"\"transmission dynamics reproductive incubation serial interval environmental\"\"\"\n",
    "\n",
    "searching.SearchDisplayPaper(search=search_transmission,  pubdate_in=\"2019-12-01 2020-04-01\", topn=5,  topn_sent=10, tags=tags_transmission)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "                <script>\n",
       "                function visibilite(thingId)\n",
       "                {\n",
       "                var targetElement;\n",
       "                targetElement = document.getElementById(thingId) ;\n",
       "                if (targetElement.style.display == \"none\")\n",
       "                {\n",
       "                targetElement.style.display = \"\" ;\n",
       "                } else {\n",
       "                targetElement.style.display = \"none\" ;\n",
       "                }\n",
       "                }\n",
       "                </script>\n",
       "                </body>\n",
       "                \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 1.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> A realistic two-strain model for MERS-CoV infection uncovers the high risk for epidemic propagation </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Sardar, Tridip; Ghosh, Indrajit; Rodó, Xavier; Chattopadhyay, Joydev </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-14 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7046297/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid110');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid210');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid310');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid110\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) causes<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"> Acut</span>e respiratory<span style=\"color: #ff00ff;\"> Ill</span>ness with a case<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Fatal</span></span>ity rate (CFR) of 35,5%. The<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>est</span> number of MERS-CoV cases are from Saudi-Arabia, the major worldwide hotspot for this<span style=\"color: #ff0000;\"> Diseas</span>e. In the absence of neither effective treatment nor a ready-to-use vaccine and with yet an incomplete understanding of its epidemiological cycle, prevention and containment measures can be derived from mathematical models of<span style=\"color: #ff0000;\"> Diseas</span>e epidemiology. We constructed 2-strain models to predict past outbreaks in the interval 2012–2016 and derive key epidemiological information for Macca, Madina and Riyadh. We approached variability in infection through three different<span style=\"color: #ff0000;\"> Diseas</span>e<span style=\"color: #ff00ff;\"> Incid</span>ence functions capturing social behavior in response to an epidemic (e.g. Bilinear, BL; Non-monotone, NM; and Saturated, SAT models). The best model combination successfully anticipated the total number of MERS-CoV clinical cases for the 2015–2016 season and accurately predicted both the number of cases at the peak of seasonal<span style=\"color: #ff00ff;\"> Incid</span>ence and the overall shape of the epidemic cycle. The evolution in the basic reproduction number (R(0)) warns that MERS-CoV may easily take an epidemic form. The best model correctly captures this feature, indicating a<span style=\"color: #ff0000;\"> High</span> epidemic<span style=\"color: #ff0000;\"> Risk</span> (1≤R(0)≤2,5) in Riyadh and Macca and confirming the alleged co-circulation of more than one strain. Accurate predictions of the future MERS-CoV peak week, as well as the number of cases at the peak are now possible. These results indicate public health agencies should be<span style=\"color: #ff00ff;\"> Awar</span>e that measures for strict containment are urgently needed before new epidemics take off in the region.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid210\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The aim of the present study was to determine the full-length genome of the first HCoV OC43 isolated from Mexico, which was obtained from a viral metagenomic analysis of a nasal washing from a child who required<span style=\"color: #ff0000;\"> Hospit</span>alization due to clinical or radiological signs of pneumonia of unknown origin, negative for commonly associated respiratory viruses and bacteria, using the xTAG Bioplex respiratory viral panel and multiplex PCR Seeplex Pneumobacter ACE detection kit, respectively (5) .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The sequencing was carried out with the<span style=\"color: #ff00ff;\"> Ill</span>umina Genome Analyzer IIx single-end platform.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">HCoV OC43 and HCoV 229E are responsible for one-third of all common colds, infecting all age groups (2), but there have been reports of a more<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Low</span>er</span> respiratory tract involvement (3, 4) .</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid310\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">coronavirus (HCoV) OC43 belongs to the Betacoronavirus genus of the Coronaviridae and its genome is formed by a positivesense, single-stranded RNA of ca. 31.5 kb (1) . HCoV OC43 and HCoV 229E are responsible for one-third of all common colds, infecting all age groups (2), but there have been reports of a more<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Low</span>er</span> respiratory tract involvement (3, 4) . The aim of the present study was to determine the full-length genome of the first HCoV OC43 isolated from Mexico, which was obtained from a viral metagenomic analysis of a nasal washing from a child who required<span style=\"color: #ff0000;\"> Hospit</span>alization due to clinical or radiological signs of pneumonia of unknown origin, negative for commonly associated respiratory viruses and bacteria, using the xTAG Bioplex respiratory viral panel and multiplex PCR Seeplex Pneumobacter ACE detection kit, respectively (5) . Nucleic acids from the sample were extracted with the PureLink Viral RNA/DNA kit. Before extraction, the sample was treated with Turbo DNase and RNase, and a random-primer amplification of the genetic material was performed as previously described (5) . The sequencing was carried out with the<span style=\"color: #ff00ff;\"> Ill</span>umina Genome Analyzer IIx single-end platform. A total of 12,330,418 reads were generated and 1,657,101 (13.44%) were mapped to the consensus sequence of HCoV OC43. Mapping was accomplished by SMALT v.0.7.6 (6) with default parameters (only Ϫy set to 0.8). The whole viral genome sequence was obtained with an average coverage of 3,368ϫ. The same genome sequence was obtained by de novo assembly using Velvet (7) . The complete Mexican HCoV OC43 genome was composed of 30,712 nucleotides, with an average GϩC content of 35.51%. The strain shows the typical OC43 organization, with the following 5= untranslated region (UTR) (nt 1 to 199), genes: orf1ab (nt 200 to Accession number(s). The complete genome sequence of the Mexican OC43 strain was deposited at GenBank under the accession number KX344031.</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 2.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Increasing Host Cellular Receptor—Angiotensin-Converting Enzyme 2 (ACE2) Expression by Coronavirus may Facilitate 2019-nCoV Infection </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Pei-Hui Wang; Yun Cheng </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-27 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1101/2020.02.24.963348 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid111');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid211');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid311');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid111\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">AbstractThe ongoing outbreak of a new coronavirus (2019-nCoV) causes an epidemic of<span style=\"color: #ff00ff;\"> Acut</span>e respiratory syndrome in humans. 2019-nCoV rapidly spread to national regions and multiple other countries, thus, pose a<span style=\"color: #ff00ff;\"> Serious</span> threat to public health. Recent studies show that spike (S) proteins of 2019-nCoV and SARS-CoV may use the same host cell receptor called angiotensin-converting enzyme 2 (ACE2) for entering into host cells. The affinity between ACE2 and 2019-nCoV S is much<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>er</span> than ACE2 binding to SARS-CoV S protein, explaining that why 2019-nCoV seems to be more readily transmitted from the human to human. Here, we reported that ACE2 can be significantly upregulated after infection of various viruses including SARS-CoV and MERS-CoV. Basing on findings here, we propose that coronavirus infection can positively induce its cellular entry receptor to accelerate their replication and spread, thus drugs targeting ACE2 expression may be prepared for the future emerging infectious<span style=\"color: #ff0000;\"> Diseas</span>es caused by this cluster of viruses.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid211\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Many<span style=\"color: #ff00ff;\"> Patient</span>s in each group had a decreased level of EC, and 0 EC was detected in 65.63%<span style=\"color: #ff00ff;\"> Patient</span>s in critical group, 55.88% in<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e group, and 25.93% in<span style=\"color: #ff00ff;\"> Mild</span> group.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">According to the news, a research team from First Affiliated<span style=\"color: #ff0000;\"> Hospit</span>al of University of Science and Technology of China has used IL-6 receptor recombinant monoclonal antibody, Tozhu monoclonal antibody, in 14 critically or<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>ely<span style=\"color: #ff00ff;\"> Ill</span> COVID-19<span style=\"color: #ff00ff;\"> Patient</span>s [10] .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Presumably, lymphocyte deposition in lung tissue might contribute to lymphocyte<span style=\"color: #ff00ff;\"> Reduc</span>tion in peripheral blood.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Pathological examination found lymphocyte-dominated mononuclear cell infiltration in interstitial pulmonary tissue, CD4+ and CD8+ T cells were significantly<span style=\"color: #ff00ff;\"> Reduc</span>ed in peripheral blood cells, but a<span style=\"color: #ff0000;\"> High</span> ratio of CD38+ (CD4 3.47%) and HLA-DR+ (CD8 39.4%) T cells [9] , which suggested excessive activation of pro-inflammatory cells.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">With age >67.5 years, IL2R >793.5U/mL, CRP >30.7ng/mL, ferroprotein >2252μg/L, WBC >9.5*10^9/L or NC>7.305*10^9/L, progress to critical<span style=\"color: #ff00ff;\"> Ill</span>ness should be closely observed and prevented.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">IL-8 levels in<span style=\"color: #ff00ff;\"> Mild</span> and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"> Patient</span>s were normal, and for<span style=\"color: #ff00ff;\"> Patient</span>s with IL-8 >62 pg/mL, more attention need to avoid the<span style=\"color: #ff0000;\"> Diseas</span>e progression.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">For eight<span style=\"color: #ff00ff;\"> Patient</span>s who were deceased in critical group, IL-6 level was >100pg/mL in three persons.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 8:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">In addition, IL2R levels, ferroprotein levels, PCT levels, and EC counts were also related to the<span style=\"color: #ff0000;\"> Diseas</span>e<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>ity.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 9:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">In this study, a retrospective analysis was conducted about cytokines and other inflammatory indicators of 100<span style=\"color: #ff00ff;\"> Patient</span>s (including 66<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e or critical<span style=\"color: #ff00ff;\"> Patient</span>s), and more indicators were expected to better identify critical<span style=\"color: #ff00ff;\"> Patient</span>s and help clinical decision-making.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 10:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Clinically, many<span style=\"color: #ff00ff;\"> Patient</span>s appeared short-term progressive aggravation, scholars speculated that \"inflammatory storms\" occurred, namely, overreaction of cytokine.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid311\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">Critical<span style=\"color: #ff00ff;\"> Patient</span>s should have any of the following conditions: 1. Respiratory failure occurs and mechanical ventilation required; 2. Shock occurs; 3. Combining other organ failure and requiring treatment in<span style=\"color: #ff00ff;\"> Icu</span>. All tests were completed in the clinical laboratory in Tongji<span style=\"color: #ff0000;\"> Hospit</span>al. Interleukin-1β (IL-1β), interleukin-2 receptor (IL2R), interleukin-8 (IL-8), IL-10 and TNFα were detected by Siemens chemiluminescence method and IL-6 were detected by Roche electrochemiluminescence method according to the manufacturer's instruction. The ultrasensitive CRP regent was provided by Nippon Denkasei Co., Ltd, and CRP was detected by immunoturbidimetry method. Ferroprotein was detected by Roche granule-enhanced immune turbidimetry. Procalcitonin (PCT) was detected by Roche electrochemiluminescence method. ESR was measured by Westergren's international standard method. Peripheral blood cell was detected by fluorescence staining flow cytometry, and we analyzed the differences of white blood cell (WBC), NC, LC and eosinophils (EC) among three groups. For data of normal distribution (IL2R, ESR, ferroprotein, WBC and NC), comparisons among critical,<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e and<span style=\"color: #ff00ff;\"> Mild</span> groups were analyzed by ANOVA analysis. With different homogeneity of variance, the pairwise comparison between groups was performed using the Bonferroni test or Dunnett's T3 test. For non-normal distribution (CRP and LC), the data were conversed using square root, followed by ANOVA analysis and pairwise comparison. With data below the detectable limit, including IL-1β, IL-6, IL-8, IL-10, TNFα, PCT, and EC, the data were ranked<span style=\"color: #ff00ff;\"> Referr</span>ing to the reference range and value rank (Table 1) ; then non-parametric Kruskal-Wallis test were performed. In correlation analysis, Spearman correlation coefficient was used for the variables of normal distribution, Pearson correlation coefficient for those of skewed distribution, and Kendall's tau-b correlation coefficient for ranked data. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.25.20025643 doi: medRxiv preprint<span style=\"color: #ff00ff;\"> Patient</span>s were <31 pg/mL, three ranked grades were set, namely, <31 pg/mL, 31-62 pg/mL and >62 pg/mL. Figure 1D and Table 2 showed that all IL-8 levels in<span style=\"color: #ff00ff;\"> Mild</span> and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"> Patient</span>s were within the reference range (<62 pg/mL), and there were significant differences between critical and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"> Patient</span>s or critical and<span style=\"color: #ff00ff;\"> Mild</span> group. With regard to IL-10 and TNFα, there was significant difference between<span style=\"color: #ff00ff;\"> Mild</span> and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e or<span style=\"color: #ff00ff;\"> Mild</span> and critical<span style=\"color: #ff00ff;\"> Patient</span>s, and no significant difference was found between<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e and critical group ( Figure 1E -F). With regard to cytokine IL-1β, no significant difference was found among three groups, and the data were not shown. In addition, levels of CRP, ferroprotein and PCT were statistically different among<span style=\"color: #ff00ff;\"> Mild</span>,<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e and critical<span style=\"color: #ff00ff;\"> Patient</span>s ( Figure Figure 2D ) indicated the best cut-off point was 2252μg/L with a specificity of 96.6% and a sensitivity of 52.2%. There was no statistical difference on ESR among three groups ( Figure 3D ). In peripheral blood cell analysis, there were significant differences in WBC count between the critical and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e groups or critical and<span style=\"color: #ff00ff;\"> Mild</span> groups, and no significance was found between the<span style=\"color: #ff00ff;\"> Mild</span> and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e groups ( Figure 4A ). ROC curve of WBC (AUC =0.838, p =0.000, Figure 2E ) suggested the best cut-off point was 7.92*10^9/L with 77% specificity and 78.1% sensitivity. If the detected values of WBC are above the upper reference limit, it often indicates infection. Because WBC count is influenced by therapy, clinically the upper limit of reference range of WBC, namely 9.5*10^9/L, seems more meaningful. NC was significantly different among the three groups ( Figure 4B ), and the average was 10.80*10^9/L in critical group, 5.47*10^9/L in<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e group and 3.53 *10^9/L in<span style=\"color: #ff00ff;\"> Mild</span> group. ROC curve of NC (AUC=0.814, P=0.000, Figure 2F ) indicated the best cut-off point was 7.305*10^9/L with 85.2% specificity and 75% sensitivity. There was . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.25.20025643 doi: medRxiv preprint significantly difference about LC between critical and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e group or critical and<span style=\"color: #ff00ff;\"> Mild</span> group ( Figure 4C ). Many<span style=\"color: #ff00ff;\"> Patient</span>s in each group had a decreased level of EC, and 0 EC was detected in 65.63%<span style=\"color: #ff00ff;\"> Patient</span>s in critical group, 55.88% in<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e group, and 25.93% in<span style=\"color: #ff00ff;\"> Mild</span> group. There were significant differences between<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e and<span style=\"color: #ff00ff;\"> Mild</span> or critical and<span style=\"color: #ff00ff;\"> Mild</span><span style=\"color: #ff00ff;\"> Patient</span>s (Table 2, Figure 4D ). Table 3 (Table 3 ). There was no significant correlation on IL-1β, gender and ESR. The new coronavirus (COVID-19) pneumonia outbreaking at the end of 2019 is<span style=\"color: #ff0000;\"> High</span>ly contagious, with a crude<span style=\"color: #ff00ff;\"> Mortal</span>ity rate of about 2.3% [1] . Currently, more than 74677<span style=\"color: #ff00ff;\"> Patient</span>s have been reported in the Mainland of China. About 80.9%<span style=\"color: #ff00ff;\"> Patient</span>s are with<span style=\"color: #ff00ff;\"> Mild</span> to moderate<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>ity [1] , and with a better prognosis. However, for<span style=\"color: #ff00ff;\"> Patient</span>s developing into<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e or critical levels, the<span style=\"color: #ff00ff;\"> Mortal</span>ity rate was significantly<span style=\"color: #ff00ff;\"> Increas</span>ed, and crude<span style=\"color: #ff00ff;\"> Mortal</span>ity rate reached 49% in critical<span style=\"color: #ff00ff;\"> Patient</span>s [1] . It plays key roles to identify<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e and critical<span style=\"color: #ff00ff;\"> Patient</span>s even earlier, aiming to improve the recovery rate and<span style=\"color: #ff00ff;\"> Reduc</span>e<span style=\"color: #ff00ff;\"> Mortal</span>ity. Clinically, many<span style=\"color: #ff00ff;\"> Patient</span>s appeared short-term progressive aggravation, scholars speculated that \"inflammatory storms\" occurred, namely, overreaction of cytokine. In a preprint article in medrxiv, the lymphocyte subsets and cytokines of 123<span style=\"color: #ff00ff;\"> Patient</span>s (102<span style=\"color: #ff00ff;\"> Mild</span> and 21<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"> Patient</span>s) were analyzed and the researchers found the numbers of CD4+ T cells and CD8+ T cells decreased, and the levels of IL-6 and IL-10<span style=\"color: #ff00ff;\"> Increas</span>ed in<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e cases [7] . In this study, a retrospective analysis was conducted about cytokines and other inflammatory indicators of 100<span style=\"color: #ff00ff;\"> Patient</span>s (including 66<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e or critical<span style=\"color: #ff00ff;\"> Patient</span>s), and more indicators were expected to better identify critical<span style=\"color: #ff00ff;\"> Patient</span>s and help clinical decision-making. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02. 25.20025643 doi: medRxiv preprint In accordance with Wan S's [7] and Liu J's [8] study, this study also found that the levels of IL-6 and IL-10 were associated with the<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>ity of COVID-19 pneumonia; similar to Chaolin Huang's study [4] , TNFα concentration, NC count and LC count were also found correlated with<span style=\"color: #ff0000;\"> Diseas</span>e<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>ity. In addition, IL2R levels, ferroprotein levels, PCT levels, and EC counts were also related to the<span style=\"color: #ff0000;\"> Diseas</span>e<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>ity. Besides, we also found the IL-6 levels in<span style=\"color: #ff00ff;\"> Mild</span><span style=\"color: #ff00ff;\"> Patient</span>s were<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Low</span>er</span> than 100pg/mL. For eight<span style=\"color: #ff00ff;\"> Patient</span>s who were deceased in critical group, IL-6 level was >100pg/mL in three persons. IL-6 >100pg/mL might represent the emergence of \"inflammatory storm\". IL-8 levels in<span style=\"color: #ff00ff;\"> Mild</span> and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"> Patient</span>s were normal, and for<span style=\"color: #ff00ff;\"> Patient</span>s with IL-8 >62 pg/mL, more attention need to avoid the<span style=\"color: #ff0000;\"> Diseas</span>e progression. Based on clinical practice and ROC analysis between critical and non-critical<span style=\"color: #ff00ff;\"> Patient</span>s, some cut-off values of the test items were obtained. With age >67.5 years, IL2R >793.5U/mL, CRP >30.7ng/mL, ferroprotein >2252μg/L, WBC >9.5*10^9/L or NC>7.305*10^9/L, progress to critical<span style=\"color: #ff00ff;\"> Ill</span>ness should be closely observed and prevented. Preventing recognition or blocking the occurrence of inflammatory action, new drugs development on immune regulation, might be new breakthroughs in the control of COVID-19 pneumonia. Pathological examination found lymphocyte-dominated mononuclear cell infiltration in interstitial pulmonary tissue, CD4+ and CD8+ T cells were significantly<span style=\"color: #ff00ff;\"> Reduc</span>ed in peripheral blood cells, but a<span style=\"color: #ff0000;\"> High</span> ratio of CD38+ (CD4 3.47%) and HLA-DR+ (CD8 39.4%) T cells [9] , which suggested excessive activation of pro-inflammatory cells. Presumably, lymphocyte deposition in lung tissue might contribute to lymphocyte<span style=\"color: #ff00ff;\"> Reduc</span>tion in peripheral blood. In addition, researchers found that CCR4+ CCR6+ Th17 cells were<span style=\"color: #ff00ff;\"> Increas</span>ed, and the IL-17 inhibitor (Secukinumab) against activated Th17 cells is promising for<span style=\"color: #ff0000;\"> Diseas</span>e control [9] . In this study, IL-6, TNFα and IL-8 may be potential targets for immunotherapy of COVID-19. With IL-6 >100pg/mL, the<span style=\"color: #ff00ff;\"> Patient</span>'s condition was extremely critical. According to the news, a research team from First Affiliated<span style=\"color: #ff0000;\"> Hospit</span>al of University of Science and Technology of China has used IL-6 receptor recombinant monoclonal antibody, Tozhu monoclonal antibody, in 14 critically or<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>ely<span style=\"color: #ff00ff;\"> Ill</span> COVID-19<span style=\"color: #ff00ff;\"> Patient</span>s [10] . The results were reported to be encouraging. Respiratory oxygenation indices of 14 Table 3 Correlation coefficient and P value between items and<span style=\"color: #ff0000;\"> Diseas</span>e<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>ity. . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.25.20025643 doi: medRxiv preprint . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.02.25.20025643 doi: medRxiv preprint</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 3.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Yonghua Wu </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-04 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1101/2020.03.04.975995 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid112');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid212');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid312');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid112\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">AbstractCoronavirus<span style=\"color: #ff0000;\"> Diseas</span>e 2019 (COVID-19) and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"> Acut</span>e respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually<span style=\"color: #ff0000;\"> High</span> proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid212\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Respiratory<span style=\"color: #ff00ff;\"> Pathogen</span>s continue to present an ever<span style=\"color: #ff00ff;\"> Increas</span>ing threat to public health (1,2).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Influenza, Respiratory syncytial virus, human metapneumovirus and other respiratory viruses are major etiological<span style=\"color: #ff00ff;\"> Agent</span>s for influenza like<span style=\"color: #ff00ff;\"> Ill</span>nesses (ILI) (3-5).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Makerere University Walter Reed Project (MUWRP) together with the Ugandan Ministry of Health and partners undertook surveillance to determine viral causes of influenza-like<span style=\"color: #ff00ff;\"> Ill</span>ness in Uganda.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">From 2008, MUWRP established<span style=\"color: #ff0000;\"> Hospit</span>al-based sentinel sites for surveillance activities.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">A total of five<span style=\"color: #ff0000;\"> Hospit</span>al-based sites were established, where<span style=\"color: #ff00ff;\"> Patient</span>s aged 6 months or older presenting with ILI were enrolled.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">From October 2008 to March 2017 a total of 9,472 participants were enrolled in the study from five<span style=\"color: #ff0000;\"> Hospit</span>al-based surveillance sentinel sites.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Detection of other viral causes of ILI is an indication of the public health threats posed by respiratory<span style=\"color: #ff00ff;\"> Pathogen</span>s.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 8:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Establishment of viral causes of ILI is critical for prevention and mitigation strategies to<span style=\"color: #ff0000;\"> Diseas</span>e threats.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 9:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Influenza viruses account for about 11% of the causes of influenza like<span style=\"color: #ff00ff;\"> Ill</span>ness, with influenza A being the dominant type.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid312\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">Respiratory<span style=\"color: #ff00ff;\"> Pathogen</span>s continue to present an ever<span style=\"color: #ff00ff;\"> Increas</span>ing threat to public health (1,2). Influenza, Respiratory syncytial virus, human metapneumovirus and other respiratory viruses are major etiological<span style=\"color: #ff00ff;\"> Agent</span>s for influenza like<span style=\"color: #ff00ff;\"> Ill</span>nesses (ILI) (3-5). Establishment of viral causes of ILI is critical for prevention and mitigation strategies to<span style=\"color: #ff0000;\"> Diseas</span>e threats. Makerere University Walter Reed Project (MUWRP) together with the Ugandan Ministry of Health and partners undertook surveillance to determine viral causes of influenza-like<span style=\"color: #ff00ff;\"> Ill</span>ness in Uganda. From 2008, MUWRP established<span style=\"color: #ff0000;\"> Hospit</span>al-based sentinel sites for surveillance activities. A total of five<span style=\"color: #ff0000;\"> Hospit</span>al-based sites were established, where<span style=\"color: #ff00ff;\"> Patient</span>s aged 6 months or older presenting with ILI were enrolled. Consents were obtained as required, and a throat and/ or nasopharyngeal swab collected. Samples were screened by PCR for viral causes. From October 2008 to March 2017 a total of 9,472 participants were enrolled in the study from five<span style=\"color: #ff0000;\"> Hospit</span>al-based surveillance sentinel sites. Majority of participants were children under 5 years n= 8,169 (86.2%). 615 (6.5%) samples tested positive for influenza A, while 385 (4.1%) tested positive for influenza B viruses and 10 (0.1%) were co-infections between influenza A and B. Of the 2,062 influenza negative samples, results indicated positivity for the following organisms; adenoviruses (9.4%), respiratory syncytial B (7.3%), parainfluenza-3 (4.5%), parainfluenza-1 (4.3%), respiratory syncytial A (3.5%), human bocavirus (1.7%), human metapneumovirus (1.7%), human coronavirus (1.5%), parainfluenza-4 (1.4%) and parainfluenza-2 (0.9%) by PCR. Influenza viruses account for about 11% of the causes of influenza like<span style=\"color: #ff00ff;\"> Ill</span>ness, with influenza A being the dominant type. Among the other viral causes of ILI, adenoviruses were the most dominant. Detection of other viral causes of ILI is an indication of the public health threats posed by respiratory<span style=\"color: #ff00ff;\"> Pathogen</span>s. Influenza; Influenza Like<span style=\"color: #ff00ff;\"> Ill</span>ness; Surveillance</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 4.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Bing Zhao; Chao Ni; Ran Gao; Yuyan Wang; Li Yang; Jinsong Wei; Ting Lv; Jianqing Liang; Qisheng Zhang; Wei Xu; Youhua Xie; Xiaoyue Wang; Zhenghong Yuan; Junbo Liang; Rong Zhang; Xinhua Lin </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-17 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1101/2020.03.16.990317 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid113');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid213');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid313');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid113\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">The newly emerged pandemic coronavirus, SARS-CoV-2, has posed a significant public health threat worldwide. However, the mode of virus transmission and tissue tropism is not well established yet. Recent findings of substantial liver damage in<span style=\"color: #ff00ff;\"> Patient</span>s and ACE2+ cholangiocytes in healthy liver tissues prompted us to hypothesize that human liver ductal organoids could serve as a model to determine the susceptibility and mechanisms underlining the liver damage upon SARS-CoV-2 infection. By single-cell RNA sequencing, we found that long-term liver ductal organoid culture preserved the human specific ACE2+ population of cholangiocytes. Moreover, human liver ductal organoids were permissive to SARS-CoV-2 infection and support robust replication. Notably, virus infection impaired the barrier and bile acid transporting functions of cholangiocytes through dysregulation of genes involved in tight junction formation and bile acid transportation, which could explain the bile acid accumulation and consequent liver damage in<span style=\"color: #ff00ff;\"> Patient</span>s. These results indicate that control of liver damage caused directly by viral infection should be valued in treating COVID-19<span style=\"color: #ff00ff;\"> Patient</span>s. Our findings also provide an application of human organoids in investigating the tropism and<span style=\"color: #ff00ff;\"> Pathogen</span>esis of SARS-CoV-2, which would facilitate novel drug discovery.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid213\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">48 The dominant<span style=\"color: #ff00ff;\"> Symptom</span>s of coronavirus<span style=\"color: #ff0000;\"> Diseas</span>e 2019 (COVID-19) are fever and 49 cough 2,3 .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">However, a proportion of<span style=\"color: #ff00ff;\"> Patient</span>s showed multi-organ damage and 50 dysfunction 2-4 .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">We isolated and plaque-purified the SARS-CoV-2 from a COVID-19 111<span style=\"color: #ff00ff;\"> Patient</span> in Shanghai.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Of note, liver damage is emerging as a co-existed<span style=\"color: #ff00ff;\"> Symptom</span> reported 51 in<span style=\"color: #ff00ff;\"> Patient</span>s with COVID-19.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid313\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">Wuhan (China) has caused emergent and significant threats to global public health 1 . 48 The dominant<span style=\"color: #ff00ff;\"> Symptom</span>s of coronavirus<span style=\"color: #ff0000;\"> Diseas</span>e 2019 (COVID-19) are fever and 49 cough 2,3 . However, a proportion of<span style=\"color: #ff00ff;\"> Patient</span>s showed multi-organ damage and 50 dysfunction 2-4 . Of note, liver damage is emerging as a co-existed<span style=\"color: #ff00ff;\"> Symptom</span> reported 51 in<span style=\"color: #ff00ff;\"> Patient</span>s with COVID-19. A recent epidemiologic study in Shanghai (China) 52 reported that 75 out of 148 (50.7%) COVID-19<span style=\"color: #ff00ff;\"> Patient</span>s had liver function 53 abnormality, indicated by key liver function parameters above the normal range, 54 including alanine aminotransferase (ALT), aspartate aminotransferase (AST), alkaline 55 phosphatase (ALP) or total bilirubin (TBIL) 5 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint ACE2+ cholangiocytes are preserved in human liver ductal organoid cultures. 88 To establish the SARS-CoV-2 infection model with human liver ductal organoids, 89 we first determined whether the long-term organoid culture could preserve the 90 ACE2+ cholangiocytes ex vivo. We processed single-cell RNA sequencing (scRNA-seq) 91 to interrogate the transcriptome of cholangiocytes in human liver ductal organoids. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint 109 We next examined the susceptibility of human liver ductal organoids to 110 SARS-CoV-2. We isolated and plaque-purified the SARS-CoV-2 from a COVID-19 111<span style=\"color: #ff00ff;\"> Patient</span> in Shanghai. The liver ductal organoids from two individuals were inoculated 112 with SARS-CoV-2 for 1 hour then re-embedded in Matrigel and maintained in culture Statistical analysis. 249 We employed Student's t-test or ANOVA test to analyze the parametric 250 experimental results. Significant differences were noted with asterisks. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 5.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Rapid detection of respiratory organisms with FilmArray respiratory panel and its impact on clinical decisions in Shanghai, China, 2016‐2018 </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Qian, Yiyi; Ai, Jingwen; Wu, Jing; Yu, Shenglei; Cui, Peng; Gao, Yan; Jin, Jialin; Weng, Xinhua; Zhang, Wenhong </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2019-12-01 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7040966/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid114');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid214');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid314');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid114\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">BACKGROUND: In this study, we evaluated the diagnostic potential and clinical impact of an automated multiplex PCR platform (the FilmArray Respiratory Panel; FA‐RP), specially designed for<span style=\"color: #ff00ff;\"> Pathogen</span> detection in respiratory tract infections in adults with<span style=\"color: #ff00ff;\"> Unexplain</span>ed pneumonia (UP). METHODS: A total of 112 UP<span style=\"color: #ff00ff;\"> Patient</span>s in Shanghai, China, were enrolled prospectively and assessed using the FA‐RP from October 2016 to March 2018. We examined the test results and their influence on clinical decisions. Furthermore, as a control group, we retrospectively obtained the clinical data of 70 UP<span style=\"color: #ff00ff;\"> Patient</span>s between October 2014 and March 2016 (before the FA‐RP was available). The two<span style=\"color: #ff00ff;\"> Patient</span> groups were compared with respect to factors, including general antimicrobial use and defined daily dose (DDD) numbers. RESULTS: Between October 2016 and March 2018, the positive rate obtained using FA‐RP for UP was 76.8%. The primary<span style=\"color: #ff00ff;\"> Pathogen</span>s in adults with UP were Influenza A/B (47.3%, 53/112). Compared with the<span style=\"color: #ff00ff;\"> Patient</span>s before FA‐RP was available,<span style=\"color: #ff00ff;\"> Patient</span>s who underwent FA‐RP testing had<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>er</span> rates of antiviral drug use and antibiotic de‐escalation during clinical treatment. FA‐RP significantly decreased the total DDDs of antibiotic or antifungal drugs DDDs by 7 days after admission (10.6 ± 2.5 vs 14.1 ± 8.8, P < .01). CONCLUSIONS: The FA‐RP is a rapid and sensitive nucleic acid amplification test method for UP diagnosis in adults. The application of FA‐RP may lead to a more accurately targeted antimicrobial treatment and<span style=\"color: #ff00ff;\"> Reduc</span>ed use of antibiotic/antifungal drugs.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid214\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">After the<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"> Acut</span>e respiratory syndrome outbreak, in order to track similarly<span style=\"color: #ff0000;\"> High</span>ly contagious and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Low</span>er</span> respiratory<span style=\"color: #ff00ff;\"> Ill</span>ness with pneumonia<span style=\"color: #ff00ff;\"> Symptom</span>s, the Chinese government has paid great attention to<span style=\"color: #ff00ff;\"> Unexplain</span>ed pneumonia (UP), a phrase coined by the Chinese Human<span style=\"color: #ff00ff;\"> Unexplain</span>ed Pneumonia Surveillance Network (CHUPSN) and published in 2004.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">As a consequence of the findings of the FA-FP regarding Influenza A, 15<span style=\"color: #ff00ff;\"> Patient</span>s were isolated in a negative pressure<span style=\"color: #ff00ff;\"> Ward</span> or single<span style=\"color: #ff00ff;\"> Ward</span>, including seven<span style=\"color: #ff0000;\"> High</span>ly<span style=\"color: #ff00ff;\"> Pathogen</span>ic H7N9 cases.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">We found that the FA-RP detected<span style=\"color: #ff00ff;\"> Pathogen</span>s in 76.8% (86/112) of all specimens, which is<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>er</span> than that the detection rate obtained when this technique was applied for general community-acquired pneumonia (38.6%, 49/127).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">7 These results, therefore, show that when applied clinically for UP, the FA-RP would provide a<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>er</span> probability of identifying causal<span style=\"color: #ff00ff;\"> Pathogen</span>ic microorganisms.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">18 Nevertheless, a<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>er</span> number of<span style=\"color: #ff00ff;\"> Patient</span>s in the FA-RP group received single doses or brief courses of antibiotics than those in the control group.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">As a result, intervention group unexpectedly showed longer duration of<span style=\"color: #ff0000;\"> Hospit</span>al stay than the control group (12.1 vs 7.6 days, P = .001).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">One reasonable explanation is that subjects in the intervention group had more<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e cases of pneumonia than those in the control group (higher PSI, This present study is one of the<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>al that have examined the application of FA-RP in Asia.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 8:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">[19] [20] [21] [22] [23] [24] Compared with other published studies, our study focused on adults with more<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e pneumonia conditions, and to the best of our knowledge, it is the only study that Our study does, however, have a few limitations.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 9:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Furthermore, the<span style=\"color: #ff00ff;\"> Incid</span>ence of various respiratory viruses may have varied across the two periods that may thus limit the validity of a direct comparison of clinical outcomes.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 10:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Thus, rapid and precise diagnosis of the causative<span style=\"color: #ff00ff;\"> Agent</span>s is critical for the prompt management of UP.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid314\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">After the<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"> Acut</span>e respiratory syndrome outbreak, in order to track similarly<span style=\"color: #ff0000;\"> High</span>ly contagious and<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Low</span>er</span> respiratory<span style=\"color: #ff00ff;\"> Ill</span>ness with pneumonia<span style=\"color: #ff00ff;\"> Symptom</span>s, the Chinese government has paid great attention to<span style=\"color: #ff00ff;\"> Unexplain</span>ed pneumonia (UP), a phrase coined by the Chinese Human<span style=\"color: #ff00ff;\"> Unexplain</span>ed Pneumonia Surveillance Network (CHUPSN) and published in 2004. UP refers to pneumonia, which could be life-threatening, characterized by rapid progression, a normal white blood cell count range, and poor response to antibiotics. 1 Although the common<span style=\"color: #ff00ff;\"> Pathogen</span>s causing UP are viruses or atypical<span style=\"color: #ff00ff;\"> Pathogen</span>s, their overlapping clinical<span style=\"color: #ff00ff;\"> Manifest</span>ations can impede the ability of clinicians to directly diagnose the causative<span style=\"color: #ff00ff;\"> Pathogen</span>s, which may lead to unnecessary antimicrobial usage. Thus, rapid and precise diagnosis of the causative<span style=\"color: #ff00ff;\"> Agent</span>s is critical for the prompt management of UP. To date, culture-based and other traditional methods have been used to help the clinical approach to diagnose UP; however, these methods have certain disadvantages. For example, rapid antigen testing has only a 40%-100% sensitivity for adult influenza and is dependent on the use of specific scientific techniques, whereas culturing viruses can take up to 10 days. 2 In recent years, there has been an<span style=\"color: #ff00ff;\"> Increas</span>e in the use of nucleic acid amplification test (NAAT)-based methods for the detection of viral<span style=\"color: #ff00ff;\"> Pathogen</span>s due to their excellent diagnostic ability in identifying a broad spectrum of<span style=\"color: #ff00ff;\"> Pathogen</span>s. 3  The prospective cohort was enrolled in Shanghai, China, between  Within 24 hours of<span style=\"color: #ff00ff;\"> Patient</span> admission, respiratory tract specimens were collected, including bronchoalveolar lavage fluid (BALF: among those<span style=\"color: #ff00ff;\"> Patient</span>s who gave consent to undergo bronchoscopy), sputum (if expectoration was evident), and nasopharyngeal swab (NPS, if no bronchoscopy was performed or no expectoration sample was collected). Notably,<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Low</span>er</span> respiratory tract specimens were given<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>er</span> priority (BALF > sputum > NPS) during the study. As soon as possible after sampling, the specimens were transferred to the Laboratory of Infectious<span style=\"color: #ff0000;\"> Diseas</span>es Research Institute, Huashan<span style=\"color: #ff0000;\"> Hospit</span>al, Shanghai. Professional pulmonary physicians oversaw BALF sampling, while the collection of NPS and sputum was performed by clinical physicians per the clinical standard operating procedure. Notably, those samples of NPS and BALF for which the viscosity was<span style=\"color: #ff00ff;\"> Low</span> required no further sample preparation before FilmArray panel testing, whereas viscous samples, such as sputum, were pre-treated using the previously described dunk and swirl method. 5 The remaining specimens were stored in a −80°C freezer for further use. In this study, we used version 1.7 of the FA-RP. It is an automated multiplex PCR detection platform designed to detect 20 common<span style=\"color: #ff00ff;\"> Pathogen</span>s causing respiratory infection and has a single specimen turnaround time of <2 hours. Tests using the panel were performed following the manufacturer's instructions. For specimens that the FA-RP reported as \"Influenza A, not sub-  For both<span style=\"color: #ff00ff;\"> Patient</span> groups, traditional standard-of-care laboratory diagnostic tests for respiratory infections were performed according to the requests of physicians, including smears, cultures, and serological tests. The serological test in our study was the PNEUMOSLIDE IgM (Vircell) commercial kit based on indirect immunofluorescence. In this study, clinical data, including antimicrobial use, and adjustment or de-escalation of antimicrobial therapy, were compared between the FA-RP group and retrospective control group. Turnaround time was defined as the length of time from sample collection to receiving the FA-RP results. Antimicrobial use was quantified as the defined daily dose (DDD)) number, calculated as ∑ (Drug Total Dosage (g)/Drug DDD index). 6 Adjustment of antimicrobial therapy was defined as a change in antimicrobial regimen immediately after receiving the FA-RP results. Overview of<span style=\"color: #ff00ff;\"> Patient</span> Enrollment Workflow. A total of 112<span style=\"color: #ff00ff;\"> Patient</span>s selected prospectively were submitted to FA-RP analyses and are<span style=\"color: #ff00ff;\"> Referr</span>ed to as the \"FA-RP group.\" Those who were not lost to follow-up and had complete data are<span style=\"color: #ff00ff;\"> Referr</span>ed to as the \"intervention group.\" Comparison was made between it and the historical control group The Mann-Whitney U test was used for the comparison of quantitative data. spss 20.0 (IBM) was used for data analyses. We used Graphpad 6.0 for drawing graphs. The Ethics Committee of the Huashan<span style=\"color: #ff0000;\"> Hospit</span>al, Shanghai, approved the study (KY2013-251; July 2, 2013). All participants gave their written informed consent. Table 1 . Mixed infections of multiple<span style=\"color: #ff00ff;\"> Pathogen</span>s were detected in six<span style=\"color: #ff00ff;\"> Patient</span>s: Influenza virus/RSV (n = 1), Influenza virus/human metapneumovirus (n = 1), Influenza virus/parainfluenza-2 (n = 1), M pneumoniae/human rhinovirus/enterovirus (n = 1), human rhinovirus/ enterovirus/parainfluenza-2 (n = 1), and adenovirus/coronavirus HKU1/coronavirus OC43 (n = 1). For 13 samples, the FA-RP results were \"Influenza A/not subtyped.\" In these cases, further subtyping was performed using real-time RT-PCR. As a consequence, Influenza A virus subtype H7N9 was detected in 11 samples and H1N1-2009 in the remaining two samples. As a consequence of the findings of the FA-FP regarding Influenza A, 15<span style=\"color: #ff00ff;\"> Patient</span>s were isolated in a negative pressure<span style=\"color: #ff00ff;\"> Ward</span> or single<span style=\"color: #ff00ff;\"> Ward</span>, including seven<span style=\"color: #ff0000;\"> High</span>ly<span style=\"color: #ff00ff;\"> Pathogen</span>ic H7N9 cases. In contrast, in the control group, no comparable isolation measures were undertaken. When we compared antimicrobial use between the intervention and control groups, we found that the two groups were comparable for the most baseline characteristics (Table 3) . Compared with the control group, the intervention group showed no significant decrease of antimicrobial DDDs (control group 14.3 ± 9.6 vs intervention group 13.8 ± 4.6, P > .05). However, after excluding antiviral drugs, we detected significantly<span style=\"color: #ff00ff;\"><span style=\"color: #ff00ff;\"> Low</span>er</span> antibiotic/antifungal DDDs  In this study, we applied FA-RP on diagnosis of UP, which is a certain type of pneumonia causing mainly by viruses and atypical<span style=\"color: #ff00ff;\"> Pathogen</span>s. We found that the FA-RP detected<span style=\"color: #ff00ff;\"> Pathogen</span>s in 76.8% (86/112) of all specimens, which is<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>er</span> than that the detection rate obtained when this technique was applied for general community-acquired pneumonia (38.6%, 49/127). 7 These results, therefore, show that when applied clinically for UP, the FA-RP would provide a<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>er</span> probability of identifying causal<span style=\"color: #ff00ff;\"> Pathogen</span>ic microorganisms. In the present study, we found that 53.8% (60/112) of the<span style=\"color: #ff00ff;\"> Patient</span>s in the FA-RP group received antiviral treatment, which is significantly<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>er</span> than that in the control group (14.3%, 10/70; randomized controlled trial discovered that use of the FA-RP was not associated with a<span style=\"color: #ff00ff;\"> Reduc</span>tion in the overall duration of antibiotic usage. 18 Nevertheless, a<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> High</span>er</span> number of<span style=\"color: #ff00ff;\"> Patient</span>s in the FA-RP group received single doses or brief courses of antibiotics than those in the control group. Given these inconclusive findings, further studies are required concerning the impact of FA-RP on the clinical decisions regarding pneumonia. We compared the duration of<span style=\"color: #ff0000;\"> Hospit</span>al stay between the two groups. As a result, intervention group unexpectedly showed longer duration of<span style=\"color: #ff0000;\"> Hospit</span>al stay than the control group (12.1 vs 7.6 days, P = .001). One reasonable explanation is that subjects in the intervention group had more<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e cases of pneumonia than those in the control group (higher PSI, This present study is one of the<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>al that have examined the application of FA-RP in Asia. [19] [20] [21] [22] [23] [24] Compared with other published studies, our study focused on adults with more<span style=\"color: #ff00ff;\"><span style=\"color: #ff0000;\"> Sever</span></span>e pneumonia conditions, and to the best of our knowledge, it is the only study that Our study does, however, have a few limitations. Notably, the sample size was limited, and the study was conducted only at a single location, which may thus limit the general applicability of the findings. In addition, the study included two cohorts from different time periods; hence, there may have been differences in factors that influenced clinician behavior between the two periods that were not accounted in our analysis. Furthermore, the<span style=\"color: #ff00ff;\"> Incid</span>ence of various respiratory viruses may have varied across the two periods that may thus limit the validity of a direct comparison of clinical outcomes. In conclusion, the FA-RP is a rapid and sensitive NAAT method that can be used for the diagnosis of UP in adults. In the two recent years in Shanghai, the most common<span style=\"color: #ff00ff;\"> Pathogen</span> causing UP was identified as the Influenza virus. However, in this regard, it should be taken into consideration that the current version of the FA-RP is unable to directly detect H7N9. Nevertheless, the results obtained in this study show that the FA-RP may make a valuable contribution to clinical decision making and facilitate the reasonable use of antibiotics. We would like to thank Yinlei Zhang, Tracy Wang, Xuyang Wang, and Xinchang Chen for their substantial contributions to this study. Yiyi Qian https://orcid.org/0000-0002-0714-3894</strong></p>\n",
       "            </div>\n",
       "            "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "search_severity = \"\"\"Severity of disease including risk of fatality \n",
    "among symptomatic hospitalized patients and high-risk patient groups about covid-19\"\"\"\n",
    "\n",
    "tags_severity  = \"\"\"severity disease fatality symptomatic hospitalized high-risk\"\"\"\n",
    "\n",
    "searching.SearchDisplayPaper(search=search_severity, pubdate_in=\"2019-12-01 2020-04-01\", topn=5, topn_sent=10, tags=tags_severity)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "                <script>\n",
       "                function visibilite(thingId)\n",
       "                {\n",
       "                var targetElement;\n",
       "                targetElement = document.getElementById(thingId) ;\n",
       "                if (targetElement.style.display == \"none\")\n",
       "                {\n",
       "                targetElement.style.display = \"\" ;\n",
       "                } else {\n",
       "                targetElement.style.display = \"none\" ;\n",
       "                }\n",
       "                }\n",
       "                </script>\n",
       "                </body>\n",
       "                \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 1.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Rapid detection of respiratory organisms with FilmArray respiratory panel and its impact on clinical decisions in Shanghai, China, 2016‐2018 </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Qian, Yiyi; Ai, Jingwen; Wu, Jing; Yu, Shenglei; Cui, Peng; Gao, Yan; Jin, Jialin; Weng, Xinhua; Zhang, Wenhong </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2019-12-01 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7040966/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid115');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid215');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid315');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid115\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">BACKGROUND: In this study, we evaluated the diagnostic potential and clinical impact of an automated multiplex PCR platform (the FilmArray Respiratory Panel; FA‐RP), specially designed for pathogen detection in respiratory tract<span style=\"color: #ff00ff;\"> Infect</span>ions in adults with unexplained pneumonia (UP). METHODS: A total of 112 UP patients in Shanghai, China, were enrolled prospectively and assessed using the FA‐RP from October 2016 to March 2018. We examined the test results and their influence on clinical decisions. Furthermore, as a control group, we retrospectively obtained the clinical data of 70 UP patients between October 2014 and March 2016 (before the FA‐RP was available). The two patient groups were compared with respect to factors, including general antimicrobial use and defined daily dose (DDD) numbers. RESULTS: Between October 2016 and March 2018, the positive rate obtained using FA‐RP for UP was 76.8%. The primary pathogens in adults with UP were Influenza A/B (47.3%, 53/112). Compared with the patients before FA‐RP was available, patients who underwent FA‐RP testing had higher rates of antiviral drug use and antibiotic de‐escalation during clinical treatment. FA‐RP significantly decreased the total DDDs of antibiotic or antifungal drugs DDDs by 7 days after admission (10.6 ± 2.5 vs 14.1 ± 8.8, P < .01). CONCLUSIONS: The FA‐RP is a rapid and sensitive nucleic acid amplification test method for UP diagnosis in adults. The application of FA‐RP may lead to a more accurately targeted antimicrobial treatment and reduced use of antibiotic/antifungal drugs.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid215\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">After the severe acute respiratory syndrome outbreak, in order to track similarly highly<span style=\"color: #ff00ff;\"> Contagi</span>ous and severe lower respiratory illness with pneumonia symptoms, the Chinese government has paid great attention to unexplained pneumonia (UP), a phrase coined by the Chinese Human Unexplained Pneumonia Surveillance Network (CHUPSN) and published in 2004.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Mixed<span style=\"color: #ff00ff;\"> Infect</span>ions of multiple pathogens were detected in six patients: Influenza virus/RSV (n = 1), Influenza virus/human metapneumovirus (n = 1), Influenza virus/parainfluenza-2 (n = 1), M pneumoniae/human rhinovirus/enterovirus (n = 1), human rhinovirus/ enterovirus/parainfluenza-2 (n = 1), and adenovirus/coronavirus HKU1/coronavirus OC43 (n = 1).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">It is an automated multiplex PCR detection platform designed to detect 20 common pathogens causing respiratory<span style=\"color: #ff00ff;\"> Infect</span>ion and has a single specimen turnaround time of <2 hours.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">As soon as possible after sampling, the specimens were transferred to the Laboratory of<span style=\"color: #ff00ff;\"> Infect</span>ious Diseases Research Institute, Huashan Hospital, Shanghai.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">For specimens that the FA-RP reported as \"Influenza A, not sub-  For both patient groups, traditional standard-of-care laboratory diagnostic tests for respiratory<span style=\"color: #ff00ff;\"> Infect</span>ions were performed according to the requests of physicians, including smears, cultures, and serological tests.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid315\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">After the severe acute respiratory syndrome outbreak, in order to track similarly highly<span style=\"color: #ff00ff;\"> Contagi</span>ous and severe lower respiratory illness with pneumonia symptoms, the Chinese government has paid great attention to unexplained pneumonia (UP), a phrase coined by the Chinese Human Unexplained Pneumonia Surveillance Network (CHUPSN) and published in 2004. UP refers to pneumonia, which could be life-threatening, characterized by rapid progression, a normal white blood cell count range, and poor response to antibiotics. 1 Although the common pathogens causing UP are viruses or atypical pathogens, their overlapping clinical manifestations can impede the ability of clinicians to directly diagnose the causative pathogens, which may lead to unnecessary antimicrobial usage. Thus, rapid and precise diagnosis of the causative agents is critical for the prompt management of UP. To date, culture-based and other traditional methods have been used to help the clinical approach to diagnose UP; however, these methods have certain disadvantages. For example, rapid antigen testing has only a 40%-100% sensitivity for adult influenza and is dependent on the use of specific scientific techniques, whereas culturing viruses can take up to 10 days. 2 In recent years, there has been an increase in the use of nucleic acid amplification test (NAAT)-based methods for the detection of viral pathogens due to their excellent diagnostic ability in identifying a broad spectrum of pathogens. 3  The prospective cohort was enrolled in Shanghai, China, between  Within 24 hours of patient admission, respiratory tract specimens were collected, including bronchoalveolar lavage fluid (BALF: among those patients who gave consent to undergo bronchoscopy), sputum (if expectoration was evident), and nasopharyngeal swab (NPS, if no bronchoscopy was performed or no expectoration sample was collected). Notably, lower respiratory tract specimens were given higher priority (BALF > sputum > NPS) during the study. As soon as possible after sampling, the specimens were transferred to the Laboratory of<span style=\"color: #ff00ff;\"> Infect</span>ious Diseases Research Institute, Huashan Hospital, Shanghai. Professional pulmonary physicians oversaw BALF sampling, while the collection of NPS and sputum was performed by clinical physicians per the clinical standard operating procedure. Notably, those samples of NPS and BALF for which the viscosity was low required no further sample preparation before FilmArray panel testing, whereas viscous samples, such as sputum, were pre-treated using the previously described dunk and swirl method. 5 The remaining specimens were stored in a −80°C freezer for further use. In this study, we used version 1.7 of the FA-RP. It is an automated multiplex PCR detection platform designed to detect 20 common pathogens causing respiratory<span style=\"color: #ff00ff;\"> Infect</span>ion and has a single specimen turnaround time of <2 hours. Tests using the panel were performed following the manufacturer's instructions. For specimens that the FA-RP reported as \"Influenza A, not sub-  For both patient groups, traditional standard-of-care laboratory diagnostic tests for respiratory<span style=\"color: #ff00ff;\"> Infect</span>ions were performed according to the requests of physicians, including smears, cultures, and serological tests. The serological test in our study was the PNEUMOSLIDE IgM (Vircell) commercial kit based on indirect immunofluorescence. In this study, clinical data, including antimicrobial use, and adjustment or de-escalation of antimicrobial therapy, were compared between the FA-RP group and retrospective control group. Turnaround time was defined as the length of time from sample collection to receiving the FA-RP results. Antimicrobial use was quantified as the defined daily dose (DDD)) number, calculated as ∑ (Drug Total Dosage (g)/Drug DDD index). 6 Adjustment of antimicrobial therapy was defined as a change in antimicrobial regimen immediately after receiving the FA-RP results. Overview of Patient Enrollment Workflow. A total of 112 patients selected prospectively were submitted to FA-RP analyses and are referred to as the \"FA-RP group.\" Those who were not lost to follow-up and had complete data are referred to as the \"intervention group.\" Comparison was made between it and the historical control group The Mann-Whitney U test was used for the comparison of quantitative data. spss 20.0 (IBM) was used for data analyses. We used Graphpad 6.0 for drawing graphs. The Ethics Committee of the Huashan Hospital, Shanghai, approved the study (KY2013-251; July 2, 2013). All participants gave their written informed consent. Table 1 . Mixed<span style=\"color: #ff00ff;\"> Infect</span>ions of multiple pathogens were detected in six patients: Influenza virus/RSV (n = 1), Influenza virus/human metapneumovirus (n = 1), Influenza virus/parainfluenza-2 (n = 1), M pneumoniae/human rhinovirus/enterovirus (n = 1), human rhinovirus/ enterovirus/parainfluenza-2 (n = 1), and adenovirus/coronavirus HKU1/coronavirus OC43 (n = 1). For 13 samples, the FA-RP results were \"Influenza A/not subtyped.\" In these cases, further subtyping was performed using real-time RT-PCR. As a consequence, Influenza A virus subtype H7N9 was detected in 11 samples and H1N1-2009 in the remaining two samples. As a consequence of the findings of the FA-FP regarding Influenza A, 15 patients were isolated in a negative pressure ward or single ward, including seven highly pathogenic H7N9 cases. In contrast, in the control group, no comparable isolation measures were undertaken. When we compared antimicrobial use between the intervention and control groups, we found that the two groups were comparable for the most baseline characteristics (Table 3) . Compared with the control group, the intervention group showed no significant decrease of antimicrobial DDDs (control group 14.3 ± 9.6 vs intervention group 13.8 ± 4.6, P > .05). However, after excluding antiviral drugs, we detected significantly lower antibiotic/antifungal DDDs  In this study, we applied FA-RP on diagnosis of UP, which is a certain type of pneumonia causing mainly by viruses and atypical pathogens. We found that the FA-RP detected pathogens in 76.8% (86/112) of all specimens, which is higher than that the detection rate obtained when this technique was applied for general community-acquired pneumonia (38.6%, 49/127). 7 These results, therefore, show that when applied clinically for UP, the FA-RP would provide a higher probability of identifying causal pathogenic microorganisms. In the present study, we found that 53.8% (60/112) of the patients in the FA-RP group received antiviral treatment, which is significantly higher than that in the control group (14.3%, 10/70; randomized controlled trial discovered that use of the FA-RP was not associated with a reduction in the overall duration of antibiotic usage. 18 Nevertheless, a higher number of patients in the FA-RP group received single doses or brief courses of antibiotics than those in the control group. Given these inconclusive findings, further studies are required concerning the impact of FA-RP on the clinical decisions regarding pneumonia. We compared the duration of hospital stay between the two groups. As a result, intervention group unexpectedly showed longer duration of hospital stay than the control group (12.1 vs 7.6 days, P = .001). One reasonable explanation is that subjects in the intervention group had more severe cases of pneumonia than those in the control group (higher PSI, This present study is one of the several that have examined the application of FA-RP in Asia. [19] [20] [21] [22] [23] [24] Compared with other published studies, our study focused on adults with more severe pneumonia conditions, and to the best of our knowledge, it is the only study that Our study does, however, have a few limitations. Notably, the sample size was limited, and the study was conducted only at a single location, which may thus limit the general applicability of the findings. In addition, the study included two cohorts from different time periods; hence, there may have been differences in factors that influenced clinician behavior between the two periods that were not accounted in our analysis. Furthermore, the incidence of various respiratory viruses may have varied across the two periods that may thus limit the validity of a direct comparison of clinical outcomes. In conclusion, the FA-RP is a rapid and sensitive NAAT method that can be used for the diagnosis of UP in adults. In the two recent years in Shanghai, the most common pathogen causing UP was identified as the Influenza virus. However, in this regard, it should be taken into consideration that the current version of the FA-RP is unable to directly detect H7N9. Nevertheless, the results obtained in this study show that the FA-RP may make a valuable contribution to clinical decision making and facilitate the reasonable use of antibiotics. We would like to thank Yinlei Zhang, Tracy Wang, Xuyang Wang, and Xinchang Chen for their substantial contributions to this study. Yiyi Qian https://orcid.org/0000-0002-0714-3894</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 2.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> A realistic two-strain model for MERS-CoV infection uncovers the high risk for epidemic propagation </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Sardar, Tridip; Ghosh, Indrajit; Rodó, Xavier; Chattopadhyay, Joydev </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-14 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7046297/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid116');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid216');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid316');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid116\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) causes severe acute respiratory illness with a case fatality rate (CFR) of 35,5%. The highest number of MERS-CoV cases are from Saudi-Arabia, the major worldwide hotspot for this disease. In the absence of neither effective treatment nor a ready-to-use vaccine and with yet an incomplete understanding of its epidemiological cycle, prevention and containment measures can be derived from mathematical models of disease epidemiology. We constructed 2-strain models to predict past outbreaks in the interval 2012–2016 and derive key epidemiological information for Macca, Madina and Riyadh. We approached variability in<span style=\"color: #ff00ff;\"> Infect</span>ion through three different disease incidence functions capturing social behavior in response to an epidemic (e.g. Bilinear, BL; Non-monotone, NM; and Saturated, SAT models). The best model combination successfully anticipated the total number of MERS-CoV clinical cases for the 2015–2016 season and accurately predicted both the number of cases at the peak of seasonal incidence and the overall shape of the epidemic cycle. The evolution in the basic reproduction number (R(0)) warns that MERS-CoV may easily take an epidemic form. The best model correctly captures this feature, indicating a high epidemic risk (1≤R(0)≤2,5) in Riyadh and Macca and confirming the alleged co-circulation of more than one strain. Accurate predictions of the future MERS-CoV peak week, as well as the number of cases at the peak are now possible. These results indicate public health agencies should be aware that measures for strict containment are urgently needed before new epidemics take off in the region.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid216\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">HCoV OC43 and HCoV 229E are responsible for one-third of all common colds,<span style=\"color: #ff00ff;\"> Infect</span>ing all age groups (2), but there have been reports of a more severe lower respiratory tract involvement (3, 4) .</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid316\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">coronavirus (HCoV) OC43 belongs to the Betacoronavirus genus of the Coronaviridae and its genome is formed by a positivesense, single-stranded RNA of ca. 31.5 kb (1) . HCoV OC43 and HCoV 229E are responsible for one-third of all common colds,<span style=\"color: #ff00ff;\"> Infect</span>ing all age groups (2), but there have been reports of a more severe lower respiratory tract involvement (3, 4) . The aim of the present study was to determine the full-length genome of the first HCoV OC43 isolated from Mexico, which was obtained from a viral metagenomic analysis of a nasal washing from a child who required hospitalization due to clinical or radiological signs of pneumonia of unknown origin, negative for commonly associated respiratory viruses and bacteria, using the xTAG Bioplex respiratory viral panel and multiplex PCR Seeplex Pneumobacter ACE detection kit, respectively (5) . Nucleic acids from the sample were extracted with the PureLink Viral RNA/DNA kit. Before extraction, the sample was treated with Turbo DNase and RNase, and a random-primer amplification of the genetic material was performed as previously described (5) . The sequencing was carried out with the Illumina Genome Analyzer IIx single-end platform. A total of 12,330,418 reads were generated and 1,657,101 (13.44%) were mapped to the consensus sequence of HCoV OC43. Mapping was accomplished by SMALT v.0.7.6 (6) with default parameters (only Ϫy set to 0.8). The whole viral genome sequence was obtained with an average coverage of 3,368ϫ. The same genome sequence was obtained by de novo assembly using Velvet (7) . The complete Mexican HCoV OC43 genome was composed of 30,712 nucleotides, with an average GϩC content of 35.51%. The strain shows the typical OC43 organization, with the following 5= untranslated region (UTR) (nt 1 to 199), genes: orf1ab (nt 200 to Accession number(s). The complete genome sequence of the Mexican OC43 strain was deposited at GenBank under the accession number KX344031.</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 3.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> RNA decay during gammaherpesvirus infection reduces RNA polymerase II occupancy of host promoters but spares viral promoters </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Hartenian, Ella; Gilbertson, Sarah; Federspiel, Joel D.; Cristea, Ileana M.; Glaunsinger, Britt A. </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-07 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7032723/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid117');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid217');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid317');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid117\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">In mammalian cells, widespread acceleration of cytoplasmic mRNA degradation is linked to impaired RNA polymerase II (Pol II) transcription. This mRNA decay-induced transcriptional repression occurs during<span style=\"color: #ff00ff;\"> Infect</span>ion with gammaherpesviruses including Kaposi’s sarcoma-associated herpesvirus (KSHV) and murine gammaherpesvirus 68 (MHV68), which encode an mRNA endonuclease that initiates widespread RNA decay. Here, we show that MHV68-induced mRNA decay leads to a genome-wide reduction of Pol II occupancy at mammalian promoters. This reduced Pol II occupancy is accompanied by down-regulation of multiple Pol II subunits and TFIIB in the nucleus of<span style=\"color: #ff00ff;\"> Infect</span>ed cells, as revealed by mass spectrometry-based global measurements of protein abundance. Viral genes, despite the fact that they require Pol II for transcription, escape transcriptional repression. Protection is not governed by viral promoter sequences; instead, location on the viral genome is both necessary and sufficient to escape the transcriptional repression effects of mRNA decay. We propose a model in which the ability to escape from transcriptional repression is linked to the localization of viral DNA within replication compartments, providing a means for these viruses to counteract decay-induced transcript loss.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid217\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Outbreaks of PEDV<span style=\"color: #ff00ff;\"> Infect</span>ion have been documented mainly in European and Asian<span style=\"color: #ff00ff;\"> Countri</span>es since the disease was first recognized in an acute outbreak of diarrhea in pigs (1) .</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid317\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">family Coronaviridae, is an enveloped virus with a singlestranded, positive-sense RNA and causes porcine epidemic diarrhea, an enteric disease characterized by acute watery diarrhea, dehydration, vomiting, and high mortality in nursery piglets (1, 2) . Outbreaks of PEDV<span style=\"color: #ff00ff;\"> Infect</span>ion have been documented mainly in European and Asian<span style=\"color: #ff00ff;\"> Countri</span>es since the disease was first recognized in an acute outbreak of diarrhea in pigs (1) . In the autumn of 2010, severe diarrhea in piglets emerged in China, affecting more than 1,000,000 piglets (3). A PEDV variant has been identified as a main causative agent in the disease (4) (5) (6) (7) . In this study, a PEDV strain, JS2008, was detected and identified in stool samples from a piglet with severe diarrhea on a vaccinated farm in eastern China. This strain can be easily isolated in Vero cells. Thus, we sought to analyze the genome sequence of PEDV strain JS2008 and understand its molecular characteristics. Twenty pairs of oligonucleotide primers to amplify regions of the JS2008 genome were designed from alignments of available PEDV genomes. The PCR products were purified and cloned into the pMD18-T vector (TaKaRa) and sequenced with an ABI3730XL genome sequencer. The terminal sequences were acquired by using a kit for rapid amplification of cDNA ends (RACE) (Clontech, Japan). The complete genome sequence of JS2008 is 27,953 nucleotides (nt) in length, excluding the poly(A) tail. The genome showed 99.7% and 98.0% nucleotide sequence similarity to the attenuated DR13 and CV777 vaccine strains. It shared 97.3 to 97.5% sequence identity with the PEDV variant that has occurred in recent years in China. The organization of the genome of JS2008 is similar to that of other reported PEDV genomes, with the characteristic gene order 5=-ORF1a/1b-S-ORF3-E-M-N-3= (2, 8, 9) . The ORF1a/1b gene is 20,321 nt in length. The sizes of the other genes, S, ORF3, E, M, and N, are 4,149 nt, 276 nt, 231 nt, 681 nt, and 1,326 nt, respectively. Sequence analysis showed that there was a 49-nt deletion in the ORF3 gene, and this characteristic was similar to a characteristic of the attenuated DR13. Meanwhile, the E gene was 21 nt longer than that of the attenuated DR13 and had the same sequence as that of the PEDV variant. These findings may indicate that JS2008 is a recombinant of the PEDV vaccine strain and the variant. Phylogenetic analysis of the complete genome showed that JS2008 has a close relationship to the attenuated DR13 vaccine strain and a distant relationship to the prevalent PEDV variant in China. The availability of the genome sequence of JS2008 will facilitate future investigations of the epidemiology and evolution of PEDV. Nucleotide sequence accession number. The genome sequence of PEDV strain JS2008 was deposited in GenBank under accession number KC109141. This work was supported by the Special Fund for Independent Innovation of Agricultural Science and Technology in Jiangsu Province [CX (12) 3076].</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 4.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Yonghua Wu </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-04 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1101/2020.03.04.975995 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid118');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid218');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid318');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid118\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of<span style=\"color: #ff00ff;\"> Infect</span>ing human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to<span style=\"color: #ff00ff;\"> Infect</span> humans in the future.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid218\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Respiratory pathogens<span style=\"color: #ff00ff;\"> Contin</span>ue to present an ever increasing threat to public health (1,2).</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid318\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">Respiratory pathogens<span style=\"color: #ff00ff;\"> Contin</span>ue to present an ever increasing threat to public health (1,2). Influenza, Respiratory syncytial virus, human metapneumovirus and other respiratory viruses are major etiological agents for influenza like illnesses (ILI) (3-5). Establishment of viral causes of ILI is critical for prevention and mitigation strategies to disease threats. Makerere University Walter Reed Project (MUWRP) together with the Ugandan Ministry of Health and partners undertook surveillance to determine viral causes of influenza-like illness in Uganda. From 2008, MUWRP established hospital-based sentinel sites for surveillance activities. A total of five hospital-based sites were established, where patients aged 6 months or older presenting with ILI were enrolled. Consents were obtained as required, and a throat and/ or nasopharyngeal swab collected. Samples were screened by PCR for viral causes. From October 2008 to March 2017 a total of 9,472 participants were enrolled in the study from five hospital-based surveillance sentinel sites. Majority of participants were children under 5 years n= 8,169 (86.2%). 615 (6.5%) samples tested positive for influenza A, while 385 (4.1%) tested positive for influenza B viruses and 10 (0.1%) were co-infections between influenza A and B. Of the 2,062 influenza negative samples, results indicated positivity for the following organisms; adenoviruses (9.4%), respiratory syncytial B (7.3%), parainfluenza-3 (4.5%), parainfluenza-1 (4.3%), respiratory syncytial A (3.5%), human bocavirus (1.7%), human metapneumovirus (1.7%), human coronavirus (1.5%), parainfluenza-4 (1.4%) and parainfluenza-2 (0.9%) by PCR. Influenza viruses account for about 11% of the causes of influenza like illness, with influenza A being the dominant type. Among the other viral causes of ILI, adenoviruses were the most dominant. Detection of other viral causes of ILI is an indication of the public health threats posed by respiratory pathogens. Influenza; Influenza Like Illness; Surveillance</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 5.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> The effect of human mobility and control measures on the COVID-19 epidemic in China </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Moritz U.G. Kraemer; Chia-Hung Yang; Bernardo Gutierrez; Chieh-Hsi Wu; Brennan Klein; David M. Pigott; Louis du Plessis; Nuno R Faria; Ruoran Li; William P. Hanage; John S Brownstein; Maylis Layan; Alessandro Vespignani; Huaiyu Tian; Christopher Dye; Simon Cauchemez; Oliver Pybus; Samuel V Scarpino </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-06 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1101/2020.03.02.20026708 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid119');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid219');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid319');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid119\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">The ongoing COVID-19 outbreak has expanded rapidly throughout China. Major behavioral, clinical, and state interventions are underway currently to mitigate the epidemic and prevent the persistence of the virus in human<span style=\"color: #ff0000;\"> Popul</span>ations in China and worldwide. It remains unclear how these unprecedented interventions, including travel restrictions, have affected COVID-19 spread in China. We use real-time mobility data from Wuhan and detailed case data including travel history to elucidate the role of case importation on transmission in<span style=\"color: #ff00ff;\"> Citi</span>es across China and ascertain the impact of control measures. Early on, the spatial distribution of COVID-19 cases in China was well explained by human mobility data. Following the implementation of control measures, this correlation dropped and growth rates became negative in most locations, although shifts in the demographics of reported cases are still indicative of local chains of transmission outside Wuhan. This study shows that the drastic control measures implemented in China have substantially mitigated the spread of COVID-19.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid219\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">While no region is protected, this interface frequently occurs in low-and middle-income<span style=\"color: #ff00ff;\"> Countri</span>es, where preparedness funding and activities by necessity come after everyday urgent<span style=\"color: #ff0000;\"> Popul</span>ation and<span style=\"color: #ff00ff;\"> Individu</span>al healthcare needs.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">We too have been encouraged by the progress made with the Platform foR European Preparedness Against Re-emerging Epidemics (PREPARE) initiative -a model for well-resourced health care systems to consider in preparing for the certainty of uncertain emerging<span style=\"color: #ff00ff;\"> Infect</span>ious threats.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">One such example is the freely available World Health Organization -International Severe Acute Respiratory & Emerging<span style=\"color: #ff00ff;\"> Infect</span>ion Consortium (ISARIC) clinical characterization protocol [1] , developed to provide clinicians and researchers with a case report form, adaptable to many clinical<span style=\"color: #ff00ff;\"> Infect</span>ious syndromes, and with tiered data collection options, responsive to the capacity of various health care settings, for both observational and experimental research designs.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">One emerging research collaboration to use this approach is the Short PeRiod IncideNce sTudy of Severe Acute Respiratory<span style=\"color: #ff00ff;\"> Infect</span>ion (SPRINT-SARI) [2] , a growing international collaboration of critical care and<span style=\"color: #ff00ff;\"> Infect</span>ious diseases clinicians seeking to build research capacity during inter-outbreak periods.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid319\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">We thank authors Gobat, Butler, Webb and Nichol for their thoughtful and practical suggestions to advance our ability to perform practiceinforming research in the context of outbreaks and pandemics. We too have been encouraged by the progress made with the Platform foR European Preparedness Against Re-emerging Epidemics (PREPARE) initiative -a model for well-resourced health care systems to consider in preparing for the certainty of uncertain emerging<span style=\"color: #ff00ff;\"> Infect</span>ious threats. While focused upon European preparedness, one additional promising aspect of PREPARE, is the provision to assist in addressing threats wherever they emerge. Novel viral pathogens often arise at the human-animal interface. Avian influenza, Ebola virus, Severe Acute Respiratory Syndrome and Middle East Respiratory Syndrome coronaviruses are but a few recent examples. While no region is protected, this interface frequently occurs in low-and middle-income<span style=\"color: #ff00ff;\"> Countri</span>es, where preparedness funding and activities by necessity come after everyday urgent<span style=\"color: #ff0000;\"> Popul</span>ation and<span style=\"color: #ff00ff;\"> Individu</span>al healthcare needs. We must encourage proactive humanitarian research preparedness from global regions with the luxury of this capacity. One such example is the freely available World Health Organization -International Severe Acute Respiratory & Emerging<span style=\"color: #ff00ff;\"> Infect</span>ion Consortium (ISARIC) clinical characterization protocol [1] , developed to provide clinicians and researchers with a case report form, adaptable to many clinical<span style=\"color: #ff00ff;\"> Infect</span>ious syndromes, and with tiered data collection options, responsive to the capacity of various health care settings, for both observational and experimental research designs. One emerging research collaboration to use this approach is the Short PeRiod IncideNce sTudy of Severe Acute Respiratory<span style=\"color: #ff00ff;\"> Infect</span>ion (SPRINT-SARI) [2] , a growing international collaboration of critical care and<span style=\"color: #ff00ff;\"> Infect</span>ious diseases clinicians seeking to build research capacity during inter-outbreak periods. It's through such interdisciplinary collaborations across low-, middle-and high-income settings that we have the greatest hope of being prepared to perform globally relevant outbreak-related research. Sincerely,</strong></p>\n",
       "            </div>\n",
       "            "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "search_susceptibility = \"\"\"Susceptibility of populations\"\"\"\n",
    "\n",
    "tags_susceptibility  = \"\"\"susceptibility population\"\"\"\n",
    "\n",
    "searching.SearchDisplayPaper(search=search_susceptibility, pubdate_in=\"2019-12-01 2020-04-01\", topn=5, topn_sent=10, tags=tags_susceptibility)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "                <script>\n",
       "                function visibilite(thingId)\n",
       "                {\n",
       "                var targetElement;\n",
       "                targetElement = document.getElementById(thingId) ;\n",
       "                if (targetElement.style.display == \"none\")\n",
       "                {\n",
       "                targetElement.style.display = \"\" ;\n",
       "                } else {\n",
       "                targetElement.style.display = \"none\" ;\n",
       "                }\n",
       "                }\n",
       "                </script>\n",
       "                </body>\n",
       "                \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 1.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Transmissible Gastroenteritis Virus of Pigs and Porcine Epidemic Diarrhea Virus </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Liu, Qiang; Gerdts, Volker </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2019-12-31 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1016/b978-0-12-809633-8.20928-x ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid120');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid220');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid320');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid120\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">Abstract TGEV and PEDV are porcine coronaviruses with typical coronavirus virion structures and replication cycles. Both viruses cause diarrhea in pigs with similar clinical symptoms. Disease severity is inversely related to the age of animals. While TGEV infections are currently under control, recent emergence of virulent PEDV strains results in significant mortality. No effective treatments are available once animals are infected. Detection of viral antigen and/or genomic RNA is required for diagnosis.<span style=\"color: #ff00ff;\"> Prevent</span>ion<span style=\"color: #ff0000;\"> Measur</span>es include enhanced biosecurity and vaccination with the latter being the most effective approach. Commercial vaccines for both viruses are available and numerous experimental vaccines are in development.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid220\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Enhanced biosecurity procedures are a general<span style=\"color: #ff00ff;\"> Mean</span>s to contain the spread and<span style=\"color: #ff00ff;\"> Prevent</span> the entrance of viral infection in pig farms, but vaccination represents the most effective way in<span style=\"color: #ff00ff;\"> Prevent</span>ing TGEV and PEDV outbreaks.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid320\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">The life cycles of TGEV and PEDV consist of virion attachment and entry, viral RNA translation, viral RNA replication and transcription, virion assembly and release (Fig. 2) . Virion attachment to a host cell requires the interaction between viral S protein with cellular receptors. Aminopeptidase N (APN) is a major receptor for TGEV. However, whether APN is an essential receptor for PEDV is debatable. PEDV S protein can bind to sialic acid on the cell surface that may mediate virion attachment. After receptor binding, the S protein undergoes proteolytic cleavage that in turn induces membrane fusion and virion entry. Viral RNA with a 5′ cap structure and a 3′ polyadenylated tail serves as an mRNA that is translated by cellular translation machinery to generate viral replicase proteins pp1a and pp1ab. Replication and transcription complexes formed by these proteins produce both genomic and subgenomic progeny RNA species. Structural and accessory proteins are translated from subgenomic RNAs. Nucleocapsids formed by the N protein and progeny viral genomic RNA are enveloped in the endoplasmic reticulum -Golgi intermediate compartment with the involvement of S, E, and M proteins. The assembled virions are transported to the cell surface and released. Epidemiology TGEV, first described in 1946 in USA, has been detected all over the world. PEDV was first isolated in 1978 in Belgium and is widespread in Europe and Asia ever since. The appearance of PEDV in North America was reported in 2013 where the virus continues to circulate in swine herds and there is a potential for new PEDV strains to emerge. Both TGEV and PEDV cause enteritis in pigs with very similar clinical symptoms. Major clinical signs include vomiting, watery diarrhea, dehydration, and weight loss. The mortality rate can reach greater than 90% and is inversely related to the age of pigs. Penetration and uncoating occur after the S protein-mediated fusion of the viral envelope with the plasma membrane. Following disassembly, the viral genome is released into the cytoplasm and immediately translated to yield replicases ppla and pp1ab. These polyproteins are proteolytically cleaved into 16 nsps comprising the replication and transcription complex (RTC) that first engages in the minus-strand RNA synthesis using genomic RNA. Both full-and sub genomic (sg)-length minus strands are produced and used to synthesize full-length genomic RNA and sg mRNAs. Each sg mRNA is translated to yield only the protein encoded by the 5'-most ORF of the sg mRNA. The envelope S, E, and M proteins are inserted in the ER and anchored in the Golgi apparatus. The N protein interacts with newly synthesized genomic RNA to form helical RNP complexes. The progeny virus is assembled by budding of the preformed RNP at the ER-Golgi intermediate compartment (ERGIC) and then released by the exocytosis-like fusion of smoothwalled, virion-containing vesicles with the plasma membrane. Reproduced from Lee, C., 2015. Porcine epidemic diarrhea virus: An emerging and re-emerging epizootic swine virus. Virology Journal, 12, 193, under BioMed Central license agreement. Villous enterocytes in small and large intestine are the major target cells of TGEV and PEDV infections. Viral infection causes cell death that results in villous atrophy followed by malabsorption, diarrhea, dehydration, anorexia, and eventually animal death. The molecular mechanisms for pathogenesis are not very well understood. There is evidence to suggest that viral proteins, such as spike and ORF3 proteins may affect viral virulence. At the molecular level, TGEV and PEDV have been shown to modulate multiple cellular processes/pathways including endoplasmic reticulum stress, cell cycle, and mitogen-activated protein kinase signaling. Because clinical signs cannot distinguish between TGEV and PEDV infections, additional assays are required for diagnosis. Common diagnostic assays include viral antigen detection by histoimmunochemistry and enzyme-linked immuosorbent assay (ELISA), viral RNA detection and genotyping by PCR and sequencing, virus isolation by cell culture, as well as antibody detection by serology. There is no specific treatment for TGEV and PEDV infections. Enhanced biosecurity procedures are a general<span style=\"color: #ff00ff;\"> Mean</span>s to contain the spread and<span style=\"color: #ff00ff;\"> Prevent</span> the entrance of viral infection in pig farms, but vaccination represents the most effective way in<span style=\"color: #ff00ff;\"> Prevent</span>ing TGEV and PEDV outbreaks. Because of high mortality in piglets, it has become a common practice to vaccinate sows in order to transfer lactogenic immunity to protect piglets from TGEV and PEDV infections. Live attenuated and inactivated virus vaccines have been developed for both TGEV and PEDV. Since the spike protein is the major immunogen, numerous technologies have been employed to express the spike protein. These include DNA vaccine, vectored vaccine, subunit vaccine, and dendritic cell-based vaccine. Experimental and commercial TGEV and PEDV vaccines are listed in Table 1 .</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 2.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> RNA decay during gammaherpesvirus infection reduces RNA polymerase II occupancy of host promoters but spares viral promoters </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Hartenian, Ella; Gilbertson, Sarah; Federspiel, Joel D.; Cristea, Ileana M.; Glaunsinger, Britt A. </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-07 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7032723/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid121');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid221');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid321');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid121\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">In mammalian cells, widespread acceleration of cytoplasmic mRNA degradation is linked to impaired RNA polymerase II (Pol II) transcription. This mRNA decay-induced transcriptional repression occurs during infection with gammaherpesviruses including Kaposi’s sarcoma-associated herpesvirus (KSHV) and murine gammaherpesvirus 68 (MHV68), which encode an mRNA endonuclease that initiates widespread RNA decay. Here, we show that MHV68-induced mRNA decay leads to a genome-wide reduction of Pol II occupancy at mammalian promoters. This reduced Pol II occupancy is accompanied by down-regulation of multiple Pol II subunits and TFIIB in the nucleus of infected cells, as revealed by mass spectrometry-based<span style=\"color: #ff00ff;\"> Global</span><span style=\"color: #ff0000;\"> Measur</span>ements of protein abundance. Viral genes, despite the fact that they require Pol II for transcription, escape transcriptional repression. Protection is not<span style=\"color: #ff0000;\"> Govern</span>ed by viral promoter sequences; instead, location on the viral genome is both necessary and sufficient to escape the transcriptional repression effects of mRNA decay. We propose a model in which the ability to escape from transcriptional repression is linked to the localization of viral DNA within replication compartments, providing a<span style=\"color: #ff00ff;\"> Mean</span>s for these viruses to counteract decay-induced transcript loss.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid221\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\"></strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid321\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">family Coronaviridae, is an enveloped virus with a singlestranded, positive-sense RNA and causes porcine epidemic diarrhea, an enteric disease characterized by acute watery diarrhea, dehydration, vomiting, and high mortality in nursery piglets (1, 2) . Outbreaks of PEDV infection have been documented mainly in European and Asian countries since the disease was first recognized in an acute outbreak of diarrhea in pigs (1) . In the autumn of 2010, severe diarrhea in piglets emerged in China, affecting more than 1,000,000 piglets (3). A PEDV variant has been identified as a main causative agent in the disease (4) (5) (6) (7) . In this study, a PEDV strain, JS2008, was detected and identified in stool samples from a piglet with severe diarrhea on a vaccinated farm in eastern China. This strain can be easily isolated in Vero cells. Thus, we sought to analyze the genome sequence of PEDV strain JS2008 and understand its molecular characteristics. Twenty pairs of oligonucleotide primers to amplify regions of the JS2008 genome were designed from alignments of available PEDV genomes. The PCR products were purified and cloned into the pMD18-T vector (TaKaRa) and sequenced with an ABI3730XL genome sequencer. The terminal sequences were acquired by using a kit for rapid amplification of cDNA ends (RACE) (Clontech, Japan). The complete genome sequence of JS2008 is 27,953 nucleotides (nt) in length, excluding the poly(A) tail. The genome showed 99.7% and 98.0% nucleotide sequence similarity to the attenuated DR13 and CV777 vaccine strains. It shared 97.3 to 97.5% sequence identity with the PEDV variant that has occurred in recent years in China. The organization of the genome of JS2008 is similar to that of other reported PEDV genomes, with the characteristic gene order 5=-ORF1a/1b-S-ORF3-E-M-N-3= (2, 8, 9) . The ORF1a/1b gene is 20,321 nt in length. The sizes of the other genes, S, ORF3, E, M, and N, are 4,149 nt, 276 nt, 231 nt, 681 nt, and 1,326 nt, respectively. Sequence analysis showed that there was a 49-nt deletion in the ORF3 gene, and this characteristic was similar to a characteristic of the attenuated DR13.<span style=\"color: #ff00ff;\"> Mean</span>while, the E gene was 21 nt longer than that of the attenuated DR13 and had the same sequence as that of the PEDV variant. These findings may indicate that JS2008 is a recombinant of the PEDV vaccine strain and the variant. Phylogenetic analysis of the complete genome showed that JS2008 has a close relationship to the attenuated DR13 vaccine strain and a distant relationship to the prevalent PEDV variant in China. The availability of the genome sequence of JS2008 will facilitate future investigations of the epidemiology and evolution of PEDV. Nucleotide sequence accession number. The genome sequence of PEDV strain JS2008 was deposited in GenBank under accession number KC109141. This work was supported by the Special Fund for Independent Innovation of Agricultural Science and Technology in Jiangsu Province [CX (12) 3076].</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 3.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> A realistic two-strain model for MERS-CoV infection uncovers the high risk for epidemic propagation </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Sardar, Tridip; Ghosh, Indrajit; Rodó, Xavier; Chattopadhyay, Joydev </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-14 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7046297/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid122');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid222');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid322');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid122\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) causes severe acute respiratory illness with a case fatality rate (CFR) of 35,5%. The highest number of MERS-CoV cases are from Saudi-Arabia, the major worldwide hotspot for this disease. In the absence of neither effective treatment nor a ready-to-use vaccine and with yet an incomplete understanding of its epidemiological cycle,<span style=\"color: #ff00ff;\"> Prevent</span>ion and containment<span style=\"color: #ff0000;\"> Measur</span>es can be derived from mathematical models of disease epidemiology. We constructed 2-strain models to predict past outbreaks in the interval 2012–2016 and derive key epidemiological information for Macca, Madina and Riyadh. We approached variability in infection through three different disease incidence functions capturing social behavior in response to an epidemic (e.g. Bilinear, BL; Non-monotone, NM; and Saturated, SAT models). The best model combination successfully anticipated the total number of MERS-CoV clinical cases for the 2015–2016 season and accurately predicted both the number of cases at the peak of seasonal incidence and the overall shape of the epidemic cycle. The evolution in the basic reproduction number (R(0)) warns that MERS-CoV may easily take an epidemic form. The best model correctly captures this feature, indicating a high epidemic risk (1≤R(0)≤2,5) in Riyadh and Macca and confirming the alleged co-circulation of more than one strain. Accurate predictions of the future MERS-CoV peak week, as well as the number of cases at the peak are now possible. These results indicate<span style=\"color: #ff0000;\"> Public</span><span style=\"color: #ff0000;\"> Health</span> agencies should be aware that<span style=\"color: #ff0000;\"> Measur</span>es for strict containment are urgently needed before new epidemics take off in the region.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid222\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The aim of the present study was to<span style=\"color: #ff00ff;\"> Determin</span>e the full-length genome of the first HCoV OC43 isolated from Mexico, which was obtained from a viral metagenomic analysis of a nasal washing from a child who required hospitalization due to clinical or radiological signs of pneumonia of unknown origin, negative for commonly associated respiratory viruses and bacteria, using the xTAG Bioplex respiratory viral panel and multiplex PCR Seeplex Pneumobacter ACE detection kit, respectively (5) .</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid322\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">coronavirus (HCoV) OC43 belongs to the Betacoronavirus genus of the Coronaviridae and its genome is formed by a positivesense, single-stranded RNA of ca. 31.5 kb (1) . HCoV OC43 and HCoV 229E are responsible for one-third of all common colds, infecting all age groups (2), but there have been reports of a more severe lower respiratory tract involvement (3, 4) . The aim of the present study was to<span style=\"color: #ff00ff;\"> Determin</span>e the full-length genome of the first HCoV OC43 isolated from Mexico, which was obtained from a viral metagenomic analysis of a nasal washing from a child who required hospitalization due to clinical or radiological signs of pneumonia of unknown origin, negative for commonly associated respiratory viruses and bacteria, using the xTAG Bioplex respiratory viral panel and multiplex PCR Seeplex Pneumobacter ACE detection kit, respectively (5) . Nucleic acids from the sample were extracted with the PureLink Viral RNA/DNA kit. Before extraction, the sample was treated with Turbo DNase and RNase, and a random-primer amplification of the genetic material was performed as previously described (5) . The sequencing was carried out with the Illumina Genome Analyzer IIx single-end platform. A total of 12,330,418 reads were generated and 1,657,101 (13.44%) were mapped to the consensus sequence of HCoV OC43. Mapping was accomplished by SMALT v.0.7.6 (6) with default parameters (only Ϫy set to 0.8). The whole viral genome sequence was obtained with an average coverage of 3,368ϫ. The same genome sequence was obtained by de novo assembly using Velvet (7) . The complete Mexican HCoV OC43 genome was composed of 30,712 nucleotides, with an average GϩC content of 35.51%. The strain shows the typical OC43 organization, with the following 5= untranslated region (UTR) (nt 1 to 199), genes: orf1ab (nt 200 to Accession number(s). The complete genome sequence of the Mexican OC43 strain was deposited at GenBank under the accession number KX344031.</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 4.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Yonghua Wu </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-04 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1101/2020.03.04.975995 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid123');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid223');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid323');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid123\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid223\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Establishment of viral causes of ILI is critical for<span style=\"color: #ff00ff;\"> Prevent</span>ion and<span style=\"color: #ff0000;\"> Mitig</span>ation strategies to disease threats.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Respiratory pathogens continue to present an ever increasing threat to<span style=\"color: #ff0000;\"> Public</span><span style=\"color: #ff0000;\"> Health</span> (1,2).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Makerere University Walter Reed Project (MUWRP) together with the Ugandan Ministry of<span style=\"color: #ff0000;\"> Health</span> and partners undertook surveillance to<span style=\"color: #ff00ff;\"> Determin</span>e viral causes of influenza-like illness in Uganda.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">From October 2008 to March 2017 a total of 9,472<span style=\"color: #ff00ff;\"> Parti</span>cipants were enrolled in the study from five hospital-based surveillance sentinel sites.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Detection of other viral causes of ILI is an indication of the<span style=\"color: #ff0000;\"> Public</span><span style=\"color: #ff0000;\"> Health</span> threats posed by respiratory pathogens.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid323\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">Respiratory pathogens continue to present an ever increasing threat to<span style=\"color: #ff0000;\"> Public</span><span style=\"color: #ff0000;\"> Health</span> (1,2). Influenza, Respiratory syncytial virus, human metapneumovirus and other respiratory viruses are major etiological agents for influenza like illnesses (ILI) (3-5). Establishment of viral causes of ILI is critical for<span style=\"color: #ff00ff;\"> Prevent</span>ion and<span style=\"color: #ff0000;\"> Mitig</span>ation strategies to disease threats. Makerere University Walter Reed Project (MUWRP) together with the Ugandan Ministry of<span style=\"color: #ff0000;\"> Health</span> and partners undertook surveillance to<span style=\"color: #ff00ff;\"> Determin</span>e viral causes of influenza-like illness in Uganda. From 2008, MUWRP established hospital-based sentinel sites for surveillance activities. A total of five hospital-based sites were established, where patients aged 6 months or older presenting with ILI were enrolled. Consents were obtained as required, and a throat and/ or nasopharyngeal swab collected. Samples were screened by PCR for viral causes. From October 2008 to March 2017 a total of 9,472<span style=\"color: #ff00ff;\"> Parti</span>cipants were enrolled in the study from five hospital-based surveillance sentinel sites. Majority of<span style=\"color: #ff00ff;\"> Parti</span>cipants were children under 5 years n= 8,169 (86.2%). 615 (6.5%) samples tested positive for influenza A, while 385 (4.1%) tested positive for influenza B viruses and 10 (0.1%) were co-infections between influenza A and B. Of the 2,062 influenza negative samples, results indicated positivity for the following organisms; adenoviruses (9.4%), respiratory syncytial B (7.3%), parainfluenza-3 (4.5%), parainfluenza-1 (4.3%), respiratory syncytial A (3.5%), human bocavirus (1.7%), human metapneumovirus (1.7%), human coronavirus (1.5%), parainfluenza-4 (1.4%) and parainfluenza-2 (0.9%) by PCR. Influenza viruses account for about 11% of the causes of influenza like illness, with influenza A being the dominant type. Among the other viral causes of ILI, adenoviruses were the most dominant. Detection of other viral causes of ILI is an indication of the<span style=\"color: #ff0000;\"> Public</span><span style=\"color: #ff0000;\"> Health</span> threats posed by respiratory pathogens. Influenza; Influenza Like Illness; Surveillance</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 5.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Bing Zhao; Chao Ni; Ran Gao; Yuyan Wang; Li Yang; Jinsong Wei; Ting Lv; Jianqing Liang; Qisheng Zhang; Wei Xu; Youhua Xie; Xiaoyue Wang; Zhenghong Yuan; Junbo Liang; Rong Zhang; Xinhua Lin </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-17 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1101/2020.03.16.990317 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid124');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid224');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid324');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid124\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">The newly emerged pandemic coronavirus, SARS-CoV-2, has posed a significant<span style=\"color: #ff0000;\"> Public</span><span style=\"color: #ff0000;\"> Health</span> threat worldwide. However, the mode of virus transmission and tissue tropism is not well established yet. Recent findings of substantial liver damage in patients and ACE2+ cholangiocytes in<span style=\"color: #ff0000;\"> Health</span>y liver tissues prompted us to hypothesize that human liver ductal organoids could serve as a model to<span style=\"color: #ff00ff;\"> Determin</span>e the susceptibility and mechanisms underlining the liver damage upon SARS-CoV-2 infection. By single-cell RNA sequencing, we found that long-term liver ductal organoid culture preserved the human specific ACE2+ population of cholangiocytes. Moreover, human liver ductal organoids were permissive to SARS-CoV-2 infection and support robust replication. Notably, virus infection impaired the barrier and bile acid transporting functions of cholangiocytes through dysregulation of genes involved in tight junction formation and bile acid transportation, which could explain the bile acid accumulation and consequent liver damage in patients. These results indicate that control of liver damage caused directly by viral infection should be valued in treating COVID-19 patients. Our findings also provide an application of human organoids in investigating the tropism and pathogenesis of SARS-CoV-2, which would facilitate novel drug discovery.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid224\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Wuhan (China) has caused emergent and significant threats to<span style=\"color: #ff00ff;\"> Global</span><span style=\"color: #ff0000;\"> Public</span><span style=\"color: #ff0000;\"> Health</span> 1 .</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">88 To establish the SARS-CoV-2 infection model with human liver ductal organoids, 89 we first<span style=\"color: #ff00ff;\"> Determin</span>ed whether the long-term organoid culture could preserve the 90 ACE2+ cholangiocytes ex vivo.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid324\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">Wuhan (China) has caused emergent and significant threats to<span style=\"color: #ff00ff;\"> Global</span><span style=\"color: #ff0000;\"> Public</span><span style=\"color: #ff0000;\"> Health</span> 1 . 48 The dominant symptoms of coronavirus disease 2019 (COVID-19) are fever and 49 cough 2,3 . However, a proportion of patients showed multi-organ damage and 50 dysfunction 2-4 . Of note, liver damage is emerging as a co-existed symptom reported 51 in patients with COVID-19. A recent epidemiologic study in Shanghai (China) 52 reported that 75 out of 148 (50.7%) COVID-19 patients had liver function 53 abnormality, indicated by key liver function parameters above the normal range, 54 including alanine aminotransferase (ALT), aspartate aminotransferase (AST), alkaline 55 phosphatase (ALP) or total bilirubin (TBIL) 5 The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint ACE2+ cholangiocytes are preserved in human liver ductal organoid cultures. 88 To establish the SARS-CoV-2 infection model with human liver ductal organoids, 89 we first<span style=\"color: #ff00ff;\"> Determin</span>ed whether the long-term organoid culture could preserve the 90 ACE2+ cholangiocytes ex vivo. We processed single-cell RNA sequencing (scRNA-seq) 91 to interrogate the transcriptome of cholangiocytes in human liver ductal organoids. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint 109 We next examined the susceptibility of human liver ductal organoids to 110 SARS-CoV-2. We isolated and plaque-purified the SARS-CoV-2 from a COVID-19 111 patient in Shanghai. The liver ductal organoids from two individuals were inoculated 112 with SARS-CoV-2 for 1 hour then re-embedded in Matrigel and maintained in culture Statistical analysis. 249 We employed Student's t-test or ANOVA test to analyze the parametric 250 experimental results. Significant differences were noted with asterisks. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.16.990317 doi: bioRxiv preprint</strong></p>\n",
       "            </div>\n",
       "            "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "search_mitigation = \"\"\"Public health mitigation measures\"\"\"\n",
    "\n",
    "tags_mitigation  = \"\"\"public health mitigation measures government\"\"\"\n",
    "\n",
    "searching.SearchDisplayPaper(search=search_mitigation, pubdate_in=\"2019-12-01 2020-04-01\", topn=5, topn_sent=10, tags=tags_mitigation)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "                <script>\n",
       "                function visibilite(thingId)\n",
       "                {\n",
       "                var targetElement;\n",
       "                targetElement = document.getElementById(thingId) ;\n",
       "                if (targetElement.style.display == \"none\")\n",
       "                {\n",
       "                targetElement.style.display = \"\" ;\n",
       "                } else {\n",
       "                targetElement.style.display = \"none\" ;\n",
       "                }\n",
       "                }\n",
       "                </script>\n",
       "                </body>\n",
       "                \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 1.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> A realistic two-strain model for MERS-CoV infection uncovers the high risk for epidemic propagation </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Sardar, Tridip; Ghosh, Indrajit; Rodó, Xavier; Chattopadhyay, Joydev </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-14 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7046297/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid125');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid225');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid325');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid125\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) causes severe acute respiratory illness with a case fatality rate (CFR) of 35,5%. The highest number of<span style=\"color: #ff00ff;\"> Mer</span>S-CoV cases are from Saudi-Arabia, the major worldwide hotspot for this disease. In the absence of neither effective treatment nor a ready-to-use vaccine and with yet an incomplete understanding of its epidemiological cycle, prevention and containment measures can be derived from mathematical models of disease epidemiology. We constructed 2-strain models to predict past outbreaks in the interval 2012–2016 and derive key epidemiological information for Macca, Madina and Riyadh. We approached variability in infection through three different disease<span style=\"color: #ff00ff;\"> Incid</span>ence functions capturing social behavior in response to an epidemic (e.g. Bilinear, BL; Non-monotone, NM; and Saturated, SAT models). The best model combination successfully anticipated the total number of<span style=\"color: #ff00ff;\"> Mer</span>S-CoV clinical cases for the 2015–2016 season and accurately predicted both the number of cases at the peak of seasonal<span style=\"color: #ff00ff;\"> Incid</span>ence and the overall shape of the epidemic cycle. The evolution in the basic reproduction number (R(0)) warns that<span style=\"color: #ff00ff;\"> Mer</span>S-CoV may easily take an epidemic form. The best model correctly captures this feature, indicating a high epidemic<span style=\"color: #ff0000;\"> Risk</span> (1≤R(0)≤2,5) in Riyadh and Macca and confirming the alleged co-circulation of more than one strain. Accurate predictions of the future<span style=\"color: #ff00ff;\"> Mer</span>S-CoV peak week, as well as the number of cases at the peak are now possible. These results indicate public health agencies should be<span style=\"color: #ff00ff;\"> Awar</span>e that measures for strict containment are urgently needed before new epidemics take off in the region.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid225\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\"></strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid325\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">coronavirus (HCoV) OC43 belongs to the Betacoronavirus genus of the Coronaviridae and its genome is formed by a positivesense, single-stranded RNA of ca. 31.5 kb (1) . HCoV OC43 and HCoV 229E are responsible for one-third of all common colds, infecting all age groups (2), but there have been reports of a more severe lower respiratory tract involvement (3, 4) . The aim of the present study was to determine the full-length genome of the first HCoV OC43 isolated from Mexico, which was obtained from a viral metagenomic analysis of a nasal washing from a child who required hospitalization due to clinical or radiological signs of pneumonia of unknown origin, negative for commonly associated respiratory viruses and bacteria, using the xTAG Bioplex respiratory viral panel and multiplex PCR Seeplex Pneumobacter ACE detection kit, respectively (5) . Nucleic acids from the sample were extracted with the PureLink Viral RNA/DNA kit. Before extraction, the sample was treated with Turbo DNase and RNase, and a random-primer amplification of the genetic material was performed as previously described (5) . The sequencing was carried out with the Illumina Genome Analyzer IIx single-end platform. A total of 12,330,418 reads were generated and 1,657,101 (13.44%) were mapped to the consensus sequence of HCoV OC43. Mapping was accomplished by SMALT v.0.7.6 (6) with default parameters (only Ϫy set to 0.8). The whole viral genome sequence was obtained with an average coverage of 3,368ϫ. The same genome sequence was obtained by de novo assembly using Velvet (7) . The complete Mexican HCoV OC43 genome was composed of 30,712 nucleotides, with an average GϩC content of 35.51%. The strain shows the typical OC43 organization, with the following 5= untranslated region (UTR) (nt 1 to 199), genes: orf1ab (nt 200 to Accession number(s). The complete genome sequence of the Mexican OC43 strain was deposited at GenBank under the accession number KX344031.</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 2.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> RNA decay during gammaherpesvirus infection reduces RNA polymerase II occupancy of host promoters but spares viral promoters </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Hartenian, Ella; Gilbertson, Sarah; Federspiel, Joel D.; Cristea, Ileana M.; Glaunsinger, Britt A. </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-02-07 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7032723/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid126');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid226');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid326');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid126\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">In mammalian cells, widespread acceleration of cytoplasmic mRNA degradation is linked to impaired RNA polymerase II (Pol II) transcription. This mRNA decay-induced transcriptional repression occurs during infection with gammaherpesviruses including Kaposi’s<span style=\"color: #ff00ff;\"> Sar</span>coma-associated herpesvirus (KSHV) and murine gammaherpesvirus 68 (MHV68), which encode an mRNA endonuclease that initiates widespread RNA decay. Here, we show that MHV68-induced mRNA decay leads to a genome-wide reduction of Pol II occupancy at mammalian promoters. This reduced Pol II occupancy is accompanied by down-regulation of multiple Pol II subunits and TFIIB in the nucleus of infected cells, as revealed by mass spectrometry-based global measurements of protein abundance. Viral genes, despite the fact that they require Pol II for transcription, escape transcriptional repression. Protection is not governed by viral promoter sequences; instead, location on the viral genome is both necessary and sufficient to escape the transcriptional repression effects of mRNA decay. We propose a model in which the ability to escape from transcriptional repression is linked to the localization of viral DNA within replication compartments, providing a means for these viruses to counteract decay-induced transcript loss.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid226\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">In the autumn of 2010, severe diarrhea in piglets emerged in<span style=\"color: #ff0000;\"> China</span>, affecting more than 1,000,000 piglets (3).</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">It shared 97.3 to 97.5% sequence identity with the PEDV variant that has occurred in recent years in<span style=\"color: #ff0000;\"> China</span>.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Phylogenetic analysis of the complete genome showed that JS2008 has a close relationship to the attenuated DR13 vaccine strain and a distant relationship to the<span style=\"color: #ff00ff;\"> Preval</span>ent PEDV variant in<span style=\"color: #ff0000;\"> China</span>.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">In this study, a PEDV strain, JS2008, was detected and identified in stool samples from a piglet with severe diarrhea on a vaccinated farm in eastern<span style=\"color: #ff0000;\"> China</span>.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">This work was supported by the Special Fund for Independent Innovation of Agricultural Science and Technology in<span style=\"color: #ff00ff;\"> Jiangsu</span><span style=\"color: #ff00ff;\"> Provinc</span>e [CX (12) 3076].</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid326\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">family Coronaviridae, is an enveloped virus with a singlestranded, positive-sense RNA and causes porcine epidemic diarrhea, an enteric disease characterized by acute watery diarrhea, dehydration, vomiting, and high mortality in nursery piglets (1, 2) . Outbreaks of PEDV infection have been documented mainly in European and Asian countries since the disease was first recognized in an acute outbreak of diarrhea in pigs (1) . In the autumn of 2010, severe diarrhea in piglets emerged in<span style=\"color: #ff0000;\"> China</span>, affecting more than 1,000,000 piglets (3). A PEDV variant has been identified as a main causative agent in the disease (4) (5) (6) (7) . In this study, a PEDV strain, JS2008, was detected and identified in stool samples from a piglet with severe diarrhea on a vaccinated farm in eastern<span style=\"color: #ff0000;\"> China</span>. This strain can be easily isolated in Vero cells. Thus, we sought to analyze the genome sequence of PEDV strain JS2008 and understand its molecular characteristics. Twenty pairs of oligonucleotide primers to amplify regions of the JS2008 genome were designed from alignments of available PEDV genomes. The PCR products were purified and cloned into the pMD18-T vector (TaKaRa) and sequenced with an ABI3730XL genome sequencer. The terminal sequences were acquired by using a kit for rapid amplification of cDNA ends (RACE) (Clontech, Japan). The complete genome sequence of JS2008 is 27,953 nucleotides (nt) in length, excluding the poly(A) tail. The genome showed 99.7% and 98.0% nucleotide sequence similarity to the attenuated DR13 and CV777 vaccine strains. It shared 97.3 to 97.5% sequence identity with the PEDV variant that has occurred in recent years in<span style=\"color: #ff0000;\"> China</span>. The organization of the genome of JS2008 is similar to that of other reported PEDV genomes, with the characteristic gene order 5=-ORF1a/1b-S-ORF3-E-M-N-3= (2, 8, 9) . The ORF1a/1b gene is 20,321 nt in length. The sizes of the other genes, S, ORF3, E, M, and N, are 4,149 nt, 276 nt, 231 nt, 681 nt, and 1,326 nt, respectively. Sequence analysis showed that there was a 49-nt deletion in the ORF3 gene, and this characteristic was similar to a characteristic of the attenuated DR13. Meanwhile, the E gene was 21 nt longer than that of the attenuated DR13 and had the same sequence as that of the PEDV variant. These findings may indicate that JS2008 is a recombinant of the PEDV vaccine strain and the variant. Phylogenetic analysis of the complete genome showed that JS2008 has a close relationship to the attenuated DR13 vaccine strain and a distant relationship to the<span style=\"color: #ff00ff;\"> Preval</span>ent PEDV variant in<span style=\"color: #ff0000;\"> China</span>. The availability of the genome sequence of JS2008 will facilitate future investigations of the epidemiology and evolution of PEDV. Nucleotide sequence accession number. The genome sequence of PEDV strain JS2008 was deposited in GenBank under accession number KC109141. This work was supported by the Special Fund for Independent Innovation of Agricultural Science and Technology in<span style=\"color: #ff00ff;\"> Jiangsu</span><span style=\"color: #ff00ff;\"> Provinc</span>e [CX (12) 3076].</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 3.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Yonghua Wu </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-03-04 </strong></span></p>\n",
       "            <p><a href=https://doi.org/10.1101/2020.03.04.975995 ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid127');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid227');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid327');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid127\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and<span style=\"color: #ff00ff;\"> Sar</span>S-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and<span style=\"color: #ff00ff;\"> Sar</span>S-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between<span style=\"color: #ff0000;\"> Covid</span>-19 and<span style=\"color: #ff00ff;\"> Sar</span>S-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between<span style=\"color: #ff0000;\"> Covid</span>-19 and<span style=\"color: #ff00ff;\"> Sar</span>S-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat<span style=\"color: #ff00ff;\"> Sar</span>S-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and<span style=\"color: #ff00ff;\"> Sar</span>S-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid227\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\"></strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid327\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">Respiratory pathogens continue to present an ever increasing threat to public health (1,2). Influenza, Respiratory syncytial virus, human metapneumovirus and other respiratory viruses are major etiological agents for influenza like illnesses (ILI) (3-5). Establishment of viral causes of ILI is critical for prevention and mitigation strategies to disease threats. Makerere University Walter Reed Project (MUWRP) together with the Ugandan Ministry of Health and partners undertook surveillance to determine viral causes of influenza-like illness in Uganda. From 2008, MUWRP established hospital-based sentinel sites for surveillance activities. A total of five hospital-based sites were established, where patients aged 6 months or older presenting with ILI were enrolled. Consents were obtained as required, and a throat and/ or nasopharyngeal swab collected. Samples were screened by PCR for viral causes. From October 2008 to March 2017 a total of 9,472 participants were enrolled in the study from five hospital-based surveillance sentinel sites. Majority of participants were children under 5 years n= 8,169 (86.2%). 615 (6.5%) samples tested positive for influenza A, while 385 (4.1%) tested positive for influenza B viruses and 10 (0.1%) were co-infections between influenza A and B. Of the 2,062 influenza negative samples, results indicated positivity for the following organisms; adenoviruses (9.4%), respiratory syncytial B (7.3%), parainfluenza-3 (4.5%), parainfluenza-1 (4.3%), respiratory syncytial A (3.5%), human bocavirus (1.7%), human metapneumovirus (1.7%), human coronavirus (1.5%), parainfluenza-4 (1.4%) and parainfluenza-2 (0.9%) by PCR. Influenza viruses account for about 11% of the causes of influenza like illness, with influenza A being the dominant type. Among the other viral causes of ILI, adenoviruses were the most dominant. Detection of other viral causes of ILI is an indication of the public health threats posed by respiratory pathogens. Influenza; Influenza Like Illness; Surveillance</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 4.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> The Fight against the 2019-nCoV Outbreak: an Arduous March Has Just Begun </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Yoo, Jin-Hong </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2020-01-30 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6995816/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid128');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid228');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid328');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid128\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">&lt;![CDATA[No abstract available.]]&gt;</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid228\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">7 As much as the 2015<span style=\"color: #ff00ff;\"> Mer</span>S-CoV outbreak, we are also learning a lot of lesson from this disaster.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Our country is excellent at coping with this disaster, thanks to the experiences that we have gained during the 2015<span style=\"color: #ff00ff;\"> Mer</span>S-CoV outbreak.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The 2019-nCoV is rapidly spreading throughout<span style=\"color: #ff0000;\"> China</span> and around the world in a relatively short period of time.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 4:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">As of 29 January 2020, a total of 6,140 cases were confirmed in 19 countries, including<span style=\"color: #ff00ff;\"> Korea</span>, of which 132 died (in<span style=\"color: #ff0000;\"> China</span> only) and had a mortality of 2.1%.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 5:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The current outbreak is also suspected of being caused by mutants from species spillover in<span style=\"color: #ff00ff;\"> Wuhan</span>'s wild animal market.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 6:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Most coronaviruses cause only mild upper respiratory infections, but sometimes they cause fatal respiratory disease and outbreaks, as experienced in cases of<span style=\"color: #ff00ff;\"> Sar</span>S-CoV or<span style=\"color: #ff00ff;\"> Mer</span>S-CoV.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 7:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Based on the<span style=\"color: #ff00ff;\"> Sar</span>S epidemic precedence, the outbreak is expected to last at least three months.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 8:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">And this outbreak is expected to have a greater amount of transmission than the 2015<span style=\"color: #ff00ff;\"> Mer</span>S-CoV.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 9:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">9,10 After the<span style=\"color: #ff00ff;\"> Mer</span>S-CoV outbreak in 2015, there was a self-tormenting expression in the medical profession society: 'Patients die, hospitals die, and civil servants are praised.' But the bad memories of the past should never be repeated.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 10:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">The<span style=\"color: #ff00ff;\"> Korea</span> Center for Disease Control and Prevention (KCDC) must be the control tower of the present disaster in<span style=\"color: #ff00ff;\"> Korea</span> and all other central or local governmental organizations must cooperate with the KCDC.</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid328\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">This virus is a positive sense single stranded RNA virus belonging to the family Coronaviridae. Most coronaviruses cause only mild upper respiratory infections, but sometimes they cause fatal respiratory disease and outbreaks, as experienced in cases of<span style=\"color: #ff00ff;\"> Sar</span>S-CoV or<span style=\"color: #ff00ff;\"> Mer</span>S-CoV. This disaster has been warned until recently that new mutants of coronavirus can occur anytime. 5, 6 The emergence of these mutants is caused by species jumping between human and other animals. Therefore, it is likely to occur in an environment where human and animals are in close contact. The current outbreak is also suspected of being caused by mutants from species spillover in<span style=\"color: #ff00ff;\"> Wuhan</span>'s wild animal market. The 2019-nCoV is rapidly spreading throughout<span style=\"color: #ff0000;\"> China</span> and around the world in a relatively short period of time. As of 29 January 2020, a total of 6,140 cases were confirmed in 19 countries, including<span style=\"color: #ff00ff;\"> Korea</span>, of which 132 died (in<span style=\"color: #ff0000;\"> China</span> only) and had a mortality of 2.1%. 7 As much as the 2015<span style=\"color: #ff00ff;\"> Mer</span>S-CoV outbreak, we are also learning a lot of lesson from this disaster. Because epidemic is a national disaster, not only medical institutions but also governments have to be active. Hence, honesty and transparency are, above all, the virtues that governments should have. Our country is excellent at coping with this disaster, thanks to the experiences that we have gained during the 2015<span style=\"color: #ff00ff;\"> Mer</span>S-CoV outbreak. 8 Our defense system is at least more solid and faster than it used to be. What do you expect this nCoV outbreak to be in the future? Based on the<span style=\"color: #ff00ff;\"> Sar</span>S epidemic precedence, the outbreak is expected to last at least three months. And this outbreak is expected to have a greater amount of transmission than the 2015<span style=\"color: #ff00ff;\"> Mer</span>S-CoV. Recently, the possibility of transmission by asymptomatic infected people has also been raised carefully although its evidence is unclear. Therefore, we should also prepare for the spread to communities by asymptomatic infections. 9,10 After the<span style=\"color: #ff00ff;\"> Mer</span>S-CoV outbreak in 2015, there was a self-tormenting expression in the medical profession society: 'Patients die, hospitals die, and civil servants are praised.' But the bad memories of the past should never be repeated. In fact, at present, cooperation between community health centers and private hospitals is not always harmonious. The government needs to be more active, not just to leave everything to the medical staffs. After all, the responsibility for the controlling nationwide epidemics lies with health authorities of the government. The<span style=\"color: #ff00ff;\"> Korea</span> Center for Disease Control and Prevention (KCDC) must be the control tower of the present disaster in<span style=\"color: #ff00ff;\"> Korea</span> and all other central or local governmental organizations must cooperate with the KCDC. Toward the end of the 2019-nCoV outbreak we must go on a march of arduousness. Health workers, the government, and the people will need to unite to overcome this disaster.</strong></p>\n",
       "            </div>\n",
       "            \n",
       "            <p>&nbsp;</p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #333300; background-color: #ffffff;\">Paper 5.</span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <h1><span style=\"color: #000000\";>Title:<span style=\"color: #000080;\"> Rapid detection of respiratory organisms with FilmArray respiratory panel and its impact on clinical decisions in Shanghai, China, 2016‐2018 </span></span></h1>\n",
       "            <p>&nbsp;</p>\n",
       "            <p><strong><span style=\"color: #800000; background-color: #999999;\"><span style=\"background-color: #ffffff;\">Authors: Qian, Yiyi; Ai, Jingwen; Wu, Jing; Yu, Shenglei; Cui, Peng; Gao, Yan; Jin, Jialin; Weng, Xinhua; Zhang, Wenhong </span></span></strong></p>\n",
       "            <p><span style=\"color: #003366;\"><strong>Publication date: 2019-12-01 </strong></span></p>\n",
       "            <p><a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7040966/ ><span style=\"color: #3366ff;\"><strong>Link to full paper</strong></span> </a></p>\n",
       "            <p>&nbsp;</p>\n",
       "            <h2>\n",
       "                <a href=\"javascript:visibilite('divid129');\"><span style=\"background-color: ##ffffff;\">Abstract</span></a>\n",
       "                | <a href=\"javascript:visibilite('divid229');\"><span style=\"background-color: ##ffffff;\">Relevant sentences</span></a> |\n",
       "                | <a href=\"javascript:visibilite('divid329');\"><span style=\"background-color: ##ffffff;\">Read paper</span></a> |          \n",
       "            </h2>\n",
       "            <p>&nbsp;</p>\n",
       "            <div id=\"divid129\" style=\"display:none;\"></p>\n",
       "                <h4  style=\"text-align: justify;\">BACKGROUND: In this study, we evaluated the diagnostic potential and clinical impact of an automated multiplex PCR platform (the FilmArray Respiratory Panel; FA‐RP), specially designed for pathogen detection in respiratory tract infections in adults with unexplained pneumonia (UP). METHODS: A total of 112 UP patients in Shanghai,<span style=\"color: #ff0000;\"> China</span>, were enrolled prospectively and assessed using the FA‐RP from October 2016 to March 2018. We examined the test results and their influence on clinical decisions. Furthermore, as a control group, we retrospectively obtained the clinical data of 70 UP patients between October 2014 and March 2016 (before the FA‐RP was available). The two patient groups were compared with respect to factors, including general antimicrobial use and defined daily dose (DDD) numbers. RESULTS: Between October 2016 and March 2018, the positive rate obtained using FA‐RP for UP was 76.8%. The primary pathogens in adults with UP were Influenza A/B (47.3%, 53/112). Compared with the patients before FA‐RP was available, patients who underwent FA‐RP testing had higher rates of antiviral drug use and antibiotic de‐escalation during clinical treatment. FA‐RP significantly decreased the total DDDs of antibiotic or antifungal drugs DDDs by 7 days after admission (10.6 ± 2.5 vs 14.1 ± 8.8, P < .01). CONCLUSIONS: The FA‐RP is a rapid and sensitive nucleic acid amplification test method for UP diagnosis in adults. The application of FA‐RP may lead to a more accurately targeted antimicrobial treatment and reduced use of antibiotic/antifungal drugs.</strong></h4>\n",
       "            </div>\n",
       "            <div id=\"divid229\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">\n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 1:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">After the severe acute respiratory syndrome outbreak, in order to track similarly highly contagious and severe lower respiratory illness with pneumonia symptoms, the<span style=\"color: #ff00ff;\"> Chines</span>e government has paid great attention to unexplained pneumonia (UP), a phrase coined by the<span style=\"color: #ff00ff;\"> Chines</span>e Human Unexplained Pneumonia Surveillance Network (CHUPSN) and published in 2004.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 2:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">Furthermore, the<span style=\"color: #ff00ff;\"> Incid</span>ence of various respiratory viruses may have varied across the two periods that may thus limit the validity of a direct comparison of clinical outcomes.</span></span></h4>\n",
       "                 \n",
       "                <p>&nbsp;</p>\n",
       "                <h3 style=\"text-align: justify;\"><span style=\"color: #800080;\"><strong>Sentence 3:</h3>\n",
       "                <h4 style=\"text-align: justify;\"></strong> <span style=\"color: #000000;\">3  The prospective cohort was enrolled in Shanghai,<span style=\"color: #ff0000;\"> China</span>, between  Within 24 hours of patient admission, respiratory tract specimens were collected, including bronchoalveolar lavage fluid (BALF: among those patients who gave consent to undergo bronchoscopy), sputum (if expectoration was evident), and nasopharyngeal swab (NPS, if no bronchoscopy was performed or no expectoration sample was collected).</span></span></h4>\n",
       "                </strong></p>\n",
       "            </div>\n",
       "            <div id=\"divid329\" style=\"display:none;\"></p>\n",
       "                <p  style=\"text-align: justify;\">After the severe acute respiratory syndrome outbreak, in order to track similarly highly contagious and severe lower respiratory illness with pneumonia symptoms, the<span style=\"color: #ff00ff;\"> Chines</span>e government has paid great attention to unexplained pneumonia (UP), a phrase coined by the<span style=\"color: #ff00ff;\"> Chines</span>e Human Unexplained Pneumonia Surveillance Network (CHUPSN) and published in 2004. UP refers to pneumonia, which could be life-threatening, characterized by rapid progression, a normal white blood cell count range, and poor response to antibiotics. 1 Although the common pathogens causing UP are viruses or atypical pathogens, their overlapping clinical manifestations can impede the ability of clinicians to directly diagnose the causative pathogens, which may lead to unnecessary antimicrobial usage. Thus, rapid and precise diagnosis of the causative agents is critical for the prompt management of UP. To date, culture-based and other traditional methods have been used to help the clinical approach to diagnose UP; however, these methods have certain disadvantages. For example, rapid antigen testing has only a 40%-100% sensitivity for adult influenza and is dependent on the use of specific scientific techniques, whereas culturing viruses can take up to 10 days. 2 In recent years, there has been an increase in the use of nucleic acid amplification test (NAAT)-based methods for the detection of viral pathogens due to their excellent diagnostic ability in identifying a broad spectrum of pathogens. 3  The prospective cohort was enrolled in Shanghai,<span style=\"color: #ff0000;\"> China</span>, between  Within 24 hours of patient admission, respiratory tract specimens were collected, including bronchoalveolar lavage fluid (BALF: among those patients who gave consent to undergo bronchoscopy), sputum (if expectoration was evident), and nasopharyngeal swab (NPS, if no bronchoscopy was performed or no expectoration sample was collected). Notably, lower respiratory tract specimens were given higher priority (BALF > sputum > NPS) during the study. As soon as possible after sampling, the specimens were transferred to the Laboratory of Infectious Diseases Research Institute, Huashan Hospital, Shanghai. Professional pulmonary physicians oversaw BALF sampling, while the collection of NPS and sputum was performed by clinical physicians per the clinical standard operating procedure. Notably, those samples of NPS and BALF for which the viscosity was low required no further sample preparation before FilmArray panel testing, whereas viscous samples, such as sputum, were pre-treated using the previously described dunk and swirl method. 5 The remaining specimens were stored in a −80°C freezer for further use. In this study, we used version 1.7 of the FA-RP. It is an automated multiplex PCR detection platform designed to detect 20 common pathogens causing respiratory infection and has a single specimen turnaround time of <2 hours. Tests using the panel were performed following the manufacturer's instructions. For specimens that the FA-RP reported as \"Influenza A, not sub-  For both patient groups, traditional standard-of-care laboratory diagnostic tests for respiratory infections were performed according to the requests of physicians, including smears, cultures, and serological tests. The serological test in our study was the PNEUMOSLIDE IgM (Vircell) commercial kit based on indirect immunofluorescence. In this study, clinical data, including antimicrobial use, and adjustment or de-escalation of antimicrobial therapy, were compared between the FA-RP group and retrospective control group. Turnaround time was defined as the length of time from sample collection to receiving the FA-RP results. Antimicrobial use was quantified as the defined daily dose (DDD)) number, calculated as ∑ (Drug Total Dosage (g)/Drug DDD index). 6 Adjustment of antimicrobial therapy was defined as a change in antimicrobial regimen immediately after receiving the FA-RP results. Overview of Patient Enrollment Workflow. A total of 112 patients selected prospectively were submitted to FA-RP analyses and are referred to as the \"FA-RP group.\" Those who were not lost to follow-up and had complete data are referred to as the \"intervention group.\" Comparison was made between it and the historical control group The Mann-Whitney U test was used for the comparison of quantitative data. spss 20.0 (IBM) was used for data analyses. We used Graphpad 6.0 for drawing graphs. The Ethics Committee of the Huashan Hospital, Shanghai, approved the study (KY2013-251; July 2, 2013). All participants gave their written informed consent. Table 1 . Mixed infections of multiple pathogens were detected in six patients: Influenza virus/RSV (n = 1), Influenza virus/human metapneumovirus (n = 1), Influenza virus/parainfluenza-2 (n = 1), M pneumoniae/human rhinovirus/enterovirus (n = 1), human rhinovirus/ enterovirus/parainfluenza-2 (n = 1), and adenovirus/coronavirus HKU1/coronavirus OC43 (n = 1). For 13 samples, the FA-RP results were \"Influenza A/not subtyped.\" In these cases, further subtyping was performed using real-time RT-PCR. As a consequence, Influenza A virus subtype H7N9 was detected in 11 samples and H1N1-2009 in the remaining two samples. As a consequence of the findings of the FA-FP regarding Influenza A, 15 patients were isolated in a negative pressure ward or single ward, including seven highly pathogenic H7N9 cases. In contrast, in the control group, no comparable isolation measures were undertaken. When we compared antimicrobial use between the intervention and control groups, we found that the two groups were comparable for the most baseline characteristics (Table 3) . Compared with the control group, the intervention group showed no significant decrease of antimicrobial DDDs (control group 14.3 ± 9.6 vs intervention group 13.8 ± 4.6, P > .05). However, after excluding antiviral drugs, we detected significantly lower antibiotic/antifungal DDDs  In this study, we applied FA-RP on diagnosis of UP, which is a certain type of pneumonia causing mainly by viruses and atypical pathogens. We found that the FA-RP detected pathogens in 76.8% (86/112) of all specimens, which is higher than that the detection rate obtained when this technique was applied for general community-acquired pneumonia (38.6%, 49/127). 7 These results, therefore, show that when applied clinically for UP, the FA-RP would provide a higher probability of identifying causal pathogenic microorganisms. In the present study, we found that 53.8% (60/112) of the patients in the FA-RP group received antiviral treatment, which is significantly higher than that in the control group (14.3%, 10/70; randomized controlled trial discovered that use of the FA-RP was not associated with a reduction in the overall duration of antibiotic usage. 18 Nevertheless, a higher number of patients in the FA-RP group received single doses or brief courses of antibiotics than those in the control group. Given these inconclusive findings, further studies are required concerning the impact of FA-RP on the clinical decisions regarding pneumonia. We compared the duration of hospital stay between the two groups. As a result, intervention group unexpectedly showed longer duration of hospital stay than the control group (12.1 vs 7.6 days, P = .001). One reasonable explanation is that subjects in the intervention group had more severe cases of pneumonia than those in the control group (higher PSI, This present study is one of the several that have examined the application of FA-RP in Asia. [19] [20] [21] [22] [23] [24] Compared with other published studies, our study focused on adults with more severe pneumonia conditions, and to the best of our knowledge, it is the only study that Our study does, however, have a few limitations. Notably, the sample size was limited, and the study was conducted only at a single location, which may thus limit the general applicability of the findings. In addition, the study included two cohorts from different time periods; hence, there may have been differences in factors that influenced clinician behavior between the two periods that were not accounted in our analysis. Furthermore, the<span style=\"color: #ff00ff;\"> Incid</span>ence of various respiratory viruses may have varied across the two periods that may thus limit the validity of a direct comparison of clinical outcomes. In conclusion, the FA-RP is a rapid and sensitive NAAT method that can be used for the diagnosis of UP in adults. In the two recent years in Shanghai, the most common pathogen causing UP was identified as the Influenza virus. However, in this regard, it should be taken into consideration that the current version of the FA-RP is unable to directly detect H7N9. Nevertheless, the results obtained in this study show that the FA-RP may make a valuable contribution to clinical decision making and facilitate the reasonable use of antibiotics. We would like to thank Yinlei Zhang, Tracy Wang, Xuyang Wang, and Xinchang Chen for their substantial contributions to this study. Yiyi Qian https://orcid.org/0000-0002-0714-3894</strong></p>\n",
       "            </div>\n",
       "            "
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<function ipywidgets.widgets.interaction._InteractFactory.__call__.<locals>.<lambda>(*args, **kwargs)>"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "interact(searching.SearchDisplayPaper, search=\"What are covid-19 risks factors?\", \n",
    "         pubdate_before=\"\", pubdate_after=\"\", pubdate_in=\"2019-12-01 2020-04-01\",\n",
    "         topn=5, topn_sent=10, tags=\"risk covid China\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {
    "height": "182px",
    "width": "367.5px"
   },
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": true,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": false
  },
  "widgets": {
   "application/vnd.jupyter.widget-state+json": {
    "state": {
     "05c79225476042c7af4f16c366617658": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "SliderStyleModel",
      "state": {
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "SliderStyleModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "StyleView",
       "description_width": "",
       "handle_color": null
      }
     },
     "05df4f9595d54cb89053ca55eee66b96": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "TextModel",
      "state": {
       "_dom_classes": [],
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "TextModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/controls",
       "_view_module_version": "1.5.0",
       "_view_name": "TextView",
       "continuous_update": true,
       "description": "pubdate_after",
       "description_tooltip": null,
       "disabled": false,
       "layout": "IPY_MODEL_849e531f003a49a89b13fd7a483ffac0",
       "placeholder": "​",
       "style": "IPY_MODEL_c53c4f52fffd4eecad49e33fe16a52e0",
       "value": ""
      }
     },
     "0d0e233a546e42c690ceb81d9c619368": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "_model_module": "@jupyter-widgets/base",
       "_model_module_version": "1.2.0",
       "_model_name": "LayoutModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "LayoutView",
       "align_content": null,
       "align_items": null,
       "align_self": null,
       "border": null,
       "bottom": null,
       "display": null,
       "flex": null,
       "flex_flow": null,
       "grid_area": null,
       "grid_auto_columns": null,
       "grid_auto_flow": null,
       "grid_auto_rows": null,
       "grid_column": null,
       "grid_gap": null,
       "grid_row": null,
       "grid_template_areas": null,
       "grid_template_columns": null,
       "grid_template_rows": null,
       "height": null,
       "justify_content": null,
       "justify_items": null,
       "left": null,
       "margin": null,
       "max_height": null,
       "max_width": null,
       "min_height": null,
       "min_width": null,
       "object_fit": null,
       "object_position": null,
       "order": null,
       "overflow": null,
       "overflow_x": null,
       "overflow_y": null,
       "padding": null,
       "right": null,
       "top": null,
       "visibility": null,
       "width": null
      }
     },
     "146549536b004999a9be54bedca0c0ff": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "DescriptionStyleModel",
      "state": {
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "DescriptionStyleModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "StyleView",
       "description_width": ""
      }
     },
     "2c5f0e9c2fc646fc881cf05b413073fe": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "DescriptionStyleModel",
      "state": {
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "DescriptionStyleModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "StyleView",
       "description_width": ""
      }
     },
     "404149b091e64c2886af02f5b5585661": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "DescriptionStyleModel",
      "state": {
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "DescriptionStyleModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "StyleView",
       "description_width": ""
      }
     },
     "456fdfc699824b3c9040adcb607b827e": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "_model_module": "@jupyter-widgets/base",
       "_model_module_version": "1.2.0",
       "_model_name": "LayoutModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "LayoutView",
       "align_content": null,
       "align_items": null,
       "align_self": null,
       "border": null,
       "bottom": null,
       "display": null,
       "flex": null,
       "flex_flow": null,
       "grid_area": null,
       "grid_auto_columns": null,
       "grid_auto_flow": null,
       "grid_auto_rows": null,
       "grid_column": null,
       "grid_gap": null,
       "grid_row": null,
       "grid_template_areas": null,
       "grid_template_columns": null,
       "grid_template_rows": null,
       "height": null,
       "justify_content": null,
       "justify_items": null,
       "left": null,
       "margin": null,
       "max_height": null,
       "max_width": null,
       "min_height": null,
       "min_width": null,
       "object_fit": null,
       "object_position": null,
       "order": null,
       "overflow": null,
       "overflow_x": null,
       "overflow_y": null,
       "padding": null,
       "right": null,
       "top": null,
       "visibility": null,
       "width": null
      }
     },
     "46ea633952e0454faac2e4c53ea0774b": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "_model_module": "@jupyter-widgets/base",
       "_model_module_version": "1.2.0",
       "_model_name": "LayoutModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "LayoutView",
       "align_content": null,
       "align_items": null,
       "align_self": null,
       "border": null,
       "bottom": null,
       "display": null,
       "flex": null,
       "flex_flow": null,
       "grid_area": null,
       "grid_auto_columns": null,
       "grid_auto_flow": null,
       "grid_auto_rows": null,
       "grid_column": null,
       "grid_gap": null,
       "grid_row": null,
       "grid_template_areas": null,
       "grid_template_columns": null,
       "grid_template_rows": null,
       "height": null,
       "justify_content": null,
       "justify_items": null,
       "left": null,
       "margin": null,
       "max_height": null,
       "max_width": null,
       "min_height": null,
       "min_width": null,
       "object_fit": null,
       "object_position": null,
       "order": null,
       "overflow": null,
       "overflow_x": null,
       "overflow_y": null,
       "padding": null,
       "right": null,
       "top": null,
       "visibility": null,
       "width": null
      }
     },
     "5681e0643efe4ef0afd43076015d06b4": {
      "model_module": "@jupyter-widgets/output",
      "model_module_version": "1.0.0",
      "model_name": "OutputModel",
      "state": {
       "_dom_classes": [],
       "_model_module": "@jupyter-widgets/output",
       "_model_module_version": "1.0.0",
       "_model_name": "OutputModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/output",
       "_view_module_version": "1.0.0",
       "_view_name": "OutputView",
       "layout": "IPY_MODEL_97b61172f06c476bae683b153056401e",
       "msg_id": "",
       "outputs": []
      }
     },
     "667b11eeaad9445189901dc450c31d4c": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "_model_module": "@jupyter-widgets/base",
       "_model_module_version": "1.2.0",
       "_model_name": "LayoutModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "LayoutView",
       "align_content": null,
       "align_items": null,
       "align_self": null,
       "border": null,
       "bottom": null,
       "display": null,
       "flex": null,
       "flex_flow": null,
       "grid_area": null,
       "grid_auto_columns": null,
       "grid_auto_flow": null,
       "grid_auto_rows": null,
       "grid_column": null,
       "grid_gap": null,
       "grid_row": null,
       "grid_template_areas": null,
       "grid_template_columns": null,
       "grid_template_rows": null,
       "height": null,
       "justify_content": null,
       "justify_items": null,
       "left": null,
       "margin": null,
       "max_height": null,
       "max_width": null,
       "min_height": null,
       "min_width": null,
       "object_fit": null,
       "object_position": null,
       "order": null,
       "overflow": null,
       "overflow_x": null,
       "overflow_y": null,
       "padding": null,
       "right": null,
       "top": null,
       "visibility": null,
       "width": null
      }
     },
     "695eaba57e304ffa9d3989ae469eb813": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "TextModel",
      "state": {
       "_dom_classes": [],
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "TextModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/controls",
       "_view_module_version": "1.5.0",
       "_view_name": "TextView",
       "continuous_update": true,
       "description": "search",
       "description_tooltip": null,
       "disabled": false,
       "layout": "IPY_MODEL_456fdfc699824b3c9040adcb607b827e",
       "placeholder": "​",
       "style": "IPY_MODEL_9f8247bf579a47d583c0af288b1f8f0a",
       "value": "What are covid-19 risks factors?"
      }
     },
     "6e821d5a92bb49cd8e0c79416d2a4eae": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "TextModel",
      "state": {
       "_dom_classes": [],
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "TextModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/controls",
       "_view_module_version": "1.5.0",
       "_view_name": "TextView",
       "continuous_update": true,
       "description": "pubdate_before",
       "description_tooltip": null,
       "disabled": false,
       "layout": "IPY_MODEL_667b11eeaad9445189901dc450c31d4c",
       "placeholder": "​",
       "style": "IPY_MODEL_146549536b004999a9be54bedca0c0ff",
       "value": ""
      }
     },
     "6f8909e3c3e94e7faea64063f75f42a3": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "IntSliderModel",
      "state": {
       "_dom_classes": [],
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "IntSliderModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/controls",
       "_view_module_version": "1.5.0",
       "_view_name": "IntSliderView",
       "continuous_update": true,
       "description": "topn",
       "description_tooltip": null,
       "disabled": false,
       "layout": "IPY_MODEL_46ea633952e0454faac2e4c53ea0774b",
       "max": 15,
       "min": -5,
       "orientation": "horizontal",
       "readout": true,
       "readout_format": "d",
       "step": 1,
       "style": "IPY_MODEL_05c79225476042c7af4f16c366617658",
       "value": 5
      }
     },
     "849e531f003a49a89b13fd7a483ffac0": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "_model_module": "@jupyter-widgets/base",
       "_model_module_version": "1.2.0",
       "_model_name": "LayoutModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "LayoutView",
       "align_content": null,
       "align_items": null,
       "align_self": null,
       "border": null,
       "bottom": null,
       "display": null,
       "flex": null,
       "flex_flow": null,
       "grid_area": null,
       "grid_auto_columns": null,
       "grid_auto_flow": null,
       "grid_auto_rows": null,
       "grid_column": null,
       "grid_gap": null,
       "grid_row": null,
       "grid_template_areas": null,
       "grid_template_columns": null,
       "grid_template_rows": null,
       "height": null,
       "justify_content": null,
       "justify_items": null,
       "left": null,
       "margin": null,
       "max_height": null,
       "max_width": null,
       "min_height": null,
       "min_width": null,
       "object_fit": null,
       "object_position": null,
       "order": null,
       "overflow": null,
       "overflow_x": null,
       "overflow_y": null,
       "padding": null,
       "right": null,
       "top": null,
       "visibility": null,
       "width": null
      }
     },
     "882ae7f3bad64a22abfa5103008408f8": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "_model_module": "@jupyter-widgets/base",
       "_model_module_version": "1.2.0",
       "_model_name": "LayoutModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "LayoutView",
       "align_content": null,
       "align_items": null,
       "align_self": null,
       "border": null,
       "bottom": null,
       "display": null,
       "flex": null,
       "flex_flow": null,
       "grid_area": null,
       "grid_auto_columns": null,
       "grid_auto_flow": null,
       "grid_auto_rows": null,
       "grid_column": null,
       "grid_gap": null,
       "grid_row": null,
       "grid_template_areas": null,
       "grid_template_columns": null,
       "grid_template_rows": null,
       "height": null,
       "justify_content": null,
       "justify_items": null,
       "left": null,
       "margin": null,
       "max_height": null,
       "max_width": null,
       "min_height": null,
       "min_width": null,
       "object_fit": null,
       "object_position": null,
       "order": null,
       "overflow": null,
       "overflow_x": null,
       "overflow_y": null,
       "padding": null,
       "right": null,
       "top": null,
       "visibility": null,
       "width": null
      }
     },
     "97b61172f06c476bae683b153056401e": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "_model_module": "@jupyter-widgets/base",
       "_model_module_version": "1.2.0",
       "_model_name": "LayoutModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "LayoutView",
       "align_content": null,
       "align_items": null,
       "align_self": null,
       "border": null,
       "bottom": null,
       "display": null,
       "flex": null,
       "flex_flow": null,
       "grid_area": null,
       "grid_auto_columns": null,
       "grid_auto_flow": null,
       "grid_auto_rows": null,
       "grid_column": null,
       "grid_gap": null,
       "grid_row": null,
       "grid_template_areas": null,
       "grid_template_columns": null,
       "grid_template_rows": null,
       "height": null,
       "justify_content": null,
       "justify_items": null,
       "left": null,
       "margin": null,
       "max_height": null,
       "max_width": null,
       "min_height": null,
       "min_width": null,
       "object_fit": null,
       "object_position": null,
       "order": null,
       "overflow": null,
       "overflow_x": null,
       "overflow_y": null,
       "padding": null,
       "right": null,
       "top": null,
       "visibility": null,
       "width": null
      }
     },
     "9f8247bf579a47d583c0af288b1f8f0a": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "DescriptionStyleModel",
      "state": {
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "DescriptionStyleModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "StyleView",
       "description_width": ""
      }
     },
     "aed339f4eaf24ebcbe33205bd7d36c49": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "TextModel",
      "state": {
       "_dom_classes": [],
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "TextModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/controls",
       "_view_module_version": "1.5.0",
       "_view_name": "TextView",
       "continuous_update": true,
       "description": "pubdate_in",
       "description_tooltip": null,
       "disabled": false,
       "layout": "IPY_MODEL_b1c78979a25344f4897d40ce6469d135",
       "placeholder": "​",
       "style": "IPY_MODEL_2c5f0e9c2fc646fc881cf05b413073fe",
       "value": "2019-12-01 2020-04-01"
      }
     },
     "b1c78979a25344f4897d40ce6469d135": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "_model_module": "@jupyter-widgets/base",
       "_model_module_version": "1.2.0",
       "_model_name": "LayoutModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "LayoutView",
       "align_content": null,
       "align_items": null,
       "align_self": null,
       "border": null,
       "bottom": null,
       "display": null,
       "flex": null,
       "flex_flow": null,
       "grid_area": null,
       "grid_auto_columns": null,
       "grid_auto_flow": null,
       "grid_auto_rows": null,
       "grid_column": null,
       "grid_gap": null,
       "grid_row": null,
       "grid_template_areas": null,
       "grid_template_columns": null,
       "grid_template_rows": null,
       "height": null,
       "justify_content": null,
       "justify_items": null,
       "left": null,
       "margin": null,
       "max_height": null,
       "max_width": null,
       "min_height": null,
       "min_width": null,
       "object_fit": null,
       "object_position": null,
       "order": null,
       "overflow": null,
       "overflow_x": null,
       "overflow_y": null,
       "padding": null,
       "right": null,
       "top": null,
       "visibility": null,
       "width": null
      }
     },
     "c469f5c540e9437cbfa5c53baa33139d": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "VBoxModel",
      "state": {
       "_dom_classes": [
        "widget-interact"
       ],
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "VBoxModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/controls",
       "_view_module_version": "1.5.0",
       "_view_name": "VBoxView",
       "box_style": "",
       "children": [
        "IPY_MODEL_695eaba57e304ffa9d3989ae469eb813",
        "IPY_MODEL_6e821d5a92bb49cd8e0c79416d2a4eae",
        "IPY_MODEL_05df4f9595d54cb89053ca55eee66b96",
        "IPY_MODEL_aed339f4eaf24ebcbe33205bd7d36c49",
        "IPY_MODEL_6f8909e3c3e94e7faea64063f75f42a3",
        "IPY_MODEL_cb601a53fac747b08e81c73ed38b747e",
        "IPY_MODEL_f690fce5553d4d06ae0ee4894ffbf940",
        "IPY_MODEL_5681e0643efe4ef0afd43076015d06b4"
       ],
       "layout": "IPY_MODEL_d96fe7170665420daf1f197fe0967cb5"
      }
     },
     "c53c4f52fffd4eecad49e33fe16a52e0": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "DescriptionStyleModel",
      "state": {
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "DescriptionStyleModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "StyleView",
       "description_width": ""
      }
     },
     "cb601a53fac747b08e81c73ed38b747e": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "IntSliderModel",
      "state": {
       "_dom_classes": [],
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "IntSliderModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/controls",
       "_view_module_version": "1.5.0",
       "_view_name": "IntSliderView",
       "continuous_update": true,
       "description": "topn_sent",
       "description_tooltip": null,
       "disabled": false,
       "layout": "IPY_MODEL_882ae7f3bad64a22abfa5103008408f8",
       "max": 30,
       "min": -10,
       "orientation": "horizontal",
       "readout": true,
       "readout_format": "d",
       "step": 1,
       "style": "IPY_MODEL_d994b42383c345f8a4e5d4edd47a2e3c",
       "value": 10
      }
     },
     "d96fe7170665420daf1f197fe0967cb5": {
      "model_module": "@jupyter-widgets/base",
      "model_module_version": "1.2.0",
      "model_name": "LayoutModel",
      "state": {
       "_model_module": "@jupyter-widgets/base",
       "_model_module_version": "1.2.0",
       "_model_name": "LayoutModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "LayoutView",
       "align_content": null,
       "align_items": null,
       "align_self": null,
       "border": null,
       "bottom": null,
       "display": null,
       "flex": null,
       "flex_flow": null,
       "grid_area": null,
       "grid_auto_columns": null,
       "grid_auto_flow": null,
       "grid_auto_rows": null,
       "grid_column": null,
       "grid_gap": null,
       "grid_row": null,
       "grid_template_areas": null,
       "grid_template_columns": null,
       "grid_template_rows": null,
       "height": null,
       "justify_content": null,
       "justify_items": null,
       "left": null,
       "margin": null,
       "max_height": null,
       "max_width": null,
       "min_height": null,
       "min_width": null,
       "object_fit": null,
       "object_position": null,
       "order": null,
       "overflow": null,
       "overflow_x": null,
       "overflow_y": null,
       "padding": null,
       "right": null,
       "top": null,
       "visibility": null,
       "width": null
      }
     },
     "d994b42383c345f8a4e5d4edd47a2e3c": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "SliderStyleModel",
      "state": {
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "SliderStyleModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/base",
       "_view_module_version": "1.2.0",
       "_view_name": "StyleView",
       "description_width": "",
       "handle_color": null
      }
     },
     "f690fce5553d4d06ae0ee4894ffbf940": {
      "model_module": "@jupyter-widgets/controls",
      "model_module_version": "1.5.0",
      "model_name": "TextModel",
      "state": {
       "_dom_classes": [],
       "_model_module": "@jupyter-widgets/controls",
       "_model_module_version": "1.5.0",
       "_model_name": "TextModel",
       "_view_count": null,
       "_view_module": "@jupyter-widgets/controls",
       "_view_module_version": "1.5.0",
       "_view_name": "TextView",
       "continuous_update": true,
       "description": "tags",
       "description_tooltip": null,
       "disabled": false,
       "layout": "IPY_MODEL_0d0e233a546e42c690ceb81d9c619368",
       "placeholder": "​",
       "style": "IPY_MODEL_404149b091e64c2886af02f5b5585661",
       "value": "risk covid China"
      }
     }
    },
    "version_major": 2,
    "version_minor": 0
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
